cli-output/src/cli_output.rs

```diff
@@ -343,18 +343,24 @@ impl fmt::Display for CliValidators {
             validator: &CliValidator,
             total_active_stake: u64,
             use_lamports_unit: bool,
+            highest_last_vote: u64,
+            highest_root: u64,
         ) -> fmt::Result {
-            fn non_zero_or_dash(v: u64) -> String {
+            fn non_zero_or_dash(v: u64, max_v: u64) -> String {
                 if v == 0 {
-                    "-".into()
+                    "-         ".into()
+                } else if v == max_v {
+                    format!("{:>8}      ", v)
+                } else if v > max_v.saturating_sub(100) {
+                    format!("{:>8} ({:>3})", v, -(max_v.saturating_sub(v) as isize))
                 } else {
-                    format!("{}", v)
+                    format!("{:>8} (far)", v)
```

Comment:
I don't love `far`, since it's not used elsewhere or defined, and it drops the minus symbol so "behind" is only implied by the non-`far` values (I think this might be less clear when using a sort that intermixes far/non-far).

But what's your workflow here? Is it best to have something distinct, ie. letters instead of numbers?

Reply:
I was interested in getting a quick sense from `solana validators` for how far the validators are the tip of the chain are from each other.   Normally everybody is within -10/-20 of each other, and beyond -50 are the delinquents and it gets much less interesting.  

Here's an alternative, empty space for the delinquents and `(  0)` for the highest nodes:
```
  Identity                                      Vote Account                            Commission  Last Vote        Root Slot      Epoch Credits Version  Active Stake
  wWf94sVnaXHzBYrePsRUyesq6ofndocfBH6EmzdgKMS   AKoVXpZmi8wSz3sGvCYEygbpdHvSRysWsh36b97iPvKh  100%   51687937 ( -3)  51687906 ( -3)        279942  1.5.18  -
⚠️ 4GskhQsHVuMv68guHRwr5f5DgL9vHEMcV1KfNcvS7VhG  Mx7uRGuYtxknJKuXQz3mv5EkiydRXeDXWDjNdgYmM8F   100%   50095833        50095802                   0   1.6.4  0.99771712 SOL (0.00%)
⚠️ 8KeYrkYrNSq1x5zDQqghEiXFmf4QjYUWUzVt6qwiBX5i  775UhDftsU4SMEUHK8nCK1P5o4eGnoGwX6FdhBbA6LtC  100%   44829320        44829289                   0 unknown  0.99771712 SOL (0.00%)
⚠️ 2MeGu4yCwRCthwzFiVtsLAVSa6xCsdH59cz6F6J6b33U  GrnK4vPzYLzFdnirZCBXjLSvKG4iF6pHdZ67C9rZsbSG  100%   36055114        36055083                   0 unknown  0.99771712 SOL (0.00%)
⚠️ 6xDdF6YUAKVfj6AL3TMJckQokXhrxtBwMiuHVMnSHxR   GamfjAaa1y2vNbR54vfQM3Q5NrrQ5xCDtdoVC93jWiJ7  100%   35371461        35371430                   0 unknown  0.99771712 SOL (0.00%)
⚠️ AJib74LMUmCfkRN84iEWApuhGzEpQcA4g7KC1FVgqWCt  72h1eypUDXm6ks6X8uWbpuzKgRNZiCRSwnefFHjGyxEa  100%   46034764        46034733                   0 unknown  0.99771712 SOL (0.00%)
⚠️ 7QqYu69Sh5WB58JjXEmnuKVfvybC4dxWTsDFmNqiY9d4  2Y6BbUbkcEKdnAr9ED5bLBH6nC4XnfBUnjrVcRghozcw  100%   45329091        45329060                   0 unknown  0.99843424 SOL (0.00%)
⚠️ 4mMCNHeEZwqkM99HAzQMbiPFCXhsC3VAo56BkFoTVHoY  7GHnTRB8Rz14qZQhDXf8ox1Kfu7mPcPLpKaBJJirmYj2    8%   48268420        48268389                   0 unknown  1.001521874 SOL (0.00%)
⚠️ BmSmUWB9gJkoePfnmnKGtcG9MXnW1VKDvvTRman8F1Te  BGMDEqc5TZ3xkSKWeGtiAHPNkbc4GeBqnQdNJry2xGth    0%   51517188        51517157              109190   1.6.6  1.284048473 SOL (0.00%)
⚠️ 5cntEEYR5DMoDucsjQKCjHyhR7CDbytAeW1Y8osHAd3R  CrinLuHjVGDDcQfrEoCmM4k31Ni9sMoTCEEvNSUSh7Jg  100%   41470176        41470145                   0 unknown  1.29315136 SOL (0.00%)
⚠️ G7131eBQAa4NhX8V3JXRZwctZRraLMdUPcAAH1M5xpDo  7Lm4wRjaH7CvrzaD4oRrXFu6FFdCreYupFQHouh8Uojy  100%   35126559        35126528                   0 unknown  1.88630272 SOL (0.00%)
⚠️ 82BPXtNdKBhiEEFdB9Vm6dJpj6jGuoUxEKHxZWiBcgyX  BbLFfmDiViMAXZZf1eLHRtQAuFS1aG8PJ2AmE5NLm1mY   10%   49514508        49514477                   0 unknown  2.035346098 SOL (0.00%)
⚠️ E4aMcPhHEoyLxMW8jJgMkcyK1S6qNhjjZGsyhCY4UiAd  GemaKPJ7cWmVsqbb5T5z5MyDYLreRaUqv7MswYbmqgSV  100%   40239267        40239236                   0 unknown  29.068831918 SOL (0.00%)
⚠️ H9kSDiJJFVoLjpmue8nKhvLteZfzjuzS8AjUtBovxBZE  GNkBiAHKTk5RYxi1bSEKc31xibMqSBTKtLkRbXhi1h8K  100%   32427525        32427494                   0 unknown  82.99543424 SOL (0.00%)
⚠️ DctYdX8c3qBZ7RUtYE4Ffunjv5SYFxVde4H4BDejPzMG  HK456zW1akyEDBdmvcBddyvxfefGCvTBszKBakAZPR3S  100%   12223953        12223922                   0 unknown  95.48236992 SOL (0.00%)
⚠️ EUcJwf7jXskRE6NZBtFPVH2EedNvNYko8LL2WT62XctB  Bwkz1ddKoGE8hgiSV6HZLXi9RBLqfBi3HZb2QujzVGgz  100%   39257131        39257100                   0 unknown  115.99315136 SOL (0.00%)
⚠️ FSgwdAhy9JAePSKtrCrDFmMFpKwufd1WVBimHRc2dDEk  85ShayF5HDXzZxxey8Qd89VSCfBxCgtmuL4LJx6GTkr2    9%   44780479        44780448                   0 unknown  149.24315136 SOL (0.00%)
⚠️ 815YLNgAsnMqWJo1dmy13oKwWxujq31Dh6KhCUirnhBh  DTmhJTdXM8QBQ5p4mL2v6XzVqin5t9RdtFQMkXctu2La   10%   38045501        38045470                   0 unknown  1013.495481318 SOL (0.00%)
  dv1LfzJvDF7S1fBKpFgKoKXK5yoSosmkAdfbxBo1GqJ   5MMCR4NbTZqjthjLGywmeT66iwE9J9f7kjtxzJjwfUx2    0%   51687940 (  0)  51687909 (  0)        279942   1.6.6  77917479.472601518 SOL (100.00%)
```

Reply:
I had `(xxx)` for the < -99 nodes at one point too:
```
  Identity                                      Vote Account                            Commission  Last Vote        Root Slot      Epoch Credits Version  Active Stake
  wWf94sVnaXHzBYrePsRUyesq6ofndocfBH6EmzdgKMS   AKoVXpZmi8wSz3sGvCYEygbpdHvSRysWsh36b97iPvKh  100%   51688180 ( -4)  51688149 ( -4)        280185  1.5.18  -
⚠️ 4GskhQsHVuMv68guHRwr5f5DgL9vHEMcV1KfNcvS7VhG  Mx7uRGuYtxknJKuXQz3mv5EkiydRXeDXWDjNdgYmM8F   100%   50095833 (xxx)  50095802 (xxx)             0   1.6.4  0.99771712 SOL (0.00%)
⚠️ 8KeYrkYrNSq1x5zDQqghEiXFmf4QjYUWUzVt6qwiBX5i  775UhDftsU4SMEUHK8nCK1P5o4eGnoGwX6FdhBbA6LtC  100%   44829320 (xxx)  44829289 (xxx)             0 unknown  0.99771712 SOL (0.00%)
⚠️ 2MeGu4yCwRCthwzFiVtsLAVSa6xCsdH59cz6F6J6b33U  GrnK4vPzYLzFdnirZCBXjLSvKG4iF6pHdZ67C9rZsbSG  100%   36055114 (xxx)  36055083 (xxx)             0 unknown  0.99771712 SOL (0.00%)
⚠️ 6xDdF6YUAKVfj6AL3TMJckQokXhrxtBwMiuHVMnSHxR   GamfjAaa1y2vNbR54vfQM3Q5NrrQ5xCDtdoVC93jWiJ7  100%   35371461 (xxx)  35371430 (xxx)             0 unknown  0.99771712 SOL (0.00%)
⚠️ AJib74LMUmCfkRN84iEWApuhGzEpQcA4g7KC1FVgqWCt  72h1eypUDXm6ks6X8uWbpuzKgRNZiCRSwnefFHjGyxEa  100%   46034764 (xxx)  46034733 (xxx)             0 unknown  0.99771712 SOL (0.00%)
⚠️ 7QqYu69Sh5WB58JjXEmnuKVfvybC4dxWTsDFmNqiY9d4  2Y6BbUbkcEKdnAr9ED5bLBH6nC4XnfBUnjrVcRghozcw  100%   45329091 (xxx)  45329060 (xxx)             0 unknown  0.99843424 SOL (0.00%)
⚠️ 4mMCNHeEZwqkM99HAzQMbiPFCXhsC3VAo56BkFoTVHoY  7GHnTRB8Rz14qZQhDXf8ox1Kfu7mPcPLpKaBJJirmYj2    8%   48268420 (xxx)  48268389 (xxx)             0 unknown  1.001521874 SOL (0.00%)
⚠️ BmSmUWB9gJkoePfnmnKGtcG9MXnW1VKDvvTRman8F1Te  BGMDEqc5TZ3xkSKWeGtiAHPNkbc4GeBqnQdNJry2xGth    0%   51517188 (xxx)  51517157 (xxx)        109190   1.6.6  1.284048473 SOL (0.00%)
⚠️ 5cntEEYR5DMoDucsjQKCjHyhR7CDbytAeW1Y8osHAd3R  CrinLuHjVGDDcQfrEoCmM4k31Ni9sMoTCEEvNSUSh7Jg  100%   41470176 (xxx)  41470145 (xxx)             0 unknown  1.29315136 SOL (0.00%)
⚠️ G7131eBQAa4NhX8V3JXRZwctZRraLMdUPcAAH1M5xpDo  7Lm4wRjaH7CvrzaD4oRrXFu6FFdCreYupFQHouh8Uojy  100%   35126559 (xxx)  35126528 (xxx)             0 unknown  1.88630272 SOL (0.00%)
⚠️ 82BPXtNdKBhiEEFdB9Vm6dJpj6jGuoUxEKHxZWiBcgyX  BbLFfmDiViMAXZZf1eLHRtQAuFS1aG8PJ2AmE5NLm1mY   10%   49514508 (xxx)  49514477 (xxx)             0 unknown  2.035346098 SOL (0.00%)
⚠️ E4aMcPhHEoyLxMW8jJgMkcyK1S6qNhjjZGsyhCY4UiAd  GemaKPJ7cWmVsqbb5T5z5MyDYLreRaUqv7MswYbmqgSV  100%   40239267 (xxx)  40239236 (xxx)             0 unknown  29.068831918 SOL (0.00%)
⚠️ H9kSDiJJFVoLjpmue8nKhvLteZfzjuzS8AjUtBovxBZE  GNkBiAHKTk5RYxi1bSEKc31xibMqSBTKtLkRbXhi1h8K  100%   32427525 (xxx)  32427494 (xxx)             0 unknown  82.99543424 SOL (0.00%)
⚠️ DctYdX8c3qBZ7RUtYE4Ffunjv5SYFxVde4H4BDejPzMG  HK456zW1akyEDBdmvcBddyvxfefGCvTBszKBakAZPR3S  100%   12223953 (xxx)  12223922 (xxx)             0 unknown  95.48236992 SOL (0.00%)
⚠️ EUcJwf7jXskRE6NZBtFPVH2EedNvNYko8LL2WT62XctB  Bwkz1ddKoGE8hgiSV6HZLXi9RBLqfBi3HZb2QujzVGgz  100%   39257131 (xxx)  39257100 (xxx)             0 unknown  115.99315136 SOL (0.00%)
⚠️ FSgwdAhy9JAePSKtrCrDFmMFpKwufd1WVBimHRc2dDEk  85ShayF5HDXzZxxey8Qd89VSCfBxCgtmuL4LJx6GTkr2    9%   44780479 (xxx)  44780448 (xxx)             0 unknown  149.24315136 SOL (0.00%)
⚠️ 815YLNgAsnMqWJo1dmy13oKwWxujq31Dh6KhCUirnhBh  DTmhJTdXM8QBQ5p4mL2v6XzVqin5t9RdtFQMkXctu2La   10%   38045501 (xxx)  38045470 (xxx)             0 unknown  1013.495481318 SOL (0.00%)
  dv1LfzJvDF7S1fBKpFgKoKXK5yoSosmkAdfbxBo1GqJ   5MMCR4NbTZqjthjLGywmeT66iwE9J9f7kjtxzJjwfUx2    0%   51688184 (  0)  51688153 (  0)        280186   1.6.6  77917479.472601518 SOL (100.00%)
```

Reply:
I rather like that, if it works for you

Reply:
Sorry for the ambiguity. You posted 2 things, but github only showed me one... I meant, I like empty space for the delinquents and ( 0) for the highest nodes

Reply:
wfm, so this right? :)
```
  Identity                                      Vote Account                            Commission  Last Vote        Root Slot      Epoch Credits Version  Active Stake
  wWf94sVnaXHzBYrePsRUyesq6ofndocfBH6EmzdgKMS   AKoVXpZmi8wSz3sGvCYEygbpdHvSRysWsh36b97iPvKh  100%   51691953 (-11)  51691922 (-11)        283958  1.5.18  -
⚠️ 4GskhQsHVuMv68guHRwr5f5DgL9vHEMcV1KfNcvS7VhG  Mx7uRGuYtxknJKuXQz3mv5EkiydRXeDXWDjNdgYmM8F   100%   50095833        50095802                   0   1.6.4  0.99771712 SOL (0.00%)
⚠️ 8KeYrkYrNSq1x5zDQqghEiXFmf4QjYUWUzVt6qwiBX5i  775UhDftsU4SMEUHK8nCK1P5o4eGnoGwX6FdhBbA6LtC  100%   44829320        44829289                   0 unknown  0.99771712 SOL (0.00%)
⚠️ 2MeGu4yCwRCthwzFiVtsLAVSa6xCsdH59cz6F6J6b33U  GrnK4vPzYLzFdnirZCBXjLSvKG4iF6pHdZ67C9rZsbSG  100%   36055114        36055083                   0 unknown  0.99771712 SOL (0.00%)
⚠️ 6xDdF6YUAKVfj6AL3TMJckQokXhrxtBwMiuHVMnSHxR   GamfjAaa1y2vNbR54vfQM3Q5NrrQ5xCDtdoVC93jWiJ7  100%   35371461        35371430                   0 unknown  0.99771712 SOL (0.00%)
⚠️ AJib74LMUmCfkRN84iEWApuhGzEpQcA4g7KC1FVgqWCt  72h1eypUDXm6ks6X8uWbpuzKgRNZiCRSwnefFHjGyxEa  100%   46034764        46034733                   0 unknown  0.99771712 SOL (0.00%)
⚠️ 7QqYu69Sh5WB58JjXEmnuKVfvybC4dxWTsDFmNqiY9d4  2Y6BbUbkcEKdnAr9ED5bLBH6nC4XnfBUnjrVcRghozcw  100%   45329091        45329060                   0 unknown  0.99843424 SOL (0.00%)
⚠️ 4mMCNHeEZwqkM99HAzQMbiPFCXhsC3VAo56BkFoTVHoY  7GHnTRB8Rz14qZQhDXf8ox1Kfu7mPcPLpKaBJJirmYj2    8%   48268420        48268389                   0 unknown  1.001521874 SOL (0.00%)
⚠️ BmSmUWB9gJkoePfnmnKGtcG9MXnW1VKDvvTRman8F1Te  BGMDEqc5TZ3xkSKWeGtiAHPNkbc4GeBqnQdNJry2xGth    0%   51517188        51517157              109190   1.6.6  1.284048473 SOL (0.00%)
⚠️ 5cntEEYR5DMoDucsjQKCjHyhR7CDbytAeW1Y8osHAd3R  CrinLuHjVGDDcQfrEoCmM4k31Ni9sMoTCEEvNSUSh7Jg  100%   41470176        41470145                   0 unknown  1.29315136 SOL (0.00%)
⚠️ G7131eBQAa4NhX8V3JXRZwctZRraLMdUPcAAH1M5xpDo  7Lm4wRjaH7CvrzaD4oRrXFu6FFdCreYupFQHouh8Uojy  100%   35126559        35126528                   0 unknown  1.88630272 SOL (0.00%)
⚠️ 82BPXtNdKBhiEEFdB9Vm6dJpj6jGuoUxEKHxZWiBcgyX  BbLFfmDiViMAXZZf1eLHRtQAuFS1aG8PJ2AmE5NLm1mY   10%   49514508        49514477                   0 unknown  2.035346098 SOL (0.00%)
⚠️ E4aMcPhHEoyLxMW8jJgMkcyK1S6qNhjjZGsyhCY4UiAd  GemaKPJ7cWmVsqbb5T5z5MyDYLreRaUqv7MswYbmqgSV  100%   40239267        40239236                   0 unknown  29.068831918 SOL (0.00%)
⚠️ H9kSDiJJFVoLjpmue8nKhvLteZfzjuzS8AjUtBovxBZE  GNkBiAHKTk5RYxi1bSEKc31xibMqSBTKtLkRbXhi1h8K  100%   32427525        32427494                   0 unknown  82.99543424 SOL (0.00%)
⚠️ DctYdX8c3qBZ7RUtYE4Ffunjv5SYFxVde4H4BDejPzMG  HK456zW1akyEDBdmvcBddyvxfefGCvTBszKBakAZPR3S  100%   12223953        12223922                   0 unknown  95.48236992 SOL (0.00%)
⚠️ EUcJwf7jXskRE6NZBtFPVH2EedNvNYko8LL2WT62XctB  Bwkz1ddKoGE8hgiSV6HZLXi9RBLqfBi3HZb2QujzVGgz  100%   39257131        39257100                   0 unknown  115.99315136 SOL (0.00%)
⚠️ FSgwdAhy9JAePSKtrCrDFmMFpKwufd1WVBimHRc2dDEk  85ShayF5HDXzZxxey8Qd89VSCfBxCgtmuL4LJx6GTkr2    9%   44780479        44780448                   0 unknown  149.24315136 SOL (0.00%)
⚠️ 815YLNgAsnMqWJo1dmy13oKwWxujq31Dh6KhCUirnhBh  DTmhJTdXM8QBQ5p4mL2v6XzVqin5t9RdtFQMkXctu2La   10%   38045501        38045470                   0 unknown  1013.495481318 SOL (0.00%)
  dv1LfzJvDF7S1fBKpFgKoKXK5yoSosmkAdfbxBo1GqJ   5MMCR4NbTZqjthjLGywmeT66iwE9J9f7kjtxzJjwfUx2    0%   51691964 (  0)  51691933 (  0)        283966   1.6.6  77917479.472601518 SOL (100.00%)
```

Reply:
mb looks like this:
```
  Identity                                      Vote Account                            Commission  Last Vote        Root Slot      Epoch Credits Version  Active Stake
  29rzUXiy2kYridD6zxc6nszsQpgVW8bknW9NMQEiQThi  6FzXPEhCJoBx7Zw3SN9qhekHemd6E2b8kVguitmVAngW   10%   75066990 ( -1)  75066953 ( -7)        297263  1.5.19  -
  DE1bawNcRJB9rVm3buyMVfr8mBEoyyu73NBovf2oXJsJ  8XgHUtBRY6qePVYERxosyX3MUq8NQkjtmFDSzQ2WpHTJ  100%   75066990 ( -1)  75066952 ( -8)        298862  1.5.19  -
  CvNorfaayk8vyFVpSQwQFSffPf3AHwzrNijBRLqawZiL  8L8adJFEwA4PdyWg8xXvGWYuz6s5pR7UFqtgXJVNS6vS  100%   75066991 (  0)  75066960 (  0)        206972  1.5.19  -
  7X1kgrfYRwFd94yyE8tvsQF5aTFHdmoLXnG4q79onDAg  5TkFPZf79rezPXzCTP6DjH4NLb7Hw3jRsAme1fv431oz   10%   75066991 (  0)  75066960 (  0)        297557  1.5.19  0.007865992 SOL (0.00%)
⚠️ GXYeb8vmmvvPxPvUPX6RQ139W3kv7AX9vqr39PzpVTus  6PnJSGnDvxemLEqmqWMqzfgNw1wbp37vNKextbdb59e9  100%    8072826         8072795                   0 unknown  0.01543424 SOL (0.00%)
⚠️ 8dXuaz4CSz2pizYfspXrmUGJHZKnfaQwAHMgkt3CZNDd  6Hr77bmtmxbs3eo5NTiXDqevUTVhumNHuiTYHEAyYyvJ  100%    7393633         7393602                   0 unknown  0.19543424 SOL (0.00%)
⚠️ 3XeDMpzQc5SmrSE6w9nYBXkdk8AzX7q5XohzxmwMmExG  5ayEGbt2bg5JTZVXtYYpivG2gZQQCfoJUeLzMiSBghqV   10%   65358818        65358765                   0 unknown  0.49771712 SOL (0.00%)
  betaVcnkBhHKaWx9o6LoSYrGaoDCskQLm94cUVWqDLS   9o7bxey9Y2juu9K32SscPM5YLRg1CzMywcEECDPbzA5y  100%   75066990 ( -1)  75066953 ( -7)        283455   1.7.0  0.99771712 SOL (0.00%)
  FpV6bAeWUhS1A3tcLpzLgRBniFd62UH8ZqJH61vd5H3G  4SGPMZvQZds62vJrzwSyz8643knhXA6jd7H8NddEMhEt  100%   75066989 ( -2)  75066952 ( -8)        295581   1.6.6  0.99771712 SOL (0.00%)
⚠️ 7tSdoA1mKxy5oeNP8tcD67s6qD5v9a22X29uwh27TKn9  MQCgnXw5AFHiXn88N3MhZ64iWXPm1tPaMiQBuNhLbMF     8%   72869208        72869154                   0 unknown  0.99771712 SOL (0.00%)
⚠️ GHs16J7Ny7zNPmh6tH99JEwR2JQYnaAkwn8A2UNuDWce  CnRdUzkzGCMW5bsr7dPVq4BqhQnqbb6khjQRvoABPDDh  100%   73266147        73266116                   0 unknown  0.99771712 SOL (0.00%)
⚠️ Dgwcrprgu1WeCNEVG2stkiABqWjfSFfBuXeMrR7qwJHp  HhzsBi7Y6DPfcZtYAUcbhNzStpicPar8Psy84yS8ZeVg   10%   56233163        56233110                   0 unknown  0.99771712 SOL (0.00%)
⚠️ FMcxJrnU61J7s74PhmNdbyJjDJ4b6npACF6Zs2bKeFWX  ED52ZoxYTVz2PEV687evqwDg4ozPVzyPtEb298o1Ya5b  100%   48904113        48904051                   0 unknown  0.99771712 SOL (0.00%)
⚠️ DSCQgVWCADBTwARKspCHMazz7LQ9GShjQSG3mCWZ72mH  EYAsM8PxP9wmkjN8MBDV4tPdWKWXm1AAS6ZxTWH5g9wm  100%   53903619        53903582                   0 unknown  0.99771712 SOL (0.00%)
  4mJhW6Un4CHbTicj39Yuu9GQt5VAxKFzKGHFcSftVnVq  2giMniq7L6WLUaZwDsfP2fmTbTnsJgVDyBDZJ5vnmtE3    5%   75066990 ( -1)  75066953 ( -7)        214933  1.5.19  1.001018068 SOL (0.00%)
  E9hD3ikumJx1GVswDjnpCt6Uu4WG5mz1PDWCqdE5uhmo  3a2onvgTpGynakAQwx6gigtSeL7itZewNxqb5JiAvWeA   10%   75066990 ( -1)  75066953 ( -7)        294431  1.5.16  1.016933941 SOL (0.00%)
  Ecs6YWKzgZXu3oS4WCyWzWf4kw2g7XnE7eWwq83TQBaG  GW26xx7728na3saWrqbuwtzn7mfNYv18rhUETzod5UsZ   10%   75066990 ( -1)  75066952 ( -8)        298396  1.5.19  1.107807836 SOL (0.00%)
  nVmiYamBpwzEqxykaGBWvY9W4R7rmK1JudonPRhmkAw   7k2ysYjSheYCamBximJsfCHSivXNyGobD3gBrVMK423G   10%   75066990 ( -1)  75066953 ( -7)        298607  1.5.19  1.51537003 SOL (0.00%)
⚠️ 4SqdkosjugZVRdX2kRptUng487Uece5toWHZXVh6cpQV  2f5hEM1RfEb6Sx4hiuFzcrGrGSyDSmMVZoBEjnrkETMo   10%   61708683        61708629                   0 unknown  1.99543424 SOL (0.00%)
  GdnSyH3YtwcxFvQrVVJMm1JhTS4QVX7MFsX56uJLUfiZ  sCtiJieP8B3SwYnXemiLpRFRR8KJLMtsMVN25fAFWjW   100%   75066990 ( -1)  75066952 ( -8)        297420  1.5.19  1.99771712 SOL (0.00%)
  7Np41oeYqPefeNQEHSv1UDhYrehxin3NStELsSKCT4K2  4785anyR2rYSas6cQGHtykgzwYEtChvFYhcEgdDw3gGL  100%   75066990 ( -1)  75066953 ( -7)        296760   1.6.6  1.99771712 SOL (0.00%)
⚠️ 5qsT9h2TuTMPLtX6gcD2DG6mcZwechxmWhKRFHbCt6Pu  21ryEourynXqhpLe1DsFz8yoeFKSXE14T8bKBFmzcYzt  100%    7393690         7393659                   0 unknown  2.99315136 SOL (0.00%)
  CakcnaRDHka2gXyfbEd2d3xsvkJkqsLw2akB3zsN1D2S  9bRDrYShoQ77MZKYTMoAsoCkU7dAR24mxYCBjXLpfEJx  100%   75066990 ( -1)  75066950 (-10)        298535  1.5.19  3.50471712 SOL (0.00%)
  AiXsskf38SEn4WxsJJA5ZLG24nAqjTzzcpjZM6FyT7us  DAxtXArGvM2JSL5goGTdxQE86CVzoCGFHNixwfFVzoZ9    8%   75066990 ( -1)  75066953 ( -7)        142543  1.5.20  5.99771712 SOL (0.00%)
  WUNoB9YQXmXXRcJsjY1G8PfVag5aAfnyGmFd6YwJVwp   BDn3HiXMTym7ZQofWFxDb7ZGQX6GomQzJYKfytTAqd5g   10%   75066990 ( -1)  75066950 (-10)        297642  1.5.19  16.996943846 SOL (0.00%)
⚠️ HoMBSLMokd6BUVDT4iGw21Tnxvp2G49MApewzGJr4rfe  9tJaXs1yoMHyPCtZERaPYACP4zGxsEyK4K5ErL88ZmNR   10%   55689128        55689095                   0 unknown  33.99543424 SOL (0.00%)
⚠️ 8JJAFng66sS7Lo7fX5Mi3GoBhE4k9Hjz1SWMLK18Er5h  CrpQ1ahbSMbgRsBkNs1REYShpr6ApiWcKmsmNkRTkTwd    3%   65231999        65231883                   0 unknown  56.40851681 SOL (0.00%)
⚠️ BozqcycsdJWcrNSiQQNKrx2Mx8ytCbDwPmSUH2xKnQxS  22Bw3bWQF4AExTH2H4X9CsdDpxkfWd5KqwQTJ2YQNdMM    4%   70102486        70102455                   0 unknown  95.260602944 SOL (0.00%)
  BU6V24cSbncwnJJfxnHoiw7wifPpVaKvEWznHkxB2o83  7oX5QSP9yBjT1F1sRSDCX91ZxibETqemDM4WLDju5rTM   10%   75066990 ( -1)  75066953 ( -7)        298275  1.5.18  96.516520727 SOL (0.00%)
⚠️ A1z2NeWV5WmXxV9FXWJ2DoFSTu4Ry4E7kUBYrFn2UjeJ  9BdWsxVorxe8eUrAmW6sec7YSCDtDASuqumKfPEkeGS7   10%   66352194        66352105                   0   1.5.8  100.96687411 SOL (0.00%)
⚠️ o1nodeMCRLoSgCL1LTgTTJU56AhYuBNC9Y2ttafDL3n   E27ovWovAA7UfwqezdPry9Dav5cSYh9Bpzz7s3JSAkJb    3%   71430273        71430242                   0 unknown  109.377557148 SOL (0.00%)
  87fX6AAJywaQfgMpD9Gkwxpt1e129kKZxBdRG1SQA43m  4GAmUQ8FvKcTzeYGqxu2oSBMStYNwDTmBo7LC1Csg6SE    8%   75066990 ( -1)  75066953 ( -7)        298750   1.6.4  145.353419295 SOL (0.00%)
  722RdWmHC5TGXBjTejzNjbc8xEiduVDLqZvoUGz6Xzbp  EXhYxF25PJEHb3v5G1HY8Jn8Jm7bRjJtaxEghGrUuhQw   10%   75066990 ( -1)  75066953 ( -7)        296132  1.5.19  958.183056894 SOL (0.00%)
⚠️ ArmanP3SBD1DVSoKCqK7d6S2kLpyncYey2Ybf3o5XkTn  3byuwfBCZ4JLdbsuirv6yV3opAfgv2kw3BHjqjGUSDCd    0%   73984035        73984003                   0 unknown  1071.385054698 SOL (0.00%)
  9bkyxgYxRrysC1ijd6iByp9idn112CnYTw243fdH2Uvr  6W8yrMwtDU5G6ErazhZHfLjqZV8cMvajpSRGYgrZ3d4v   10%   75066988 ( -3)  75066950 (-10)        299051  1.5.19  1109.99315136 SOL (0.00%)
  5YA8huSZrypuymBXwtoLCo3sEQD7a3rC7otRiCDAw1HA  DgDuQTSGdtVyALBEYDi8HN6QiRBF3Uvvkc3zgEgKidZi   10%   75066985 ( -6)  75066948 (-12)        298784  1.5.16  2547.244580564 SOL (0.00%)
⚠️ EUcJwf7jXskRE6NZBtFPVH2EedNvNYko8LL2WT62XctB  Bwkz1ddKoGE8hgiSV6HZLXi9RBLqfBi3HZb2QujzVGgz    1%   74735999        74735961                   5   1.5.8  2597.460484514 SOL (0.00%)
⚠️ DctYdX8c3qBZ7RUtYE4Ffunjv5SYFxVde4H4BDejPzMG  AXCmFcJrDKnigrAf6KYKZDzuaBoWbw8K6v4ZfBvDGgc     8%   66673612        66673264                   0 unknown  2641.140986259 SOL (0.00%)
  gospeLah7AeJAanrMH6EPhitTgusELN6jFZLzVHVwes   CAREq5ENgkcmKN2B89NEtK3DAb8H5zVTHhnXHphYGcFm    0%   75066987 ( -4)  75066950 (-10)        296182  1.5.18  3945.754988473 SOL (0.00%)
  8yjHdsCgx3bp2zEwGiWSMgwpFaCSzfYAHT1vk7KJBqhN  DPmsofVJ1UMRZADgwYAHotJnazMwohHzRHSoomL6Qcao    1%   75066991 (  0)  75066960 (  0)        295211  1.5.19  4116.371556386 SOL (0.00%)
⚠️ NoFeeZ3uaGtB92N62To5VbyzcD7gXmrqSvhEHcXhH3U   NoFeeZ6ddr4377ss9edzF5dscUHzaWuergj5pPwSn4H     0%   63017621        63017590                   0 unknown  4138.953882331 SOL (0.00%)
  3P5hcynUJe7u4icW8pH4ZEAh646TwGnVZWh3YR3yyjsg  56FHzcWPCXzuuxqa8CZGVVWQy4jLMFsErVed4XBv9SKG  100%   75066990 ( -1)  75066953 ( -7)        297533  1.5.19  4999.99543424 SOL (0.00%)
  7vRwVrQcv6um36mDrDQp4538oEjVPuC1SabFpLiN29br  6KirXYHeV7Pa7ZCy16TqckuJJBBwfp2NDKxv5EPbr8eh  100%   75066989 ( -2)  75066952 ( -8)        298166  1.5.19  5076.108721186 SOL (0.00%)
  2RwFJJkhr4tCTuQnZN1CFpAtn5s9pTzakagvu326W1FT  HQbdXrLm3EuFyu66mCpDS4ir9zRkhNHJVnck4vxG7jEo  100%   75066991 (  0)  75066960 (  0)        298641  1.5.19  7292.54543424 SOL (0.00%)
  BjuD62v9RysrburpKb65UKeaAWRSFyi7pFLLxdE3dPv   GHRvDXj9BfACkJ9CoLWbpi2UkMVti9DwXJGsaFT9XDcD   10%   75066988 ( -3)  75066951 ( -9)        295111  1.5.19  7513.274675849 SOL (0.00%)
  31h8ATCPaCiFRdoE6pQ4ic5WPVQMSPxaP8XKQ7XNgJyS  Hk7p1mRP65ve59cguy4XFmh5H1jxSNhCWsPeDxH4qM2q  100%   75066990 ( -1)  75066953 ( -7)        296353  1.5.18  11176.584529057 SOL (0.00%)
  4vXWg67cibovpHchtpqbrjEPtTr5HcmkWTtmcHRzNzGW  21dmSgUzTaW8bNtRLtTTEmwJTBSTzqBrtoDzqevK4JAs  100%   75066989 ( -2)  75066951 ( -9)        290048  1.5.18  15249.649289392 SOL (0.00%)
  e1afFaogyDznnUBMWQbPgnZ3VUzkGVsEhyxBPH13BVx   e2aobeynMiTfknRgrvsBXsPX2uY3GVVnEDmj5fM5ADX    99%   75066989 ( -2)  75066952 ( -8)        298151  1.5.19  15544.646004913 SOL (0.00%)
  7dGY7orXGuD4A1482yFJ6ZC4u3kS3s6aMBN2d5XYiGxM  CF7sbJDXrx25ak8Mz7JBXu98qhnUjJ1RoCQe1UsLo3B6  100%   75066991 (  0)  75066960 (  0)        297920  1.5.18  18283.874995054 SOL (0.01%)
⚠️ 9Ukj3PkyD3igEDJGt1QTj9ThzjK6hMiadQfa3gm7kjf1  37BPVW1Ne1XHrzK15xguAS2BTdobVfThDzTE2mv8SsnJ   10%   74742943        74742911                5055  1.5.19  25199.90259307 SOL (0.01%)
  AyYqAhyCCxRrNGbm3dY4aGY9SyaQC9UvPTSHvMQK4YW2  GJPXfCUjDy9PjqQ2Cvh7pSEvPu8vRjMXz2kSULBBaSLt   10%   75066991 (  0)  75066960 (  0)        295372  1.5.18  26506.222572195 SOL (0.01%)
  J78SNwDW6G86sMmh7djnBKGjewXNpjD74sJTjJ1iNgTH  AT1kLQm1uYB5XZVocNUn9QkH3NPJRjRvf5orGS469xuy   10%   75066989 ( -2)  75066952 ( -8)        280687  1.5.19  26826.300019138 SOL (0.01%)
  Cq2vTaqXwKWA7H2nkCA45XRPWSHjJf5B4Pfic1QCVwEr  Cr9oTBpsBtXJNDuUBMzPyS69zaX7u9VxNZTY5EEeTHrq  100%   75066990 ( -1)  75066953 ( -7)        294681  1.5.19  36025.553134806 SOL (0.01%)
  F3LudCbGqu4DMqjduLq5WE2g3USYcjmVK3WR8KeNBhWz  FSj7gzzNHhALp1qVP7BLqbakcLSMNosEtykLn69vHXyk    4%   75066991 (  0)  75066960 (  0)        295684  1.5.19  36092.301738683 SOL (0.01%)
  F2RceM6AEmefYAaCg7qLsUT7yhwZJi9BqJJ1NsaN6hQ   HHEq7d3vyNXfh4gMgze8ikZabWeNpjbHFVrKvKpJJcAa  100%   75066989 ( -2)  75066952 ( -8)        297811  1.5.19  82515.069385877 SOL (0.02%)
  FrN82kxBGpZEE1ZtWbaAgsry5UzekSTMRZrMz4rLZc6i  2zphXGWQAjNnbuQUgM9jtiH7CoHYPaXNxQnFGEKzbSsw  100%   75066991 (  0)  75066960 (  0)        298655  1.5.19  83488.120482492 SOL (0.03%)
  7r5cfHQBg6Ly8kR3WfuA21vJwKrXfTMzSDU71UN95c8S  9EeymqCPxrNQzUK8qfcMAMonytnQmK9bfEYoy7FyqpQ6  100%   75066991 (  0)  75066960 (  0)        298245  1.5.19  154667.913405827 SOL (0.05%)
  HUyaWfcJ452fKxNRQaJkVgBu5Qpm7SZ1EU9mYgwq7PmG  4xpaDh3Wq8tq8ztgB2w5gkG7W6bZaF89VV76KDMQQwx5  100%   75066989 ( -2)  75066952 ( -8)        299014  1.5.19  167489.754864476 SOL (0.05%)
  G1X8Jrrfz3gfqZQo29jjwZ844XwjdPPEBaqNDWoAfEA   5PZKxexLUwr63PAvt6cbShE7owVmoryaNe2jwgWzHxvJ   10%   75066991 (  0)  75066960 (  0)        297675  1.5.19  200781.932300677 SOL (0.06%)
  8ne4sDgaaMtuwY1y4BaMYSGKXpDRVc1Po9PsJQvrZPMs  CjxgHgyBPkUo6GWoKn6RWEprCVRqhKxDsyKKYV4GMBz    10%   75066990 ( -1)  75066952 ( -8)        296637  1.5.19  201545.266371093 SOL (0.06%)
  GejR6mt9gxHtJ8Xe3cqamG7AL3Hx74w226o8p61QA5oG  GPHs6QbV6Jbd9FDuaxsLNz5q1qHQKgHm1whtKDhKjdWp   10%   75066990 ( -1)  75066953 ( -7)        298791  1.5.19  201566.993478155 SOL (0.06%)
  B5GABybkqGaxxFE6bcN6TEejDF2tuz6yREciLhvvKyMp  9mPYMu9MUQuykKA9QK1jSVymJJTTRWKMyGsdM1VJFoL7   10%   75066990 ( -1)  75066953 ( -7)        296080  1.5.19  202028.632707943 SOL (0.06%)
  Dfc73Czi5K3xa6yKMq98LHJx69PDsSzUvSdCSSqr8NDQ  2xT5m25m9mDkdpLV1mT5fWkvH15jGy1dm7anNQLzzYGA   10%   75066990 ( -1)  75066953 ( -7)        298632  1.5.19  202034.300071911 SOL (0.06%)
  f8e1k1Qz7zsv9gp1kVPnzBHCV8kDkfM8Rad3ZfHxFZN   2UYjYyRqKrZdUVEZdrb8xneKtbLZuMqFYJdnhwdr7Vc9   10%   75066990 ( -1)  75066953 ( -7)        298203  1.5.19  202037.420279794 SOL (0.06%)
  6pTM5pLcQFEzCnd2GxLqLm4jL97ZL8L1ThSpR5aVRfc2  7rhm6RffTmi6WLrMjviwEYebm3D9ugUNBvj9t4iUzZTW   10%   75066990 ( -1)  75066953 ( -7)        296217  1.5.19  202037.449048196 SOL (0.06%)
  8sv24V7cy7tyFPwn8nqAKXiigeNmgYACoFmG2z2XBTZw  83b3MHxfxGLYkhfjT3nFtPRmYL3eKk5EMALsVAHAmgqK   10%   75066990 ( -1)  75066953 ( -7)        296141  1.5.19  202046.417068239 SOL (0.06%)
  AEx4pxCHFpsVgZ2X9t38bt4CouZ85tXXcghjUkb6ypv7  FePX54LqimFFoShNuhTSUv6bV4L3rMZq8CX9BeH6SqYf   10%   75066990 ( -1)  75066953 ( -7)        298325  1.5.19  202057.618194273 SOL (0.06%)
  9QsFvmFWH2weMKYivw31wy6qCztnSEAgbvgAA8gMoUWH  E971NTnWRWFTTso1MA7FaYbLCgMuViKsANZTFSk1Dmz3   10%   75066990 ( -1)  75066950 (-10)        298697  1.5.19  202059.800143635 SOL (0.06%)
  GLmkRrNqTSVi9ykTBQVX9M8cV9EwiCABknXVweS4QVxj  6syLbA7pHFWGJetPdB5Hyzf33r7B6zZK7qurwCv4Afqk   10%   75066990 ( -1)  75066953 ( -7)        298706  1.5.19  202068.387000748 SOL (0.06%)
  BvXzmS5rLW89nHD7K2qVjcERKigDdJ8Xhc74MQZumj9J  GHGpf3peyQUbWadARpdXR4oNugufNDtvT9pPyBEZJa6h   10%   75066991 (  0)  75066960 (  0)        297473  1.5.19  202240.416064984 SOL (0.06%)
  H5FD5GxuE46hqi5paMkcqt9WeHvszZA7RgBfMoMpfNDK  FK466vE8ZPSuntKFT415H16qRJ26UPzmXZXb3jq42jaE   10%   75066988 ( -3)  75066951 ( -9)        296440  1.5.19  202343.111893489 SOL (0.06%)
⚠️ 9XV114Dg7cQF5zbxEoEnTpuYXR9TgdjHSimZtkzh4epF  DeJtHx8YijwzNRDuEDyJBgbnXiHdbKZzMGtHXUF73y8i   10%   74884128        74884097              132658  1.5.19  202472.711931132 SOL (0.06%)
  8F75kvPrr9SVFDDbyNJQ9J8sxpEXDNh481McKJ1YzQB2  BCAJXBV1jCJA1KPfn4u8f19qmoVWJiRStudbxVi5rW5b   10%   75066991 (  0)  75066960 (  0)        298830  1.5.19  202500.667789282 SOL (0.06%)
  FPgkPgT5agEv7JY7czn1t5bKKewAkcBiPn1xQyiCcurc  D7fnHDRyonVgZFoDd5ESmgwPxG8iZNopWcvYaDm991jx   10%   75066990 ( -1)  75066950 (-10)        298993  1.5.19  202502.230754886 SOL (0.06%)
  EEN4pf92jyVoASZ6pQQMHcKXTF4d5T3cY1a942QhRasc  DCkFCCEQCuRTcJ3y4rTnWyXnQsi5abfCFXPZZ7pjPJqG   10%   75066990 ( -1)  75066953 ( -7)        298154  1.5.18  202503.98870885 SOL (0.06%)
  8c95uL2WKofQGfr3rAMZHmJ4V1UJW5puTmpw96n47MX   6tWV1iEX8vNzboPfm6UBuCGmT3p4btaaxidVYc3wRkki   10%   75066989 ( -2)  75066952 ( -8)        298425  1.5.19  202522.605296037 SOL (0.06%)
  9WuDnZMufE2nsuWsTnHjJsjavuKtswTpCy3i4A5kKT8c  3hHSwKHgnrbVDBWfkx6TJY1WABUoRs7Ve57sRTg9rAY9   10%   75066990 ( -1)  75066953 ( -7)        298616  1.5.19  202601.632528755 SOL (0.06%)
  CgwKP4kM6B9giquCADDXQxik7BJR5L7m2EC2ntPnQoJy  7A9NSA1SZrszosi37WHSYARjAc3JuXxVqg3q94xjtpM3   10%   75066990 ( -1)  75066953 ( -7)        297523  1.5.19  202612.15268935 SOL (0.06%)
  FUyx2W6wDt7u363QgQRWQYuytE5uJWZLGJpuVh3RDiCa  AZUYzQX7nrbN1iE25YsSCSZAywZcyr1C1anMnwZtaqX9   10%   75066991 (  0)  75066960 (  0)        298205  1.5.19  202701.576230301 SOL (0.06%)
  9FXD1NXrK6xFU8i4gLAgjj2iMEWTqJhSuQN8tQuDfm2e  2g2QU1NDRax6i2mKzRwgRfdBFoDkMC6bj7Zp5Q3i8sCq   10%   75066990 ( -1)  75066953 ( -7)        296801  1.5.19  202710.496747961 SOL (0.06%)
  CfBJ2yK6hbRD3mV5VoMux6nkki3mYa3NCmWWnfTeKw4B  76SARfq9QGiq1QT7mzR23sqJNLZaGeFDCMmpK5uGxGf5   10%   75066991 (  0)  75066960 (  0)        298627  1.5.19  202734.675041191 SOL (0.06%)
  EJ38tGihX9LtQ3q2eYGxUvLm6pzx8w6rXG2Q8BNhbeiM  74aFgAgx59CsJ4meRSGDjiYbexRdM7N9yFEmVLYH9ASq   10%   75066986 ( -5)  75066949 (-11)        298256  1.5.19  202754.316545264 SOL (0.06%)
  EKyzkF3pdGDR3Wnhuf3ouMoofnJi3r2XKioUuGqwrqB1  GU5aJ7NPnwBTqreqdFj3Loi1TZBpDzgDhoX7SMNs5Wzq   10%   75066989 ( -2)  75066952 ( -8)        297667  1.5.19  202813.151485264 SOL (0.06%)
  4PRFPF7f9ERz9azkDFSFfgpye6yixPENCka994j8mQbj  Csp2hDVRX4cRitFEswJJcoH9ju4rcPUSsqU9pkh5JBQU   10%   75066991 (  0)  75066960 (  0)        298048  1.5.18  202815.979210764 SOL (0.06%)
  4daH8Aotxpk68HsMvws3P5AQL3F1gVTA44jqLaB2GuGx  4sFsBDei4m7UXwuMeAGVmnJxZe9peJ5tvq581HoT9aH9   10%   75066991 (  0)  75066960 (  0)        298870  1.5.19  202828.28015865 SOL (0.06%)
  4xp7K2vkm9LnJh1aCXjJyyF6XxL1u5gmYkrVCxHU6Cnw  9agefe58BLxF3dyDUg8JQLoGocWgTvZ6QEYeNrNTbi89   10%   75066990 ( -1)  75066953 ( -7)        296554  1.5.19  202885.207403281 SOL (0.06%)
  4ZToBgveZ5m8NySrDyPA2fiGVRVBioaoMXD31KGidm65  Dh4K8fNV6pRFZtbzQnP5a5HmyBPb2kmxvWiYmc5fJMvj   10%   75066990 ( -1)  75066953 ( -7)        298203  1.5.19  202889.233826627 SOL (0.06%)
  Cu4M3yd2LfMoGhmYxKszhVH18SPgt6TQvqnE4AWjNKwd  HmuQrPPXWkaAMhArBVvQRbknowKpsvDzNtSakvqt93st   10%   75066990 ( -1)  75066952 ( -8)        296205  1.5.19  202896.200010687 SOL (0.06%)
  2mKEUVkcttHeMeaZK8jjwkuwGSdCBm23xDgzXgbhTdPe  EoMcV8x78aGZYN3RSW8xk311Q8xKNHMXF94UJaEzsMoW   10%   75066991 (  0)  75066960 (  0)        296394  1.5.19  202908.206941707 SOL (0.06%)
  6UFmHMgP4ErpKrdjDgya1gAfHmcC6QhWK9w8wBT7iCNo  DmMwpjioQnfpKigoLADsxM7CeYqviQqSYyR4LnnhxCpu   10%   75066984 ( -7)  75066930 (-30)        297920  1.5.19  202910.266428214 SOL (0.06%)
  HtzxUabNfYNJR43FUmcpkgmtANZahbq5iASB5oiboXzF  DfmaZNQNUE9p5VQsDWjdnGkxDpcvLsiTTtrpwHZ4CoQe   10%   75066983 ( -8)  75066927 (-33)        298547  1.5.19  202910.854315547 SOL (0.06%)
  8rVubWZEYjy4YfMvUr5CDPg6YgdFdzUfZtfw84cxGjaZ  51dbk5MAe9XpNurR5nKYc7fr8ti9k4vB6LW2jVzN1zuf   10%   75066986 ( -5)  75066949 (-11)        295937  1.5.19  202914.824769419 SOL (0.06%)
  73A8PnqdHraWe9W5jiYzUBPEf4giyTxUG1CKZ6HkXbuz  5KSmoK86N7FyRkcj6ZqjY5Qxy8oZxovBCedR5onvQojh   10%   75066985 ( -6)  75066948 (-12)        295492  1.5.19  202916.616323958 SOL (0.06%)
  97MtLX5ajrR319PH8iLnctBpaLFoT3TNuUAtZfZaEn7U  cqpJdtrz63Z1ayC11aQg1FPas24r1RPqsaygyRSrCAz    10%   75066987 ( -4)  75066950 (-10)        298127  1.5.19  202918.982063788 SOL (0.06%)
  9qbqAdFqJK4MhmZPQjbRUxRUi2NTpevAiZ8wj7SiCiTQ  JyitvwAzKCjDUk6QyUiAVn92scoofctvXTWGA78T1ti    10%   75066991 (  0)  75066960 (  0)        297874  1.5.19  202920.403254417 SOL (0.06%)
  4Kbcyn7JVPAWLRLPsNGTPmcNMvCkLTw51ZLRhqsUC6jP  E5EEtMMswwfYPEtT1MgNE88op82E62TBnB6UeuQ2LRxc   10%   75066978 (-13)  75066923 (-37)        298063  1.5.19  202921.513520298 SOL (0.06%)
  2beSsAmYWPaNJ6GShkAECv617wmGfeH4sSuRFBYYX662  G6Vc9GXvFDby3bX71JCfs6FPQxmnkAjMzS2jUbKfWLAM   10%   75066988 ( -3)  75066951 ( -9)        298083  1.5.19  202921.780657925 SOL (0.06%)
  25V3EzQTGCSoLhihuH7f9jjyim1sxH6MR8MUtRLtLuoa  GLBi7kFdxSmTFiK5Y5rXVoHXscU8eoGUZhzDR7fa9ANq   10%   75066990 ( -1)  75066953 ( -7)        299209  1.5.19  202923.458093464 SOL (0.06%)
  A5dCbrjh492SU7fg31wZd6CNmz2NYqH1gEE219qyXZbe  B7rcR1Jg4GbRdFBzFhYfzRmb3GozVXwUD7QsYoS6KaSx   10%   75066981 (-10)  75066927 (-33)        298134  1.5.19  202923.525977574 SOL (0.06%)
  7WLzboF5Bu9e26MvLVps4rPbSHBgme4J1w2Hr9G4WW5C  CVhcNEoRowd91vGXpxq1f1wN44iH7jFwe57ux9jG5NKE   10%   75066990 ( -1)  75066950 (-10)        298733  1.5.19  202923.798150734 SOL (0.06%)
  DkycekX1rxCUr9FtBHfacHEgCTfRdRLaKYTiz7fXHMQ7  HYuiHu9Sn63zd2dg589wQ9M5krxrKs3fGUydTRRP8C8P   10%   75066985 ( -6)  75066948 (-12)        298211  1.5.19  202925.403551417 SOL (0.06%)
  48Dactjx3zuJujtBHZ5cd2w3maKFFGsKhn1sXzU59VGK  E3ho8z7xVRu1MTirWMoPrBL2J2AD7XJzsz5M4mUDqLBN   10%   75066989 ( -2)  75066952 ( -8)        298785  1.5.19  202934.336001664 SOL (0.06%)
  Cn2GFnp4H4mejHtR4GbV6ga48qoceZefzxNU68PxCcK8  8BXicETLFX5vbSjgyCBKbe5KRQH9nGTmWf5Y2R1k4C26   10%   75066989 ( -2)  75066952 ( -8)        296531  1.5.19  202940.957766944 SOL (0.06%)
  67iXZNZ4ytz3A23WueWr5B23WY7yHdESdRPGbVaPYkHw  AMZPPHg2Ht7PBMQuzwrk9FAx47xZVPwxSstC4GcuhfKo   10%   75066991 (  0)  75066960 (  0)        298223  1.5.19  202944.885872572 SOL (0.06%)
  8VAyNMaz9DFLNxSFrhjiusKCcgQNKgnByiG6KfeCP8vE  49SQQ2PiMPEcR2ZN5pkeGCnFKUQaNMmyZ45vEM4YzDP3   10%   75066989 ( -2)  75066952 ( -8)        299311  1.5.19  202945.43498824 SOL (0.06%)
  GosJ8GHbSUunTQPY5xEyjhY2Eg5a9qSuPhNC4Ctztr7y  DHXbmu3KUhHYn2Trned5ZUaGC6GRgHYhJoRbCr75XQxj   10%   75066990 ( -1)  75066953 ( -7)        295797  1.5.18  203003.013354917 SOL (0.06%)
  9jQ16ZjBN1vmXn3dpRDc5eZkRPU1kUTp5MPnJuSFWb1U  HMArFr5hNKhVY7bPjYc6WJEbDmJHyUvgiJZgrceqHNAL   10%   75066990 ( -1)  75066953 ( -7)        298048  1.5.19  203006.711170455 SOL (0.06%)
  72Sifwyudib2XE9nwDamapVEJr4rZLFthLtPH83ScupR  36usFSB9Xkr47ANKZ6EytFVFsdf9ngnZvNeeXEy7MBh1   10%   75066991 (  0)  75066960 (  0)        296674  1.5.19  203012.113219336 SOL (0.06%)
  GWccEztr4AFmiMvvvXcib4feDZXCi6U13XrxponUHFWC  eRBjr1X7drpEprHddEyP8CLt2BjTcfQK74nF4YoMDN8    10%   75066988 ( -3)  75066951 ( -9)        298444  1.5.19  203018.093037452 SOL (0.06%)
  BR9JfwDSvtP87kdJxeF5QJPCcj4bdmUNVDPtsK64DCkR  EY1ywqbD7W59PNLujMT6nkDLJyzVrtfLfi77hzzhLoHJ   10%   75066986 ( -5)  75066949 (-11)        297767  1.5.19  203023.37265607 SOL (0.06%)
  BsvtXMu1eGKrAhpP636EnNG8LWddxqmCDq8zcEG8CwY3  ytEKd8gXENNbci3pa3jvv1V8JAg8TxbrZ69eqJLEa5m    10%   75066990 ( -1)  75066952 ( -8)        298035  1.5.19  203059.279180985 SOL (0.06%)
  ALiZQ5XboU6qTGKSP7LExokaZcPGVYvmdFqqEw2bCJWp  DCYDbQbMB828QDBbmo8cquzSg6MtwhJ3LMq54enZwtKW   10%   75066990 ( -1)  75066953 ( -7)        298610  1.5.19  203083.371718681 SOL (0.06%)
  6mdtNmZpSycAZKXKy9JkkfdEZCMcnuyuA7fzdnpUohZr  9TUJdBxnHvAapYoq8cVFgh1bMbTh6GYfY4etDqWVKXAT   10%   75066991 (  0)  75066960 (  0)        296412  1.5.19  203086.825843685 SOL (0.06%)
  93P67u58qai4kT5XoKX4Ti4g4H8nrgJ2PKnKgCuR39S4  8Y5iRMuCwJi4pUEcSGkDuRS4v1gHqWTQ13ffB12NSFVC   10%   75066990 ( -1)  75066952 ( -8)        296078  1.5.19  203093.749541861 SOL (0.06%)
  41iM1ZT5WYS8HgweopShefLJRfDD3jbB1MMJZiuEemvE  Gk7aonsprBZXQBMMEDgRVJ6RdMRwBd3KXQcu6k2T7hyU   10%   75066989 ( -2)  75066952 ( -8)        297638  1.5.19  203108.706491067 SOL (0.06%)
  ArA1pRBW7WFcx1StiRpwhktiXkdQTdoGSj519hzr8hdJ  H6xXak7XhCx6t4wx1Wavdxtw8vShEC9Fvy3ncTJ5C3mB   10%   75066988 ( -3)  75066948 (-12)        295970  1.5.19  203116.195102669 SOL (0.06%)
  3VRzZfDDnNiSBzW259vKgtq5x1E4fzkvs5SZcVtpJEzj  Gas5iZLK44wyFdRkQY2ygb4uXwKPFtVfGje63TVqbFkw   10%   75066988 ( -3)  75066951 ( -9)        298542  1.5.19  203119.018747913 SOL (0.06%)
  9mo138RC4NbMYvWaVUE2cYPpY7KCCNGvgHhi4cVuCyM6  47vMcUcjzzkoEZDYzTuk1DgkbvZChJoMCvQyGAyx9n77   10%   75066989 ( -2)  75066949 (-11)        298092  1.5.19  203119.418557152 SOL (0.06%)
  5iJmsXsq49ZqgP9QbP2gNQRU9C8PhwA2gbgNLdgSHbjh  55MSbLywZx9nkahxHL33JqBBK6YxWoJqctcdc5EX8Ay5   10%   75066988 ( -3)  75066951 ( -9)        297962  1.5.19  203119.757833179 SOL (0.06%)
  PR3GtaLUjL8rSGcAQtV9NYGTWhpys5kYRDUSCMF5N4j   7KNoQc4inYf4gdhrwAX3FM3MzMWhzkbxBziPCQ4r7Po4   10%   75066984 ( -7)  75066930 (-30)        298180  1.5.19  203120.685649279 SOL (0.06%)
  C219S526JWRzpPUm3FfAzobwZMDuo3DM63phtJpqAz75  6vxc5PhUi1CW9jABpXxp6WP8snwhRat1GeYJJtX1n6PC   10%   75066987 ( -4)  75066950 (-10)        297874  1.5.19  203121.871919952 SOL (0.06%)
  9iZV3mMdBh9y17hM3eVXUVuH28pGLERSJBq2z64yfreM  jYdJYZhJQHAgMX8eUhZneshFkzivbyMPHbmN8iVgNpD    10%   75066982 ( -9)  75066928 (-32)        298039  1.5.19  203125.550397254 SOL (0.06%)
  AcHVKQ3LoPUN51Zx2zyA2CeSZJUR2zJ5jsghTpWFgmHf  A6haEh7R7UHL42V5z5bXpzGMVecmudSeb3acWHQGqqr4   10%   75066991 (  0)  75066960 (  0)        298742  1.5.19  203131.747679749 SOL (0.06%)
  BJafMGt4t8A9BENBg9EcXEAqUSgBLaQZujQqvrsGMgtL  8EHeAESs87EejfEMi6u1ndexywxGMAanV7QsUpDiseWP   10%   75066991 (  0)  75066960 (  0)        298534  1.5.19  203133.103103018 SOL (0.06%)
  Ch2UBdfwRY8UyAKCBzYksu7QYwjCXprkbUo7AY9CSRyS  9LWHyBSSKaGsfyFu9jn6ArSRGiTzZ2djNz7tfcCYxDNy   10%   75066991 (  0)  75066960 (  0)        297971  1.5.19  203138.851107108 SOL (0.06%)
  8ynefSJ7MVkC6VnMdM9Cm3V1F57fiYg8ugkfNV95X8Zx  E2pj379sYY2qwYejAueKipLnSUzVU4tBtzKLBpUrN2Z4   10%   75066989 ( -2)  75066952 ( -8)        295344  1.5.19  203145.405038953 SOL (0.06%)
  CUPryNhYfF8ChYyz3tahM84ppX5WSpvRxWqP7PEHMX36  7BEkR6kjnh7XPxQ4Ew5ePby7orLrXKT4zDE97kbN1fiT   10%   75066991 (  0)  75066960 (  0)        298298  1.5.19  203157.771738364 SOL (0.06%)
  G2ccHPYPic368N1b6dHJBnE3tBmiceHGZefjk2FZuVfT  2myRTwMP5fVF9w5CpBpA8Zoj322giJw7j6ofvMNQdZAW   10%   75066991 (  0)  75066960 (  0)        298286  1.5.19  203185.635314751 SOL (0.06%)
  D1KH1UwfTLaBW3NubpvNv4ze9S7SKW6jd5TLdrwZgLP1  1gqv7KGm888nQXsJoNFwGaDkNERUBztuekjzK3J3T7a    10%   75066991 (  0)  75066960 (  0)        296300  1.5.19  203191.439557191 SOL (0.06%)
  4FozAhZhAo8ZTuzNHeAHMDDLqWmRwioWBhFqybZYHamV  FXbkkbLWH4PxSK8t1JmKAHn4pEvGsCFyMT7pfgwm62us   10%   75066990 ( -1)  75066953 ( -7)        297417  1.5.19  203213.75843204 SOL (0.06%)
  WJguyKr593U9gDSdUu9Rcr1TMKo6pTse9LFZfCArR5K   BJnMDPPLRGhDQQKJchwzWFvqvNKW1oPD7qpLwwwpsKH8   10%   75066988 ( -3)  75066950 (-10)        298961  1.5.19  203222.080401501 SOL (0.06%)
  nSGZ3tv2UhskkPqiB666yDVj7PTi9qKgDqvjHyw5JgM   AY271jdvcyo5VzBiWsMGLEjpZFFrarq8FDydJHLmYgCG   10%   75066991 (  0)  75066960 (  0)        297691  1.5.19  203247.257161766 SOL (0.06%)
  HPN8ANgpeF5mcViFW6fTCXGWPdQ6LhRWvcx86JVEkFC2  Cva4NEnBRYfFv8i3RtcMTbEYgyVNmewk2aAgh4fco2mP   10%   75066990 ( -1)  75066953 ( -7)        298711  1.5.19  203299.239719798 SOL (0.06%)
  7jmCjQt2D5MVvSz7NhpHLCXGsh8Mu3qeBbA4LJAf3Xsd  H8BoV8s8v9h5ijzSxg27xbZbAJpcnv7RWpn6B5UdcRFX   10%   75066991 (  0)  75066960 (  0)        298233  1.5.19  203307.557134404 SOL (0.06%)
  J7jT9mGjhfXdGFdp9XT97Sd8ynFMe5e2L8DqXJQr8qgs  8D5bmvzWdmyeReTm5kz8D5bVDS5wuLZHTypCdSRVZ3d3   10%   75066990 ( -1)  75066953 ( -7)        298918  1.5.19  203322.402103675 SOL (0.06%)
  FdmF6aFAy3pisaTwcQnvLf43z3pd1QnqrNzFMa2vKABA  2BJUTarkNTNtiqn6g7mfZc5fjaF3sSnvV73dnuwuZnxV   10%   75066991 (  0)  75066960 (  0)        297754  1.5.19  203342.939311055 SOL (0.06%)
  FcWJ9zuq23C74KzeZtrSZMNXDnKFN9fQXwAyVDTZFpLr  3hn9yQsPUZEGdWmNNGQhDb5vYMHdZRNzNULmQRp4JY73   10%   75066991 (  0)  75066960 (  0)        298476  1.5.19  203346.736323161 SOL (0.06%)
  2EUh4NtRwhJ69UUca8HuGEcABsn1MbvufqFUDF8XrZEe  Cz2XaT611zudwqpb7FezbjmeBxz5BBwtV4BEiyVjnW5a   10%   75066991 (  0)  75066960 (  0)        298013  1.5.19  203348.65004385 SOL (0.06%)
  DJHsoHQvqYjb8G2Ni6XSbBSHxmycMAsZksRDytQ2bntK  GwjRJdT9nKmegq6a8nyvEPfoQZUSPp6N4KmWAUMo76aU   10%   75066991 (  0)  75066960 (  0)        298541  1.5.19  203349.444953732 SOL (0.06%)
  AGsrW2sp9mqNp7g9Nqw6g1A9wMyDYysqTN6d8oaw2W4s  E5sX5xLsupKowpCmSwiBRgPvhUGeKKPGHDRYbpU7rzXE   10%   75066990 ( -1)  75066953 ( -7)        298897  1.5.19  203352.518739079 SOL (0.06%)
  6kfL3zErU3z9iofwdg6iM6UJDFZaVJNguAwy8SiN82db  FvrZudS8hmEfM4PUF2xgVxzgyDqRJdT9byHmdkzAQWps   10%   75066991 (  0)  75066960 (  0)        296578  1.5.19  203356.275678646 SOL (0.06%)
  D2Wa6JtXeyqFMdoacpKMo86Pbr4YpfdVCtAhem8HjqfF  BJMdhjvPfBp3agAthKPDfYQJ9dEUmmXtnM66e337z5SL   10%   75066991 (  0)  75066960 (  0)        295811  1.5.19  203357.497156458 SOL (0.06%)
  CgX9oSHcpXjKKxUpLbrgNKWrk3ff4cQidfsf96AkMBke  5RbNzJrGrfoSgoDCMvx3n8jLAVzTjAjy9dfKFUR3enAy   10%   75066991 (  0)  75066960 (  0)        297964  1.5.19  203361.988349071 SOL (0.06%)
  2nZo1aiuRdTKYCfuoLR8jHQY1TxkTQxCw3d34K3uQkiy  76sb4FZPwewvxtST5tJMp9N43jj4hDC5DQ7bv8kBi1rA   10%   75066990 ( -1)  75066952 ( -8)        296068  1.5.19  203365.737589838 SOL (0.06%)
  9UsSQ9m5QemVf9NFmKncDDXGFGzUXMiFay9Fv7rUmjtV  HhrkPmp8drPTMBrisM91wwoWnFHJXS55VYXGBkE6gP9i   10%   75066991 (  0)  75066960 (  0)        298731  1.5.19  203366.263373799 SOL (0.06%)
  DFKj3e6WeQmLxQkRBHq7nqcEXjwkcfJGCt22VySeDiKx  o8Fcigd8DDix22S39yiS1aegYdTWUo5dBaeBasyK6F4    10%   75066991 (  0)  75066960 (  0)        297994  1.5.19  203368.189367982 SOL (0.06%)
  Anara9qw9KCKMFPX19GUE2drgwiw49u2pQhHXwVRukhn  Anara1R593HZjDT36UdmxxJVsFi4fzcWuML5BUvec44e   10%   75066991 (  0)  75066960 (  0)        296329  1.5.19  203368.466469619 SOL (0.06%)
  BT9ZFvsDfX6WpLFqmWEYuLuE5i3SxzdSJ1Vzm9arbRub  F7MeskoJC8pFwUsw3tNPA5ct8H1FRpuyi4JhFSLBsH4Y    9%   75066990 ( -1)  75066953 ( -7)        296336  1.5.19  203370.058300398 SOL (0.06%)
  ECULsxDc7pqadpQSQxFaNWAsemLW3wQq1vDKXg5P7GjW  D4J88wGP89KdA6xuDW48yZR9AutWaMhWzK8wciZxF3Ue   10%   75066991 (  0)  75066960 (  0)        297895  1.5.19  203371.105783571 SOL (0.06%)
  21REqzFbJJb7itcv2QzkjvHuCR9PyzXV1uw9251JZq1V  8HA2gPaMXKXJDXpbFVxrmTDCtZZDPuEJqP3Vu3xwyq6M   10%   75066991 (  0)  75066960 (  0)        298228  1.5.19  203372.687913472 SOL (0.06%)
  Hsb14Gkhjbp6FinDcPRwrThS8BYsTnrhGY8Ui4f3EY7V  hrmh5JKJRzjhBK3M3hosd2SSdjHBY8yzTbwbXnbBvNb    10%   75066989 ( -2)  75066952 ( -8)        298629  1.5.19  203373.120496402 SOL (0.06%)
  7oGVeZnNWa5yDH6V6ec3Ya1xzSFykc62nXKDGhnbphuY  GkBrxrDjmx2kfTMUZJgYWAbar9fEpYJW7TgLatrZSjhN   10%   75066990 ( -1)  75066953 ( -7)        296988  1.5.19  203373.9138777 SOL (0.06%)
  Zb8KTJXwFvBTnbEDbyxWs8AVWjjipbXNFAfc1Jsbhey   6XimUrvgbdoAuavV9WGgSYdYKSw6ghajLGeQgZMG9aZU   10%   75066990 ( -1)  75066953 ( -7)        297867  1.5.19  203374.555164195 SOL (0.06%)
  FuSZq1Xkvkjkj16fGdhDtZb5ATtsZ8FH5a9KGsDhtZQP  GwAvKMYMeC1Q6PUxKGDiRhj97Bs1kxPXTSWxQZKfurrW   10%   75066991 (  0)  75066960 (  0)        298127  1.5.19  203374.778512995 SOL (0.06%)
  AvAivvbUcs2ewGpj2TxW39BXmA5t6d9ofQZFzwagqAzb  9DYSMwSwMbQcckH1Zi7EQ3E6ipJKkChqRVJCQjF5FCWp   10%   75066990 ( -1)  75066953 ( -7)        296247  1.5.19  203375.605733557 SOL (0.06%)
  EwHxEEjLrrhzB2TUNMEJh8kNcDSnfBYvuZzDizBHF51J  4RyNsFHDccFEEnbYJAFt2cNufFduh8Se8eKTqXDVr82h   10%   75066989 ( -2)  75066952 ( -8)        295572  1.5.19  203376.964385681 SOL (0.06%)
  6PbhWC1XRHsW3npzCHm374yE72B2mbjBrXdUAhouozqm  BeNvYv2pd3MRJBBSGiMPSRVYcafKAXocNNp79GoaHfoP   10%   75066990 ( -1)  75066952 ( -8)        298033  1.5.19  203378.058167292 SOL (0.06%)
  Pidptxt5SEqgeK9HgqacrY2KEEnMqtUFSmAucLwnTPG   JCHsvHwF6TgeM1fapxgAkhVKDU5QtPox3bfCR5sjWirP   10%   75066991 (  0)  75066960 (  0)        297848  1.5.19  203380.608032492 SOL (0.06%)
  7VWgBZJCLtuCTGdWoiWPrVqY1TaqiZQzEqmArzGzrwGN  buhBT4ESNHyno3aJALq6efd5XHq6iC3MiCpAmpjNGLo    10%   75066989 ( -2)  75066952 ( -8)        299093  1.5.19  203383.440261961 SOL (0.06%)
  5HXxjDZwm7MAZAm2aCgGcGRr3SKiwugcQymoByyd7pfv  FX7PqJTLd1B8vD2cnLzRWx57o85j3NpMojHFNwsmet5q   10%   75066990 ( -1)  75066952 ( -8)        298039  1.5.19  203385.70064584 SOL (0.06%)
  2BHDXBCcndLg6oYu7EvSd3TuVJjUQX5xmUiKb8fDMjRt  EEdseGfQnkSE8Dj7NffoXrpTirk1xaS1bMRejszF4fAb   10%   75066989 ( -2)  75066951 ( -9)        297787  1.5.19  203416.813388611 SOL (0.06%)
  BLPBUAuWmjTKT8rF6nXnHJCcuK4ZonH4xdYSD1gZiET9  7iEP8fEN2be9CZdeiXirtVZTzLKQCEsfMA4E4eeik8T2   10%   75066991 (  0)  75066960 (  0)        295813  1.5.19  203482.574055722 SOL (0.06%)
  GUvRSvhhQRA1PhTpMaqW5hELHcPP9QP4W45tHFEbtqRi  Gphb7i7aHkapwMuYHGT7SzT68WgFw3aNT3LFuNHVkatc   10%   75066990 ( -1)  75066953 ( -7)        297576  1.5.19  203513.489362211 SOL (0.06%)
  8tZuYRKd5kx4tNsZMZHA9ovsjKkQWf56kfe7ycLees9B  HGwcdwquyMtpK6VYLsgYTgftCLQabkxwpDykCHE2CWyg   10%   75066990 ( -1)  75066953 ( -7)        296648  1.5.19  203514.715543487 SOL (0.06%)
  9vpYXvRdqNJD2YKRZ9q6Xm7fh4FdPGuc5PBZSusv8vbi  8dk8UAi3d6K49fCPiwXyPynhYjQX6wxVA8mdSNzTGVts   10%   75066990 ( -1)  75066953 ( -7)        298631  1.5.18  203514.767846348 SOL (0.06%)
  DM52rUKVkCPDY3MFmSL76XAuhB5ZN7MuogAiQtEe1Uvn  CYUoCpyMGkUWSa9kMhrMGePRvfeuPCzkboERbiukKHa9   10%   75066991 (  0)  75066960 (  0)        297886  1.5.19  203542.153721911 SOL (0.06%)
  4e2KvSCgot2RGXsExfY48NdfykQSjgozV5FAXv13bUn1  9y1fMDRdYaYbCSWEv3sivjns4Sc7eDW8wFQYT1DnCsQa   10%   75066991 (  0)  75066960 (  0)        296495  1.5.19  203544.471410217 SOL (0.06%)
  6RmSMaZbyb7SGKLp34MXY2qkFePZh41Frs79xJgEg5R6  DGu5PHMFTRhrRcqXw3EfmMxHk3eYLVL6iWbieQ63GYoN   10%   75066990 ( -1)  75066950 (-10)        298821  1.5.19  203553.601951465 SOL (0.06%)
  cZCvgqgrdX2LCScVku8CprMnAUZtSicBDJ8eiowfonn   8BbpzqTUCUa4oR71AMgcBiTQvd6Gta3xJHCG4xK9mKBp   10%   75066991 (  0)  75066960 (  0)        298476  1.5.19  203557.326112 SOL (0.06%)
⚠️ wpd8e4dzFVnCQY3jPuB9J7pKYQd6Mstz5tjWzF97Kxb   Hp1Eieo4bm5u94AG26mybZJB6NsKaB1KYR8DCPriupDA   10%   74986909        74986878              225667  1.5.19  203568.409418256 SOL (0.06%)
  EXCMwETx5Txcvxt6YYqxFmhSpQKH5BVjdat3NE5eJJ6a  3yafqDTZxjRCaF8TgPYUvGhnCeuK3sGyoXVRwWnS8ADy   10%   75066991 (  0)  75066960 (  0)        298422  1.5.19  203570.510518579 SOL (0.06%)
  JdJWLp5edqUHQB7K9mA3L1gFbFcrS8aeaar6hQ6M2Su   5EKMG4vkPiD3r7EePz2PAJ1HEbimw5zcZJh52dk8SC7R   10%   75066991 (  0)  75066960 (  0)        297730  1.5.19  203579.856651851 SOL (0.06%)
  siriXy5CcarNiz4XL8ssBQGiy2PwReVLny3Bcxq6Ymb   a1exajFBsggm7R7ydb4LwyEAdsLCUKNAiUrre9B12kV    10%   75066991 (  0)  75066960 (  0)        298845  1.5.19  203591.222119176 SOL (0.06%)
  EVkRg5yjkmBP5tAmSiM9zBcrfnFE82rv3fdVvTLR2sxL  FPS5ptyd5iJBMaWs118XwjWHvveNSELj8QeyTG584J3u   10%   75066990 ( -1)  75066953 ( -7)        298599  1.5.19  203604.028108263 SOL (0.06%)
  FEpYb3oJbdPf77DPdQagkcmcJ4SqcfRDeCydKVkks4HK  7gds7PbCzmHbJStjxA5L5K8cu2LVUakmd3MDXFHSfcic   10%   75066991 (  0)  75066960 (  0)        298441  1.5.19  203617.599957942 SOL (0.06%)
  6poikjtKFzySv2zrfEJCQorTDJWmoqCLPbSXeNLHyvL3  B2vsqPPAiLMBZqhuqQdvx24ghg4AxMw76pp6V9kNTVms   10%   75066991 (  0)  75066960 (  0)        295364  1.5.19  203629.679975823 SOL (0.06%)
  HW4zorvt6xDwhU36RqjcWNwU8YMj9tiqnAafBKW4cqV   4jEHuQZTNTRYAhxRYEjV3HJ1b4wqdQjnBRdPzFWzkCft   10%   75066990 ( -1)  75066953 ( -7)        298290  1.5.19  203646.066777971 SOL (0.06%)
  9uAtx66puPQZLaP6fMKNKKyRtGzonhaUEdbVopgeWcfS  FEtRZqL16RiizCa7CmZH9tsN7oNfWdKtBf6GM1ETzWR    10%   75066990 ( -1)  75066953 ( -7)        298721  1.5.19  203656.534158889 SOL (0.06%)
  6ZomMAu9XSu9fzkRjqM6HbAXagw6Jo9fTsfsAADTZMSp  4CtNJvygjgJ3aY77eLV6Ld5V7g2M9bD2t9XYxYFcbMJ8   10%   75066989 ( -2)  75066952 ( -8)        298980  1.5.19  203660.344278697 SOL (0.06%)
  Hmube7PkFmvattfygQgEyWe4GfZ9Dskj4Zc9YFfRj9JU  HoFU1hVTYvh4ryBfbYCzTu9dBta73jX3Ag3rzzoqYJG6   10%   75066989 ( -2)  75066952 ( -8)        266587  1.5.19  203684.414154162 SOL (0.06%)
  Gyxhb2GGcB3s5h1HcHekRJYyAHRXEDP1FfX5fncE9edA  2xjAQsHLsV36NLFkxdApzLg4SNqm15mNqYaBQ4xp5joh   10%   75066991 (  0)  75066960 (  0)        296684  1.5.19  203701.665874315 SOL (0.06%)
  ATwrKSHdF1JptZDFd2wUeU8reP6ftp9P695W9ipdB6TQ  CnYYmAhuFcyocBbXxoVzPnu37a5ctpLaSr8ja1NGKNZ7   10%   75066989 ( -2)  75066952 ( -8)        298320  1.5.19  203718.011168366 SOL (0.06%)
  EwUVzgSPe1zy2hfUGZxJAEP7Y1wheNgNsgratbzPELru  438C7o4cJrbchdmxEPvFgaE5g9DocWKeSc1k4E1aY7qw   10%   75066984 ( -7)  75066929 (-31)        297659  1.5.18  203742.279257817 SOL (0.06%)
  98vNehxpVCPz1t5RjnZZVjxQ549a2VLUbkpNwqj1rSb8  8eEzaWfM36K9XsQvryAc8Tu9nfnqEfYib9oyCzLrAM5C   10%   75066989 ( -2)  75066949 (-11)        297979  1.5.19  203770.231077469 SOL (0.06%)
  FxE69xVkPAUYh3Y2QCHJVWwVB8x1F3wbHnfKGoUvXn81  5kYPCTLoF2y7cynJEuv5LVaTPpLoPbcxiCYKk3KKcB1C   10%   75066991 (  0)  75066960 (  0)        298689  1.5.19  203809.316492539 SOL (0.06%)
  8vnrJNMDERZRwWgUMSqwLyYHEPkQQg3Ww6BM9AH4uX5R  Fqb6omRdS2gtnxmxCwQQ86mPJK8MtY7tjR4eQZX9n5J7   10%   75066990 ( -1)  75066953 ( -7)        295760  1.5.19  203810.306107189 SOL (0.06%)
  H2oJUXwghyv6BwZH68jobU8jGutBji4v3WbPA96kc5Yd  HBe7sQRV5xay6JS4notvKG8DRDRMdedWjaotv94nRC4N   10%   75066990 ( -1)  75066953 ( -7)        296472  1.5.19  203833.00573504 SOL (0.06%)
  HS8DF8wP3A8qSHPnzbE6pSvXub4FeZQ6xMNSJuqArAHS  8HKHFcDGkmeedbTcnK8ntXdqRhYJYWEGbk8325tgv47i   10%   75066991 (  0)  75066960 (  0)        298260  1.5.19  203837.524041965 SOL (0.06%)
  9bxGPEvFjGHqpAHMkm97R5d8euFnpJ3ws83tMkTbcBUJ  CqSMzh8DWZeqYVa5M1V1rHU825T19NCjYipM3pkdHncm   10%   75066990 ( -1)  75066950 (-10)        298164  1.5.19  203840.713881993 SOL (0.06%)
  BzF3s3AGgYupiL5Mhbzr1tu1ySMZSR3SXU4DajV1Zwv9  9G1FYXgP5crtEbGX87yrmGeLdtKkjvYqhFMey8cPReBq   10%   75066989 ( -2)  75066952 ( -8)        297634  1.5.19  203850.271468639 SOL (0.06%)
  C5oTsyGA1gBvgWpvS6EaWGAgEKDBkMpQaPeLTiaLuYKi  EXEGg4fRhq7YqK62oKejKeErBHLJqM9tb6iqWpVaHWDD   10%   75066991 (  0)  75066960 (  0)        298877  1.5.19  203851.242292446 SOL (0.06%)
  BJevND8rrW5f9AHSysSGTtDWKcARxSpzTyDAvCm26Qms  32PqUswoHFz1mEYqm4bsxyZmtXFQUc4x6a4JjLNwhHVo   10%   75066991 (  0)  75066960 (  0)        298519  1.5.19  203852.011522808 SOL (0.06%)
  FVsjR8faKFZSisBatLNVo5bSH1jvHz3JvneVbyVTiV9K  EY2zW89vZnPe2ts1FMzPK59cuEq69rRzEAhyLNge8gUj   10%   75066987 ( -4)  75066928 (-32)        298165  1.5.19  203856.918517695 SOL (0.06%)
  4WgsjJxehCavDYXEMTBQX91KKh1szuQtzksuxZKgymq9  5znzbnUiJTPe3TgZw9L1saWCWQsuXSAgwEP4FA1edEsi   10%   75066991 (  0)  75066960 (  0)        298406  1.5.19  203865.121997244 SOL (0.06%)
  D2RV1q6FgePVVjrMa7AMzVbvvAeg5oS7TAV7qdNKSDsX  CCxSNvJogH6LWyoiEbG7JfcWybw2FqqCExs5GuemChGr   10%   75066987 ( -4)  75066949 (-11)        295772  1.5.19  203865.166303666 SOL (0.06%)
  Azg1hC9mg7pkSq6Q5sAESsBsrxHoUuyRnUYSCJMJMdup  HQ6ywTaLZxjEVLJsR4rLeLjGw7ZDJKd2c5fSF8FHxaG7   10%   75066990 ( -1)  75066953 ( -7)        296084  1.5.19  203866.097611332 SOL (0.06%)
  5cshESzkc3hmUA3qnp5ridF6WTeiVVLUYkZUaCPYfbid  2n1UvSBV5QC5rPZaWT3vg6huK7K1oByzumeavEDg4mvR   10%   75066989 ( -2)  75066951 ( -9)        298217  1.5.19  203868.298108488 SOL (0.06%)
  4w2p2n83ter1rkk7Z6r63z3uNBKuBXzgV9KFDWBidSFQ  BHGHrKBJ9z6oE4Rjd7rBTsy9GLiFcTeDbkTkC5YmT5JG   10%   75066991 (  0)  75066960 (  0)        295592  1.5.19  203872.596421732 SOL (0.06%)
  GfCNi6GaCfEKyvpS8fs1D6VUdQcPJg4VrULvFFEBLFW6  BUY3swp2NSTMsQmLDnYgoEoLNJyCX1MQZQjnAVL7SnSx   10%   75066991 (  0)  75066960 (  0)        298923  1.5.19  203873.391747517 SOL (0.06%)
  4rhWUqqNbdoVsGGw7P6Hzc81FB8C9jgR3UGF7aUgKAcY  4qS6unxhpNh6fp2rRU3nnyMZEYyZ4hUbjnP7iEN7Jx1w   10%   75066990 ( -1)  75066953 ( -7)        295507  1.5.19  203874.005828096 SOL (0.06%)
  6npqjzgFPuL7dKdaUsgLwKokWLDe3wFyHeDaVT7Pro3d  AU4yDLbrnLzcjk2pnxvXwNeKJsj9CiUDRXWQbeSbk6Y9   10%   75066988 ( -3)  75066951 ( -9)        295826  1.5.19  203874.289736496 SOL (0.06%)
  68EWE8SmV58oRG9JJELNDt8Y6MuWBbfraoT67KFw5Ath  63FR3fGznHYCUrsNtHcdMhMgcWBc9KTY5Mc4z5k9rQ2K   10%   75066988 ( -3)  75066951 ( -9)        298034  1.5.19  203874.525828379 SOL (0.06%)
  J711fLahrgkEnPaCrbGUCpGemmeVg2LsaZhmC6HNPbXk  JB8zjnRE6FeT8N6Yq2182vj69kKHGdeKJ7kBAhHKuHRq   10%   75066990 ( -1)  75066953 ( -7)        298075  1.5.19  203877.142390293 SOL (0.06%)
  HZzEML2w5Qs6cccMd7o57Vx2sQUuuTaiecEcMU2xceSK  9EzbogBnGi8hVeLXEyFu2xUo6qi5JdEELs4y3cQXQW33   10%   75066990 ( -1)  75066953 ( -7)        297922  1.5.19  203877.502280064 SOL (0.06%)
  FvTY9S6vutvfyP1eJG7RLiR1er6nk6cNTSP8q9WRpvWA  CvqeNUd9Xq5a2681T1pwr6E86tcbEYnEViCyUfybLSQo   10%   75066989 ( -2)  75066952 ( -8)        298365  1.5.19  203878.43073806 SOL (0.06%)
  6YXp94DjBg2cP3tiVjHhgZYLP7juG62GxzU3yZFE4zoJ  A9hwhEeQ7hNm8rPbRX7ZDAZRjTVrUCjgDEDD4Tt8rmT7   10%   75066989 ( -2)  75066951 ( -9)        298054  1.5.19  203879.991825281 SOL (0.06%)
  5DhEjMqxWWAagBywA6kL7EczqTnDGvPmDfseottjyps4  FSH9xke8FBpx6YxEEzNXVWgjmT3G5SN9HpipmCSVamV    10%   75066989 ( -2)  75066951 ( -9)        297847  1.5.19  203880.687594476 SOL (0.06%)
  3zaPajeDw8FxutdgFTUSVKp1cxWFqs62dtuCZNQ18TT9  6vJGsbs5jYKEdQGUfMEYN4Nenwscgza1dBXB3WJraFyH   10%   75066991 (  0)  75066960 (  0)        297141  1.5.19  203881.010664735 SOL (0.06%)
  55rEq5xznJEMEShWB9GFv5WNX5NfCZckLZbXXDyW2seb  Fu4wz4US6dV6GzZrv9NnF18KeT47tdbDKRd7pA6DiyS4   10%   75066991 (  0)  75066960 (  0)        298305  1.5.19  203881.489274511 SOL (0.06%)
  HPdVgk7NhewBTKDCyqKYFXLu3pErnUo5W9KLSLd18JJv  FvoZJRfV8LWMbkMeiswKzMHSZ2qvU8KVsEkUaW6MdN8m   10%   75066989 ( -2)  75066952 ( -8)        297817  1.5.19  203882.152168556 SOL (0.06%)
  7zPjirsZ2GLwji87WB9EPyeLXTzigjeVAxH14nB1Ss2E  4qSZsB9QjXr97HzhzPd1zuvB8z7tqqDuM1xbxB5PcPFh   10%   75066991 (  0)  75066960 (  0)        297973  1.5.19  203882.218580038 SOL (0.06%)
  6RFbeXzEyabceq71fMBLuBVofWedpZAu2nK9vNbZiqei  8kcrp8M2c5LGYThHQxVgsp7BGfGjHZ9fLHa6YN3YpFNa   10%   75066990 ( -1)  75066953 ( -7)        297748  1.5.19  203882.345531068 SOL (0.06%)
  D2UwKTrNJGNLM1h66qjoSDTK6CLcPSzmga84MFuiWdiS  7pjeXeoiKmu6nZHoHCK1AV45Ajy2Lvsjae9tMp3aeGqx   10%   75066990 ( -1)  75066953 ( -7)        295318  1.5.19  203882.485097933 SOL (0.06%)
  25andkv98haPs8LeocDfmKwfNnpbNcHtsWoHDSBrWGYb  7LCnWqQGpNCiUvBLznYG9Q6Zo7mcLkhAHA7YBjbg8SET   10%   75066991 (  0)  75066960 (  0)        297856  1.5.19  203882.521536771 SOL (0.06%)
  AUgbNHRcxPqfmq3CXQ5bvy3BpN8KZt5LtXBGSH3gV5B5  FfE7rncxyYJvsqFu3Kn323sJpjBXkfMNXwd4d8kdURk9   10%   75066991 (  0)  75066960 (  0)        298284  1.5.19  203883.104166394 SOL (0.06%)
  3nwmifffA54NDWqiZH1oYMKHaS21Evmy7rjPeL9veegs  8HfXSEtpDodzLrWvbUuLhH2nU6axzAzL8eoLNZK4GCbM   10%   75066990 ( -1)  75066953 ( -7)        297696  1.5.19  203884.906200919 SOL (0.06%)
  9cYyjirWYs68YKw39r83qXv3rQB52dyviteRDKrDX2GC  9gm8bZcHVb4umKHaULgF1iLqNnr8Av4CAUgxfZhM9TE6   10%   75066990 ( -1)  75066953 ( -7)        297975  1.5.19  203887.297201521 SOL (0.06%)
  HU8uKcH8b2GRD5T2LYKDzV82H7CiRLTQLwYTNxAD7b62  4PTVcstjs4s89uETGapnjTNPDa4XfHF9kLUu8WCeYNQb   10%   75066991 (  0)  75066960 (  0)        298588  1.5.19  203891.344605794 SOL (0.06%)
  692qK4TkXnrGfDZfoSq9aFfnoE2WNW5TpMRnN7M3sumF  FpEPW2mhwkWobuzAeamPokAhoi731bkiLXXrptFiXTSR   10%   75066991 (  0)  75066960 (  0)        298455  1.5.19  203892.047074198 SOL (0.06%)
  BfVzQge1z9dSZ2YiGyRXoPRUvowe3S2kvbLU61zXn1cV  6g6jypXGeavZPVkWSu4Ny5bfhTMLFnuSepfGMQkQpWV1   10%   75066989 ( -2)  75066952 ( -8)        298107  1.5.19  203894.709883267 SOL (0.06%)
  G5rEbYLuVsTiXP1hUWkeJp9PptSujTGa3mFRqrxXgMx4  2rNaaG1yKrGY5KU2H8gGWtQzMitoWHcdVroMpVDg5AYy   10%   75066989 ( -2)  75066952 ( -8)        297965  1.5.19  203894.858210046 SOL (0.06%)
  2U8iE8LGdAqep5y3toyrBaZF6DHP2kf3Ev2vLFggJqiW  DqxQuDD9BZERufL2gTCipHhAqj7Bb2zoAEKfvHuXWNUL   10%   75066991 (  0)  75066960 (  0)        297750  1.5.19  203896.126080929 SOL (0.06%)
  DViARWAWKkxAzp4UCgbw5B9pLSrBY3PaztFErcwgVUKX  HeTyhZdUKswQoonJJTXqAnDN48ceyVAeFaKfYKayGPNS   10%   75066991 (  0)  75066960 (  0)        298780  1.5.19  203898.288883854 SOL (0.06%)
  7XSCAfoJ11zrQxonjbGZHLUL8tqpF7yhkxiieLds9mdH  EujFnyMz71DXdN93GeSCDU7pFiy1vrFG7cAbtfAhFeSe   10%   75066990 ( -1)  75066953 ( -7)        298662  1.5.19  203899.852525578 SOL (0.06%)
  CcTtRsmLJEjqsv5iyfXSYwjaUJdfrRK7AU9cHMnQfTb3  HiLaiF6HAUL2kYBsHcVb3CDws9c3dEphtfPbzDnKYFYB   10%   75066990 ( -1)  75066953 ( -7)        298123  1.5.19  203900.441535435 SOL (0.06%)
  81zNHWkYaQRL3TWHSNwzX6T64yUyVQ7qXzLLxvaaB6b3  2mQNruSKNnn6fWqJjKNGsQtpsMnuxxMzHsrKT6iVR7tW   10%   75066991 (  0)  75066960 (  0)        298477  1.5.19  203900.691918635 SOL (0.06%)
  DL6URBwusvYUFwEYZK4VEaoaynWSduUA9e4N7WDuZawf  7cpDpUt9GAWaD41QbHCXiobDYmBSurxjGY9ChtnzER46   10%   75066988 ( -3)  75066951 ( -9)        298197  1.5.19  203905.789965288 SOL (0.06%)
  776BzpbpsZU1rbCkNHizEP5r8RE8QL12Xqm49krkLPLy  36jVWrHfpyN2RvXCkGA75qdzcBqFfMcyKbC2qg4o2gA8   10%   75066991 (  0)  75066960 (  0)        298534  1.5.19  203908.459950678 SOL (0.06%)
  9JZXYs5TisLa6gra2PDuNibbi2979AZuXNY9zSVGz4UA  BS6tkCiU9yi4GE8E1kvjQaAByLynjRoJ4j8qd6szdwdz   10%   75066990 ( -1)  75066953 ( -7)        298209  1.5.19  203909.078266355 SOL (0.06%)
  BAHNFqttxYMMZUDqMm1JQPnxkZoHLnpash1ouM7pnLEA  puuuBCC9zKPt3hMdTBHHcgTUAAj8LfY5WUfgcCTkJoy    10%   75066990 ( -1)  75066953 ( -7)        298900  1.5.19  203909.991861489 SOL (0.06%)
  6DcjiCqt6E65w8P352JvAN7qDkdQc7YQJHBrGf5aVAze  dVcK6ZibNvBiiwqxadXEGheznJFsWY9SHiMb8afTQns    10%   75066991 (  0)  75066960 (  0)        297667  1.5.19  203910.384393066 SOL (0.06%)
  Hj2jzpAp57KyM3SmnYwJbDVrQ8tTWizMon2hhzYzwxet  686JcEJ98r8fMtUiVuKiz4WRoBpJ2Sm9zMhdc2b6H4bu   10%   75066991 (  0)  75066960 (  0)        298579  1.5.19  203912.172037669 SOL (0.06%)
  4JryygoiM1j324fYkeBzcQDcwRfd2WpgkEzUePFj1rJY  8ge8UzG9FyW4NDZ6zzZiUypnbcNQ3nUfTBKndxsmWqSB   10%   75066988 ( -3)  75066951 ( -9)        298617  1.5.19  203913.812353183 SOL (0.06%)
  9xsgKAU3pyKZZddPXdh5wLRqjdw2Fc93BL41JszhEpZz  9o4Y8WvRisPxEpuo4sZBq1CtfNsYanzp87bSgeYuGf3R   10%   75066990 ( -1)  75066953 ( -7)        298093  1.5.19  203914.021159782 SOL (0.06%)
  FAAvB4WnbSPNT35oUXDvkYj932KFRPi24dNBtrSpPvCY  6HNFsRJCHnfsT88sxsgtnq1MVsrUPba8fDiPJ1udPfF2   10%   75066989 ( -2)  75066951 ( -9)        298630  1.5.19  203918.313838097 SOL (0.06%)
  3T2nqHFbmexvkhEo25SKnJsbaAjuyUGBjvdttczxDKgs  2hjoQMqRAG5ftTPCSkUavMEaVX35qbj3LK3Q3HDmv4mZ   10%   75066991 (  0)  75066960 (  0)        299188  1.5.19  203920.385856122 SOL (0.06%)
  GRmtMtAeSL8HgX1p815ATQjaYU4Sk7XCP21i4yoFd3KS  GF9q1CPvyjZhAHex6wkGqeCioEkLGzaGJUF1QgZwGFwa   10%   75066990 ( -1)  75066953 ( -7)        298748  1.5.19  203932.869198457 SOL (0.06%)
  6UvA3z3sKbxGXobTQngyL4iVX7kV9jeGYmCUGRMYZqcK  Ay7vbs9t8KyZvyBeoSJKZ6YdweMnXSWujdS9UFKnf4vx   10%   75066989 ( -2)  75066952 ( -8)        298590  1.5.19  203935.674192235 SOL (0.06%)
  3LKjD9Cb8RKKbmwM3LphHEvfZdjEU4rAFGDDUiVnuXhJ  4rxFGSzXiTXuF9GveXbMr4fJAPPnQVjHmpEZbWV8jz9m   10%   75066991 (  0)  75066960 (  0)        298623  1.5.19  203941.327387006 SOL (0.06%)
  FtyZ7aQneFyJs6wKsMTt6m9JFjYEkHwZ2DhhtS6gw3Th  9Mi8M1JnRmtcYpB42DxYPVmYy2safgdYFmeHmMgkW8TG   10%   75066989 ( -2)  75066952 ( -8)        298219  1.5.19  203950.886576358 SOL (0.06%)
  6vi5VzR8KURahz7WheNGRTgGWHnKJQiKkqNbfua1sJiC  5ycsa9WVK6zFcUR13m3yijxbevZehZeCbuSocSawsweW   10%   75066989 ( -2)  75066951 ( -9)        295650  1.5.19  203975.033991839 SOL (0.06%)
  CLarKkpJDBiJYLXqKLVEL8VJaAxRHnWRyjjxAunCFJ41  DREVB8Ce8nLp9Ha5m66sduRcjJtHeQo8B9BkYxjC4Zx3   10%   75066991 (  0)  75066960 (  0)        298406  1.5.19  203976.300250258 SOL (0.06%)
  5kMJh1BtYFrvr6EP56XjoCZmyxx3by61dEvGewYTMuxE  DMYn88X6PkHAc2y5zDWm5jGZ2Tk2CyBUe8K1U2obF8jc   10%   75066991 (  0)  75066960 (  0)        298085  1.5.19  203983.91700785 SOL (0.06%)
  CRextgJEiudn3vzHZauno3a8BpfdweYEdLEjhpZ8C7Cm  5pcSptdL9QTfWLJS6ebhwGBKLeReVdyn5ixRdRbp7brh   10%   75066989 ( -2)  75066952 ( -8)        298782  1.5.19  203992.647679686 SOL (0.06%)
  7DBhzr38xGJeALbZB9bpA5bnzVL8N9oGCjFfWuN6t8cG  2cFGQhgkuibqREEXvz7wEb5CwUqGHfBSTB2oa1hmhkcw   10%   75066990 ( -1)  75066953 ( -7)        299070  1.5.19  203999.37449738 SOL (0.06%)
  CmoYHsxUDcReka49CbnJqoQ3y9hHYkCoTBZ59jA6k6LF  EveHm75RaKGbBGjRsJKmNaKhnaNeRGm6FZFq9xNPYdBN   10%   75066991 (  0)  75066960 (  0)        296221  1.5.19  204000.184355716 SOL (0.06%)
  JCmdhNCyzypryaQUGLGxbCM366dScTD4tVy5ooSWyaBZ  83c5NZebvsUQsL3e9xb37nHiYchLCudDYtQ5Z3gLnLxv   10%   75066990 ( -1)  75066953 ( -7)        298723  1.5.19  204008.813488001 SOL (0.06%)
  Fw3khiQmpKLSdNyoVR52WxbTWT9gjFSXFsZewknwLGb5  J6hFEnGhsGxTyss6DNW9aNQskPZ98gD8P6Wo1F5MBsCK   10%   75066990 ( -1)  75066953 ( -7)        298782  1.5.19  204021.400806271 SOL (0.06%)
  7Gjec4iDbTxLvVYNsRbZrrHdtyLByzdDJ1C5BmcMMBks  CKBuZBQBsHwxSpgVsQC8p7p3fRgCxqGnwpAR7H54Mjom   10%   75066990 ( -1)  75066953 ( -7)        296459  1.5.19  204036.066042675 SOL (0.06%)
  GVjtyyy9HrgBmQ62TxjyUuaQDvuHNbnnrZgi9gqwpVYv  FNsmGoqQm6wxcAnCtmvuLQNY915WEfh1dZS6tMeP25kd   10%   75066989 ( -2)  75066952 ( -8)        297880  1.5.19  204039.16868168 SOL (0.06%)
  7PwCuKPmGF3ZqWHgn8zXPtsWJ7Ud2q1DFggRkzctwJnJ  B2pPLcxHFAkrYYAEMMkpUb4QtSR46FJ5u6bYWapwW9Fj   10%   75066990 ( -1)  75066953 ( -7)        236040  1.5.19  204072.450895657 SOL (0.06%)
  FaGBP8LJrXE5h3cFRtPxz8x8695LQAYnvYn3NGtVtvrw  H8ByRrxt3hxFY6wWEsRSAUvD9iT5wJWdVYQzCnhuPXsU   10%   75066991 (  0)  75066960 (  0)        299072  1.5.19  204087.084728918 SOL (0.06%)
  uEhHSnCXvWgtgvVaYscPHjG13G3peMmngQQ2ghC54i3   5K8qgC9nHzKHSSyo9fKLsMfYmavYdMgEaYx86cMmVKVv   10%   75066989 ( -2)  75066952 ( -8)        298595  1.5.19  204101.902145245 SOL (0.06%)
  AXXrLQ7bVy92pPxW7heQkAHfg8LuJ4uufUWCQgH12rmR  C4MqmofwTAQKYjRawSSN9NhKXd7J78tPnZGLyNSXV2To   10%   75066990 ( -1)  75066953 ( -7)        298305  1.5.19  204146.084331083 SOL (0.06%)
  9YGn24qD8ZCU1EKZEsPLSEv7SVAJ4kpbCTmj7fR5TXYv  9DUTDovSg7Tfc2C9mBHvtGfqfgaBY7srBb2ZYPCDgUPY   10%   75066990 ( -1)  75066952 ( -8)        297004  1.5.18  204149.807337715 SOL (0.06%)
  CouDacpBtF6mmN5K9aXyVds1XRkmo6J4t8ebFZxAecGh  3v6FfdWMT2bcoQQ9hN4F2syu7qhRHzNuCPPQqV12hsw2   10%   75066991 (  0)  75066960 (  0)        295976  1.5.19  204153.316961474 SOL (0.06%)
  JD6R8sK3cu7tphNMyhzxSGnSf9DcFUacXBgqycvGAjai  GzhZDPBjhZpeteBULgSuwpPRd76yTCH8P37PCMmtN1yh   10%   75066990 ( -1)  75066953 ( -7)        298735  1.5.19  204155.462387815 SOL (0.06%)
  BcMg5hjgLZF2mCJ6L2ah1p7TxHtH1euPWP1kZs3USRyC  6Y8YTKs1nNFtvQEy24XYtXQ8JvHHgnJjdCByMrRBmwn3   10%   75066991 (  0)  75066960 (  0)        297598  1.5.18  204156.162980853 SOL (0.06%)
  6zxkTupZL736phbbktUkWQSqWTzeYKbTFoKMENoNnJW3  G4RU9qUt7tG8M8E4L4ZfXtdnwPTcPpaWwLEvSxtdRNHF   10%   75066991 (  0)  75066960 (  0)        296120  1.5.19  204156.367913267 SOL (0.06%)
  3A8F9LjrMgY37qZVAhQ4ccWmrvpQ3oosXfATUtV9ozDA  3HSgNsx9rQsAFZrL7k2BAUuL8HpCjhgfxXjrPBK9cnjD   10%   75066989 ( -2)  75066952 ( -8)        298268  1.5.19  204163.754538037 SOL (0.06%)
  6YPgmQm44UeSEeLU4NRP4fE5HWrEoKupqtRMK72pDjbS  8aGU18Nxn99AEWEQNrBYy1ZsJBhiHVFrcqYqHQPNhmEv   10%   75066991 (  0)  75066960 (  0)        298513  1.5.19  204190.157011047 SOL (0.06%)
  EUVBn58XXTX9RBTm1R7Wd8n8JkvBMQfc9uSn5wPhbdBL  8NQEDxYxwUEPQk1yKFJEGHvYj22Je5xUQWLPkHfKkwrD   10%   75066991 (  0)  75066950 (-10)        296731  1.5.18  204197.778025144 SOL (0.06%)
  6iYUKnywYPEZt58hYs6WKA5tZjxL9MFupCxDkygX13dm  3YKcH4c8eoAKkghQeGavg9HZ13fSe77RWM3QoFTCV2Gv   10%   75066989 ( -2)  75066952 ( -8)        295952  1.5.19  204217.964854848 SOL (0.06%)
  GbpomwGXpPBfYpiMz6KWXBa23usJM6Cf426NQGzAKV2W  3iD36QhXqWzx5b4HHhkRAyUcbEgCaC42hi1GcBePNsp2   10%   75066990 ( -1)  75066953 ( -7)        297990  1.5.19  204241.908158203 SOL (0.06%)
  8o59jbS8MEqSd1J72ryL7dghg9djh2BAUtpkZvTTYh4H  7kge23FTbCXc6dZxMPQfAWLnVFTgGzio5wpXUo9si2mV   10%   75066991 (  0)  75066960 (  0)        296188  1.5.19  204281.786899294 SOL (0.06%)
  GSXEn8TVBgiqfiLP5c8VmUrdAX8sWDZVE9M6p2kZFehG  HCZJjvZbaKaPTE96jz64HnBZTnXHBFv3pugqsBE5Z1D9   10%   75066989 ( -2)  75066952 ( -8)        297899  1.5.19  204387.727293153 SOL (0.06%)
  FVmo2S1GZBRJLS5GM2F5uYF6bcg4rM47dtkxHABKES1b  9TTpcbiTDUQH9goeRvhAhk4X3ahtZ6XttCjRyH8Pu7MP   10%   75066989 ( -2)  75066952 ( -8)        298027  1.5.19  204388.906407189 SOL (0.06%)
  EssmJHKwUffWWDBK4mb84QRQjWJEnuDui41T8ufcoqeZ  8vrKvjW3FAWxHkUu82VSyeJhGsPp9Y4v3g7nR9JrReqh   10%   75066991 (  0)  75066960 (  0)        296765  1.5.19  204434.313218621 SOL (0.06%)
  E8swCwkKkQ6HTMTA6e1ehoZxDKGkXqAM2o1TYgxNCX5d  9JDtYB4CsoRo5V56twWBGfD1ArMJ7JGDKsKNw2bKN3UH   10%   75066990 ( -1)  75066953 ( -7)        298878  1.5.18  204621.727872623 SOL (0.06%)
  EyhATWGrsfmfRZtUpCDiyW8vH7CfkT5gy15RtvNPmqby  A8f4eVZPybix5Ep4W8Q6iAPZ83jo5Y2wWXttwF42MH87   10%   75066987 ( -4)  75066950 (-10)        295560  1.5.19  204783.580081629 SOL (0.06%)
  9SUf5VUSACNWW5CgbLs26hfezhUgeR5pNuTB1ZmUFxVg  2Dwg3x37yN4q8SyrrwDaRPGQTp14atcwMPewe3Y8FDoL   10%   75066991 (  0)  75066960 (  0)        298304  1.5.19  204880.982326913 SOL (0.06%)
  9XQyJ5emdYVDB3PjFDCmXFG3G4cAbPHJttF2hJczhZB8  Bm3rPaD62YWXJxvpW5viF9jUVdMmd7Q2HYA6eTbDhxxW   10%   75066991 (  0)  75066960 (  0)        298716  1.5.19  204884.150785815 SOL (0.06%)
  CsKpHvEjoSkdNduEjb7mfuJbzRKqc5ypde4yUW4iGMS1  6dAvPUPbFKz1rGLTaCDLC3XRcrJKEge6SQgTvthxQXak   10%   75066989 ( -2)  75066952 ( -8)        295975  1.5.19  205160.298587497 SOL (0.06%)
  G4BaMbWsp9uX7EX7dtv848xqfxtmoi6d44cNk4irEtvZ  HhTHWzAFhebkBMMD52bYbQhxJ3e5XK4BDMbiyUjrCVf    10%   75066990 ( -1)  75066953 ( -7)        298611  1.5.19  205416.965487731 SOL (0.06%)
  HVWzcQZ12pvZN726XXgA2jBfDuj6wDFwSFVdzxooi7qx  7DJunpQm9vgLXnspokwjivkdX6UkugT3HZs4prjsPkG4   10%   75066990 ( -1)  75066952 ( -8)        298763  1.5.19  205919.94145272 SOL (0.06%)
  3ScqKCyAKGN4B27S1mFNCCna4cf3ZBZf6diuXNPq8pBq  E3CTBwiLeBzJ7AJzmVnqtDBf1wHXv33uPPg7SjkGKWxE   10%   75066991 (  0)  75066960 (  0)        298497  1.5.19  205950.813607917 SOL (0.06%)
  5hyHG7SnmJAHZmM9shuq6BXxLSG2EeosMH3ZohtaewFM  E1CzrFHxFG2LBQKMjizgSdsgfEejrEJfrvzUGNVxz6SE   10%   75066989 ( -2)  75066952 ( -8)        298839  1.5.19  206157.416427348 SOL (0.06%)
  GUFNuRw9JEAQwrJR71mRa2LbMRyrUfziYUzqY3KQwAXv  CvfYpW8gRQ7bHy9uvuvqy2vpcNmXuBnvaoa2AC8pGHJx   10%   75066989 ( -2)  75066952 ( -8)        297430  1.5.19  206162.864464428 SOL (0.06%)
  Eik59s9go24w5KoyXGoyKSK1rUNYN2MLTv1iW2DFWkS3  EPBGTvcgt8PxPcNFQk6hrFHgKfoNXY9ANkExUHWq8sm    10%   75066989 ( -2)  75066949 (-11)        298604  1.5.19  206496.063948653 SOL (0.06%)
  4i9CE2ewTTWwJkdMvNcia3D4yR5v4AKryae4nxeSULwL  2TmFPDTyCkuEAMQgf4HdEeSqim11oAJfVKzarEEyFUiU   10%   75066990 ( -1)  75066950 (-10)        296593  1.5.19  206791.168454188 SOL (0.06%)
  Hqc2qT3vXvBSKozmz7Rd7gLF6jUJHuQEGBTnCHzEBnqk  AJUWPjhNKgo37ta9ycv9bS3DYAT1DY4NP1DDi8DwLeXG   10%   75066991 (  0)  75066960 (  0)        298151  1.5.19  206812.054326781 SOL (0.06%)
  6TK3XwxN4XMEM9urq1n15CQyF1Z19idts4x5Mfibhtq9  E8ajDvUp25ibh2S5YDpLHABKNJA6JSAuymFGn4zaxPSn   10%   75066990 ( -1)  75066953 ( -7)        297490  1.5.19  206867.983696544 SOL (0.06%)
  9nkbwUu8kVZMPSznrEkroG8bXHcgxPUTQxFbVp2oXYQH  HVbuTNgNQKimdywxkyP8kprJC421RrXv6wyRS566P75M   10%   75066987 ( -4)  75066950 (-10)        300062  1.5.18  206912.885724176 SOL (0.06%)
  Czc6nGs45PPUdm1jA9MkBfdxYRrZbShyN9rNj3P9chSG  4xKms4LsZtdLGusKLdr2haHLGKJ78SLiKcwU8N5GuS46   10%   75066990 ( -1)  75066953 ( -7)        295249  1.5.19  206925.398764053 SOL (0.06%)
  B3SBGpUurKfrMAh3vykyueokU9vE7EiGDkvGFzxokQoa  7N55twSEwaY6z7s4GmvDQwLZBNjCFS2zioTzR4cb895g   10%   75066991 (  0)  75066960 (  0)        295619  1.5.19  206928.419631568 SOL (0.06%)
  F84kSGMUy81sHVF7HtZn8nPMRBau2ZC9uqGAZGJtHXYJ  8znmPrUpUrEuYSexGoSdVMFY8UkD7gRnW5JsgmRN9hSq   10%   75066990 ( -1)  75066953 ( -7)        296165  1.5.19  206936.792581406 SOL (0.06%)
  8CzsSMYcmtFmcGpYYv98kcoQQaWQLVtpQRUzEw6awPHU  AFcbrdKxdr2kS5za9U2WhRGLmGSaxC3J9ZzekM8hnXXq   10%   75066990 ( -1)  75066953 ( -7)        297395  1.5.19  206936.973908694 SOL (0.06%)
  DdgotKX6oyHpadPbAuiyAodHh3mk72SXBL4aeg4vLzZc  9eysRjQkycbokaxQ1R9i7nZ54iFKuVkrYp1HJozPGUpZ   10%   75066991 (  0)  75066960 (  0)        297833  1.5.19  206937.659229622 SOL (0.06%)
  8waGfnuaHHUgZT19xQo8LWsAY2CSrXiUMeMDUUrvkeu3  7vmQeg3tFytF8BSYbC5uEwzgCMs9vxs1s3MhSSi4VJC9   10%   75066990 ( -1)  75066953 ( -7)        296124  1.5.19  206938.47459378 SOL (0.06%)
  AJ6PKuAJmACTpB4tLPm61u5ScygG4c7EtU5HZmCQ7MDA  Cvu9WsGRH7XCiB5KQMzpaGQP3HXxj6pjtbZAjqvFk839   10%   75066991 (  0)  75066960 (  0)        298551  1.5.19  206938.666914569 SOL (0.06%)
  E96CSRSUT1WUGKzDb2Dug3zWRgj6qKffNTUZUihjFc7g  B8FoCVcA5jGm2onDN7R2EZnDnN26BKxmE9WHKRsbne67   10%   75066991 (  0)  75066960 (  0)        297684  1.5.19  206939.193729254 SOL (0.06%)
  72FqmsqNgpgF4XGueM7Ys8X7xd1ZqBzWvUQUq2Rv6bBv  EbYJNamBbseewMdYNDPrSTWCgnXrccgJt3zLU9thAaaX   10%   75066985 ( -6)  75066948 (-12)        296216  1.5.19  206939.958109785 SOL (0.06%)
  G4dd6rLMW5aQEKFcYFHw148T8afjyavMyabsc86zACmq  22MB6hXJrCgubFstzM6eDYDK5mq7w6cYtSux9G8Sogk4   10%   75066991 (  0)  75066960 (  0)        297353  1.5.19  206940.039269626 SOL (0.06%)
  GhuhvJcGcS4USKAt4JQsiYyZvcAj1MNzBhRiqR87wNv8  AwewbWAhhWFybQMPLQRdxwQazTJmxSwwYB4oZLsrc5LT   10%   75066990 ( -1)  75066953 ( -7)        298069  1.5.19  206940.571379286 SOL (0.06%)
  GinTsSooU4QChoqXFvEWnMLsM6iheLmLFFv681s9CeqY  FFqpohintDeWe78z1GoZrzZPJ9vE1DKa5oTQXSv4zeQb   10%   75066990 ( -1)  75066953 ( -7)        297628  1.5.19  206943.405491704 SOL (0.06%)
  5V32oyrRmsnuVdC4Jv178KVBeehDQUuk7U2D1YmeP6mS  J61sYWwTT3Kfkjy3gJ1ViRwtfXVp7Bi89DLqvCp5WDgC   10%   75066986 ( -5)  75066948 (-12)        297871  1.5.19  206943.957578337 SOL (0.06%)
  3xubywCu9F3ALaYRKgp6RVUnQZFf4npNcDZCH4Qjpmmd  GbxaecfPrKMXPDDCHnJaqwnZfAumqWsqA57hhC48D8jy   10%   75066990 ( -1)  75066953 ( -7)        297744  1.5.19  206945.601383041 SOL (0.06%)
  86fVAWdCDyrSHnv9y9dQCrfQCY5CrXptLP7gL5PyUDxR  5hVPfoTZcfZTcyondKxjuVczaFap9pBGYBSPKXg9Jrg5   10%   75066989 ( -2)  75066952 ( -8)        298120  1.5.19  206948.742061923 SOL (0.06%)
  9bLEjf8SjumDs28WqUfoSnyh9Eg2VwQZGMrxxrWhvLWo  FGanR6t8xwDdmkxGp5xoWHvocozfdbtjnFdkXvpy8dbh   10%   75066991 (  0)  75066960 (  0)        297737  1.5.19  206949.718730429 SOL (0.06%)
  9of5uVqC4EYZ4LLJzPecynNM4GsQWiT1WtHfA39VqcCR  GS9YcwyoG3Y3guieVbDEVyneaa9RnwddwmR98L83McKi   10%   75066990 ( -1)  75066953 ( -7)        298699  1.5.19  206950.101835075 SOL (0.06%)
  9UZ3NXk9jALnrhbWz91PXWya4Hvi94jceq5nojxHiKgL  5asX4eKb6wne3YynLZuqMoYGUWtxMAnXhNXL2z9ar2Dc   10%   75066991 (  0)  75066960 (  0)        298156  1.5.19  206950.45488313 SOL (0.06%)
  44Bivyyp6Jv3Wm4RfJdtpCPBmbEXxvyotUoMDg91ZJPo  4PogyEBKix5WGuTTG27DtD7CC3yxKsvKEqfPoTp8w4Pj   10%   75066989 ( -2)  75066952 ( -8)        297829  1.5.19  206950.718650578 SOL (0.06%)
  3omj2Ajqtb7ZjCwxaYPXPrERhKDnwQRFr2ti7F23Kd6A  5JR2Cy76A1wv4HpvmJbELkMucYpx5k1L8c26guxCNJQF   10%   75066990 ( -1)  75066953 ( -7)        297760  1.5.19  206950.959004613 SOL (0.06%)
  2UuVd6BMW97MRnj7mCpq8PhYmqmUYuojvDkqWstwASCm  EVoQv6TSBZfTiLFoc42B12iShLNUYiB9PCc2uxFxHMoV   10%   75066990 ( -1)  75066950 (-10)        298743  1.5.18  206951.666718048 SOL (0.06%)
  Hp9NMFfFhDQJCkgUHLajoWqTjm1xxdvQEyUnoYn1GWFX  FM9TSfBK3Xcdj2KY91hUZveSbiTDCBbaYkDbL1ksdwvn   10%   75066990 ( -1)  75066953 ( -7)        298035  1.5.19  206952.149633829 SOL (0.06%)
  8W6wYH4cQKroY9ymP38DQeC1wV8b92xb4BWT2foXSvGy  7WmXq6EZBGaPqtfPY3hvmE8GQUwC37iRc4saRcyoSFWR   10%   75066990 ( -1)  75066952 ( -8)        298255  1.5.19  206954.832874154 SOL (0.06%)
  DU4shVYTQz4Pf1RXR3ZMNp7Q9AqLK72sx6b96UxEpcC6  9QypR4m2iLGu3Q5tYoM9LqJ1qJyJcteFx16H8Rsai1FQ   10%   75066991 (  0)  75066960 (  0)        298529  1.5.19  206955.32861601 SOL (0.06%)
  FCQTtjeFK7Fj64aVDwdahfyxiG94uLnUjkmmXUq2esuf  BCnjkkKPBT8cegVkFQ15B6TtonqD9d27gM5UUD6imEe9   10%   75066991 (  0)  75066960 (  0)        298587  1.5.18  206957.680847058 SOL (0.06%)
  Bx6R7GBNsFCt7KJUZw5yrbvdhL1EYcXPhkUBnS2t3c7E  GMnqnaM2QsBLn6nCTxJ15DzZKuyc3UZAogCva5DC7w5M   10%   75066989 ( -2)  75066952 ( -8)        298698  1.5.19  206958.484443459 SOL (0.06%)
  9QEY6sXPc88gt1jNEYUC7YWceWV8nRuuUi9WYrDngro4  A8XYMkTzKNceJT7BKtRwGrg5KGgaXcjyoAYuthrjfKUi   10%   75066989 ( -2)  75066952 ( -8)        298262  1.5.19  206973.511514663 SOL (0.06%)
  9YDu59tKDysB9zT9P8cgvaogmuy64R1Nxbr57QHztNrW  2oxQJ1qpgUZU9JU84BHaoM1GzHkYfRDgDQY9dpH5mgGn   10%   75066989 ( -2)  75066952 ( -8)        297932  1.5.19  206982.614705385 SOL (0.06%)
  8D5rQbJD9qLNJm9HyTjFWV93CBc29ozAdGUia4hyMhw   5iJGmjy3msiPKvAfxZSULmQ1AtYYC1i1Pa3mDjzgGweT   10%   75066991 (  0)  75066960 (  0)        297466  1.5.19  207037.590211785 SOL (0.06%)
  SoLiDDAQBEK1LK8apZcZqY7FunqeqDY6nYXMabQZvPB   SoLiDFg9qMi7tFEo4tiLxEVE7mteRyGZkBHJn2YYyjP    10%   75066991 (  0)  75066960 (  0)        298515  1.5.19  207130.96961957 SOL (0.06%)
  2Ce1dmtdnvgDwD1MiZjP9wwGup6j8H7uXGuGMa9uZo5v  GTeoHau1SpLhoiDGJdQpGd7AS3gyaFLgGVG2EHQRHoWp   10%   75066989 ( -2)  75066952 ( -8)        298401  1.5.19  207148.360443317 SOL (0.06%)
  81JyjgUyLbqs2f7iSvrEBwE7Grj3MsQwULMGTxXKnLvS  CkyA3Xe8SGRtXkZ16hbske6zV9qQ2WNk2agewovcfhsN   10%   75066990 ( -1)  75066953 ( -7)        297938  1.5.19  207196.126579129 SOL (0.06%)
  ECd2JogmL8Qr4hox1zUNEn8aHFchE1tAh3JXM4UQBzzo  55d6G3qVzKfZ5ZWQxaExw7rJSZ1sRg5EQfXaJAo6vwwZ   10%   75066988 ( -3)  75066951 ( -9)        298436  1.5.19  207233.098879352 SOL (0.06%)
  7ySQKwtxuvxQkhW8rKsa4Ch83tdX5fzfpd5hu6Av66ur  ESF3vCij1t6K437j7tzDyKspPeuMnYoEtooFN9Suzico   10%   75066990 ( -1)  75066953 ( -7)        297523  1.5.19  207419.023305475 SOL (0.06%)
  DE37cgN2bGR26a1yQPPY42CozC1wXwXnTXDyyURHRsm7  5enTTfG63W4JUzCpwioeLte7827NrYXUgGr6z7Rm7xf5   10%   75066989 ( -2)  75066951 ( -9)        298158  1.5.19  207513.840867625 SOL (0.06%)
  DiFeTctQSaNczJNmZ5121kYqLaBe9wDpM9sjCzTELJLE  FXcZJuQwkcQpw2YrkTaxWJ6rnw4P12wvULttviMRn655   10%   75066991 (  0)  75066960 (  0)        298321  1.5.19  207725.319159796 SOL (0.06%)
  DqBvkYXi7HjdaKz78yakiDsaGuq1BKrQi3Z5JV6STctz  6yShbTX3KRMJLANDfo8Xo4aHYCjepjUbna9Y64mdQB1a   10%   75066990 ( -1)  75066953 ( -7)        298878  1.5.18  207800.965397053 SOL (0.06%)
  9z7sdEnttp9T9bzoRZumMcKWCU76RdmrFPi42km7Twb8  CGj63fS6gJXm8KvaUkkL5yAZvH9cjeRgHCgJwcZZEsRd   10%   75066991 (  0)  75066960 (  0)        298421  1.5.19  207922.311998271 SOL (0.06%)
  8Ey5FDayWYgJdVoquScT2hJDKWk7nQQfqzfGBt1emJpx  2YnL16L174Tj8awY9FfRfR5odpGNEbLnx3dM6CRt9L81   10%   75066991 (  0)  75066960 (  0)        297573  1.5.19  208021.699588227 SOL (0.06%)
  JDScHrbfa4DEmqA4j5JTVc3wtwqXXeDCrjntRo2pWoLP  3ywgnXHYqMZ9LAYjXxHPMxMHMQ1EDom4Pcm8whin5JPd   10%   75066991 (  0)  75066960 (  0)        298869  1.5.19  208112.092112992 SOL (0.06%)
  JokerEfTSznB2aTmowy4QPqjyajLMuYM6Jd4TDnKPNc   JokeruQoFrVevoPR5QBRPGncPQbtrpq3PJEcY22JefC    10%   75066989 ( -2)  75066952 ( -8)        298604  1.5.19  208175.615085049 SOL (0.06%)
  AgG8obWYeVY6nSkPYqDHXfssxdcG68GkuBikkearYRv1  DUxWarcMsNYD5EKj3mchf7ssT9j6h4Bdw3VFJi7QxDry   10%   75066991 (  0)  75066960 (  0)        298235  1.5.18  208211.194535378 SOL (0.06%)
⚠️ AFNiQnEvAqjFJizDtSAh5ypdXvQDRxHN61kcect5wNEx  DYGFK6DZZ1p8AnA48GUD26ygUUpQRKgNQuMHxrhyot3A   10%   74953374        74953343              195676  1.5.19  208296.818121493 SOL (0.06%)
  35AidZv6XxjDg5NNPaKUdzUxv7C2Qi9k1tF9vhMgnHhR  Dyuwspj5UMn6tKPLdHH9JPQShdhJSED4HGdAQ7wm7uq2   10%   75066991 (  0)  75066960 (  0)        298699  1.5.19  208338.103789847 SOL (0.06%)
  HPca8Y8aMnkrTi87MJFHDvMtJVkvyMxFvkuLsoFEoELy  6SHmAJwydJ9pZrbC3DY9hZ6WQPcjWsWbTSfPhLaw1UFr   10%   75066991 (  0)  75066960 (  0)        297695  1.5.19  208387.965634598 SOL (0.06%)
  BxN7s13iMFRCZEumBRzUpH9E25V77dL7VgSig18MYKbm  3Ueg3qrAVv95tJzTiKM2dd33NswZT77yRf9wXcBDCn2c   10%   75066991 (  0)  75066960 (  0)        297964  1.5.19  208415.813871446 SOL (0.06%)
  cami5ixFFZD3jLdX8Ef5tu8o21reSGoE3GpGRrQyP4z   4ysUQbvWDjUdRprcjaBEH4V2VqPJgrPSwRsS6ATTMbiB   10%   75066991 (  0)  75066960 (  0)        298102  1.5.19  208422.322580966 SOL (0.06%)
  2pRZBEB1PX12cBAbgh58VfzTPV4ePJ3txXeDTM84ejbZ  CDVkS2zVVHqiADfap8W3Mb6ZMm4KUKg4igE8axtHqFKT   10%   75066991 (  0)  75066960 (  0)        296558  1.5.19  208531.296761014 SOL (0.06%)
  8DXdM93UpEfqXezv1QTPhuA7Rci8MZujhsXQHoAsx5cN  9goiZ54nbkF7CyUYNFfzRAgPjmSDARxLSBEuNnSrunn    10%   75066991 (  0)  75066960 (  0)        298981  1.5.19  208547.781466659 SOL (0.06%)
  CBUGET5PnvLc3HvEeFYj64iTvdKhYV6pujTPDdDh785K  GuxBSrv5jnSwwPepkqnmkM7YCBSakKanbnw4BKMdda4j   10%   75066986 ( -5)  75066949 (-11)        298655  1.5.19  208551.113979147 SOL (0.06%)
  Agzs33CozCnr3eFvyHcYkb46nUZq4dL9SFMV9CNAxfGR  BL1CZTB8nZShHexKt2jWmHJe4jShRSbRn71BGhmYSFc2   10%   75066990 ( -1)  75066953 ( -7)        298942  1.5.19  208683.652491574 SOL (0.06%)
  9mfDCev5UYzN15XS2AyRPXbcrDJ3HL1jkXj3mdP96Kam  DuGCGy6inpKrECrBMWissxMQtqgAy8EYjY6CgjVk8g1b   10%   75066991 (  0)  75066960 (  0)        298947  1.5.19  208725.442050018 SOL (0.06%)
  GmZ7xFQ4GHDbQw4CSnFT9pAHa15u35qTN8e259UrBh9D  8EvEr9TchMG24F7kSE75Qdw5fcEe8qh4anVSKwj9BVfJ   10%   75066990 ( -1)  75066953 ( -7)        296235  1.5.19  208759.357394929 SOL (0.06%)
  AHg5MDTTPKvfCxYy8Zb3NpRYG7ixsx2uTT1MUs7DwzEu  9SE5oHdQ88rVFPcJZjn7fNGSXhU7JQfZ5Vks1h5VNCWj   10%   75066989 ( -2)  75066952 ( -8)        298882  1.5.19  208763.145143457 SOL (0.06%)
  3CxJofVghT3nbqtrSWo3oMQTFynpNyqtytBFawpAXi8E  C45RHYsHWCdeaLJ2x1VfsKH6PgFBbuH6XKHywRTgJCtd   10%   75066990 ( -1)  75066953 ( -7)        298311  1.5.19  208851.518995514 SOL (0.06%)
  HQATLHBJcU7DET6b6NWNMpfsZVZ1PRnbfkCtcFCT5reo  9Tu3yadiFa7xFkoTuurz1qh5FzTehn5aQ6m6C5kebQMQ   10%   75066989 ( -2)  75066952 ( -8)        298034  1.5.19  208928.462990194 SOL (0.06%)
  2tKR4mX7LzhjfdNsR6HfBaDDh2RM3wdpUrJqUU42aJTc  E8DxKVZ86hngSiBRMhSpZP5vdkUAEyZGcYAqrFBmzjZY   10%   75066990 ( -1)  75066953 ( -7)        296178  1.5.19  208945.097850822 SOL (0.06%)
  4Ko2GufbWc8hhDD9GL46He1Q1Z1s6VFfBZS4RX9yY5Rt  BGfnRRnU5zkZrSrkcNTjs54U2o9TahJ39hGrNqnXnijt   10%   75066989 ( -2)  75066952 ( -8)        298514  1.5.19  208949.759144836 SOL (0.06%)
  H7tpxumDErX9DXNTKKBFTDmynXBF32QQ8n3QjiQfLn2i  E8LaCcF1nuanxsXozu6eULNw32M1joBja9QjpcLiHYxe   10%   75066990 ( -1)  75066953 ( -7)        296740  1.5.19  208975.59913821 SOL (0.06%)
  3BKnjJPKQrC7zw7X3KtTKVtZtjLoLgzeJ9xZoAtZyVci  AkSj1eriN7dwVYfvdBmWp9txDjBv8kmgXe9xwhH6dABU   10%   75066990 ( -1)  75066953 ( -7)        298458  1.5.18  208988.003968731 SOL (0.06%)
  7ek3CDbxpGRdCVTJJpj6WPHmZrJqBCQ2RBQoqLHitx5L  3Dm1Adr8LzNsb1QrVmTxgdgqC6tHMiR4mw2fY2t1ZfBu   10%   75066988 ( -3)  75066950 (-10)        297771  1.5.19  208988.266672159 SOL (0.06%)
  2oujYrRmtDDTF3b3JUgsZ34TkcyrozMjgRHBQE9R6K8i  C1udGNjscZYuFgngT9uY38kTDudBGuNpPcmkGa5mWizG   10%   75066983 ( -8)  75066927 (-33)        295952  1.5.19  208992.983821448 SOL (0.06%)
  79BLeiBD9r6okMwEPW3zHAviUJFwCLjTsK6HTPBwrN6f  B1rr3WM3fYZMaaUQ8SrMzdUojTWExubWKikL5CJgeGdP   10%   75066991 (  0)  75066960 (  0)        298543  1.5.19  208993.884759048 SOL (0.06%)
  5GattKwm5oBpDJGgdL7ZZfCMthoETwrZFmQMDK5eq86a  3WrKhSfqEn2nrnsXaSi4ohUUdjE8btXkJ1BycnHizrYx   10%   75066990 ( -1)  75066953 ( -7)        298225  1.5.19  208995.502039093 SOL (0.06%)
  9NEsaMNVEEPHCdgxUzeziZdgx85LS5za9LVs1rQDGBdC  DLMhVf7xXa8R2D2y1fZSRVQwXdup7EfwBFbttzn5ppnC   10%   75066991 (  0)  75066960 (  0)        298398  1.5.19  208997.81788599 SOL (0.06%)
  CGoT67sSerk9ckwin6yY4mna3ymLxxKNqdYeQZ9xNdkD  2BGBakG9kjmZAaygVmAuUrBSqBZt8p5FVabbStxHEXUj   10%   75066991 (  0)  75066960 (  0)        298703  1.5.19  208998.844120006 SOL (0.06%)
  HsGWhcLqVsue45i4vMxku3YEVGF5cYxUSTaa6gwieSGN  BUyTfHHrp3HQ4TQXaXrRrNuhLsbz23auKYNrxkQRfViP   10%   75066991 (  0)  75066960 (  0)        298433  1.5.19  208999.314443549 SOL (0.06%)
  AU8CBLxah7B3HQE4rtjfkRDmqEGLqu1xKUPLSytQ8tLc  5dCwkc2Xk8fxJJK8LF7sSGPjtcWMBKMxwRkApgrHPxzj   10%   75066990 ( -1)  75066950 (-10)        297327  1.5.19  208999.964466591 SOL (0.06%)
  8X4SDANraPWJYCDB1ZYCucraB1TKki7vTFcJMiSrspTY  F7oKL1VEeL5YGNwyFZABpiBiT6i6N11HYJex7jGKSpeZ   10%   75066990 ( -1)  75066953 ( -7)        298419  1.5.19  209004.149442912 SOL (0.06%)
  AfDqsQKrpuDEHy65fsHMzVVeh4FcsRomxoe5nDLPERE8  2mgRVHFwXXn2pGgHsVefzUrGofaYujbSvsKbdeGRYmLz   10%   75066990 ( -1)  75066952 ( -8)        298490  1.5.19  209008.91209685 SOL (0.06%)
  AVUsR5RuxBcZDftuUNQqwUKwx9puBE5N1zmgLQZSxiuH  FJZSoH7f3MdQrnrpYzQ3sPpNv6wrcndgsmFYtgNJkVj8   10%   75066989 ( -2)  75066952 ( -8)        298295  1.5.19  209040.526621239 SOL (0.06%)
  J2jV3gQsvX2htBXHeNStAVvMJaPe3RgNotwfav9pyS6y  Dz4aYDW7NvtDQXB62gcLWaA467zcYtnLiSgyiH57ZwUF    8%   75066990 ( -1)  75066953 ( -7)        296277  1.5.19  209044.382823965 SOL (0.06%)
  AqGAaaACTDNGrVNVoiyCGiMZe8pcM1YjGUcUdVwgUtud  85SzhUpCkRKcGryNC4YYYBVKZU6VE1QwovfdxnseaLow   10%   75066989 ( -2)  75066952 ( -8)        296017  1.5.19  209069.591682759 SOL (0.06%)
  EjcGVYigv2PA6MeZxRmgFot1P7eGQThFMS2Yh3cj5r2n  Dn2cRSWAfQpb3NyUJ2q33t1scBLxzo8TZBAyKsWhX7zh   10%   75066991 (  0)  75066960 (  0)        295779  1.5.19  209082.384162503 SOL (0.06%)
  63sJMcjh8r7W1oXjrNZrG4nc4UX6cQVonUHp9vGDUNdW  FkK9cat6c8zhLXeYFn4UTvy6fnSLB4p6SCbRQQfdFkkP   10%   75066991 (  0)  75066960 (  0)        297921  1.5.19  209097.341204666 SOL (0.06%)
  4R8aXMwJUdE8WWtM7yE7sPFHJBfPBRd5o7ERVeQE7DKB  HNDSUSqBox85bbAL3eFsiDj3hKAp3dHioB8sn7uWojKH   10%   75066985 ( -6)  75066948 (-12)        298317  1.5.19  209187.167216706 SOL (0.06%)
  1EWZm7aZYxfZHbyiELXtTgN1yT2vU1HF9d8DWswX2Tp   HG7a8fgjTkQhGFTPukbTdf5FCwxVVjKzkbo6ToNswTXH   10%   75066990 ( -1)  75066953 ( -7)        297818  1.5.18  209191.86872599 SOL (0.06%)
  7R3t8NAU7NDAd64KLz62tj1TaFZLmSyVSMRJX8VT8Agp  GQWkfg91PsW6A7vPVCHN5SM5umBERMKjdyqzhB1nqQTB   10%   75066991 (  0)  75066960 (  0)        298746  1.5.19  209422.890448636 SOL (0.06%)
  3m5QHq24vbPVhtbcHbaDjvij58WExvfy16UgcCPLfDXP  A3MC4K2pxLXTEHVN5HFF9ikjiauGP7ioZws9FYsucAWF   10%   75066990 ( -1)  75066953 ( -7)        298580  1.5.19  209739.933196502 SOL (0.06%)
  GAvEvUKfVkGjMgFhQjC6WAtzzM6YbHGvfC4PoCtrrfVz  AQNEbXjfsvR8wSTag5GRTBeBXUNw3oCndjHV4grdwhqV   10%   75066990 ( -1)  75066953 ( -7)        298416  1.5.19  209818.662197037 SOL (0.06%)
  CwhdMezLucz7bcuWzStpLXgrzKGC2tBBiaVmJZjfprRN  VNbW721iu6uVkrx246N2BiQth8u4b4SCPJwH3JvUovD    10%   75066990 ( -1)  75066953 ( -7)        296234  1.5.19  209856.606575972 SOL (0.06%)
  2Joxdac2pgGA6xqBCChWKZNKhWgnv6vkuEkGp14JFPzT  GwoUPSb4cyess6AAxaZ823jNyMEE3MJZKpLtGdbWbMfT   10%   75066991 (  0)  75066960 (  0)        298178  1.5.19  209975.374993591 SOL (0.06%)
  7N46n4N3tiX4AjVKBDPfAuxhgirAgDFknn5pkjYj3xz2  5GNvvKabvabkBeFRoBrZEBbBXY6VDEYtzs4B84xH1amB   10%   75066989 ( -2)  75066952 ( -8)        299063  1.5.19  209996.666340286 SOL (0.06%)
  Diman2GphWLwECE3swjrAEAJniezpYLxK1edUydiDZau  voteRnv6PBzmiGP8NicWtQiqEJTwKKq2SxtqtdLUJjd    10%   75066991 (  0)  75066960 (  0)        298625  1.5.19  210267.420565421 SOL (0.06%)
  GZNnph4EvmyjjL5uzF9xNNTHyV46RzbkW4w4HYU8BQCW  6anBvYWGwkkZPAaPF6BmzF6LUPfP2HFVhQUAWckKH9LZ    5%   75066989 ( -2)  75066952 ( -8)        298958  1.5.19  210327.245569471 SOL (0.06%)
  heoSbe83UDZA5LNR6F33QHVk9iZXpxPKcNRtVpDczXL   FgdMXZmQAy5SfK3TdGiXEwb1ZqBSp5wJunUqKDwiEZPD   10%   75066991 (  0)  75066960 (  0)        298971  1.5.19  211055.593489285 SOL (0.06%)
  DAHJgPKdmncYW8DmY6meaU953a7SktQ7eDGtWduC8W8m  C6uqzABsRPmFd14iL9Ej36AbddVxXPJWV6jwbLZYdYJM   10%   75066990 ( -1)  75066953 ( -7)        298227  1.5.19  211229.359276608 SOL (0.06%)
  Cu9Ls6dsTL6cxFHZdStHwVSh1uy2ynXz8qPJMS5FRq86  5iJDEVRi1nMLwKAWhYbEokZnvBAe15rgFaHGkggVEP9z   10%   75066989 ( -2)  75066952 ( -8)        296489  1.5.19  211386.902750707 SOL (0.06%)
  JCZNdppKyur55JqY5VBctkVjxEcgoQq3RmBuVbQxSCEq  39ueFBsyhKixjBqNJ74L7DackgeZyBALqjiP8CsBhgEc   10%   75066990 ( -1)  75066953 ( -7)        298009  1.5.19  211417.262486249 SOL (0.06%)
  914RFshndUeZaNPjf8UWDCyo49ahQ1XQ2w9BnEMwpHKF  4cCd4SGrMswhqboYBJ5AcCVvCjh5NtaeZNwWFJzsnUWY   10%   75066991 (  0)  75066960 (  0)        296252  1.5.18  211842.631810344 SOL (0.06%)
  2xte5CBkCBEBLNviyAfvSaTkMy6tvg99Cy3XJj9EJJs2  EFEKcHrUBsRoQkuTSJAQNZnj1u8h9oE4LoCYerednc3F    7%   75066991 (  0)  75066960 (  0)        296896  1.5.19  212786.802182315 SOL (0.06%)
  9gFxqsXbFyrKXUkqpAatonn47uYZ7sEZSnMxhzQoXrUJ  48oxpSHQkM4sdXUY9NQ8KnEtebzZbyk8uUT7JRdVQNuf   10%   75066990 ( -1)  75066953 ( -7)        292236  1.5.19  213436.508195681 SOL (0.06%)
  pZ2nxiW1M78Ez9Tk5DcQyw1nWpmPFVZk8KDGKWbvvUV   36z2quTJD9JPAbvTgecCXnhgjJkreA2F7SHH32vXFExo   10%   75066991 (  0)  75066960 (  0)        298141  1.5.19  214890.32410499 SOL (0.06%)
  5rNErxSxMj3WysMx8bC8vHkbrt9QmwMeG9H6aTp71485  7ZJHBz4SzQw1GDQZRaCMLvo7uoUo7cY1BUioA7mdfD65   10%   75066990 ( -1)  75066953 ( -7)        298607  1.5.19  215112.906547498 SOL (0.06%)
  6GRLDLiAtx8ZjYgQgPo7UsYeJ9g1pLX5j3HK97tFmtXb  7gfSX88DYoScBCaFqNxbsgFeZom8Qotnf747iJLhhJC6   10%   75066990 ( -1)  75066953 ( -7)        295453  1.5.19  215358.235128024 SOL (0.06%)
  GwnQsVbbVsMhGqWV3gcVCF1364LRmftggyc5SmsYMLrY  4n4KiUuRwgAqdTh6ag8WRy3ibL2gbWTVJPfEGw3SAsHx   10%   75066991 (  0)  75066960 (  0)        296059  1.5.19  216134.121205153 SOL (0.06%)
  4asJHLR6DbifuQK2MRGNx9iHujYDYkQ9pqQXsmbc7fyD  DgStbviwdrLoVnRLrg9LSoUYQ1XEcwWyXUEHMfhQiZem   10%   75066989 ( -2)  75066952 ( -8)        297208  1.5.19  216874.378184387 SOL (0.07%)
  3vYPCtncFxQ1RVtSpB2HRg1udHfeVPWPpWALuJaMcLx3  6ywtN5KvEmTqdsynVraiUWGD4pxr3WQEKrWQZ4f9c5Xo   10%   75066991 (  0)  75066960 (  0)        297318  1.5.19  216875.606119653 SOL (0.07%)
  DSV16KKYW34qjxTBcTzURTyJYgw7qLq4VLKq3UavQKCk  61dX6srk1WmiGtd9UM4SHttUDzfWrY8zXg4h8gSmhYMq   10%   75066988 ( -3)  75066951 ( -9)        297546  1.5.19  217149.512398339 SOL (0.07%)
  DAS6zKbzVe5DFSwyEgo1TisuWt31HjBdZxzgBF2ASiju  7LtXiFCXqxwntUkTfzMAmS8TTehK8m3cbzx7BgvYFTiP   10%   75066989 ( -2)  75066952 ( -8)        298877  1.5.19  217157.417433668 SOL (0.07%)
  E1SFkvPjU31xWMcvgnX6vhGvfRvb1zXvHFkqmQNEGZKK  HgovzvawYMRPY6Go2kaQF1vLR4rNixooHKrUWSH6aW4C   10%   75066989 ( -2)  75066952 ( -8)        298481  1.5.19  217176.991283632 SOL (0.07%)
  94HgFvsD8zm7UXJ5KJxN4zW5nsdhZWY2LFREjfaFEHwR  J6XZBDuUpuBzgcVGmY8NgHQYKbGZegoYmbpadAKo1qRw   10%   75066989 ( -2)  75066949 (-11)        296227  1.5.19  217256.412144893 SOL (0.07%)
  9nEBir5mYz5yAnAtuZsuBxwfC7SayWZTMnyCmmTsUknf  6HNqJpARCvuabtvJFWXbTedMrRpZHZYd7MgYmuFhcbXM   10%   75066991 (  0)  75066960 (  0)        298818  1.5.19  217365.396605977 SOL (0.07%)
  AaDBW2BYPC1cpCM6bYf5hN9MCNFz79fMHbK7VLXwrW5x  DTWh2VQuUJmskHuZWy1KLNvzKqk7PeMkCoaNvZy415U7   10%   75066989 ( -2)  75066952 ( -8)        298271  1.5.19  217412.507631936 SOL (0.07%)
  CsqwHrSUAWsWpN5kUcjcv4Nsq5JtgFiDJTckaY1zuXPf  8Dfm249TXe9Wd3wYEnpBdSme659gecoEMH8zx1YWN79v   10%   75066990 ( -1)  75066952 ( -8)        295249  1.5.18  217775.068651752 SOL (0.07%)
  6uHvWHjBYiMowwKUZ5waAc83x5bnLbmGfxDnXPVQYorh  2jvDEtiywY9sjx29gCAQjvxD5gJLcw89NVhkJ6ybajZC   10%   75066991 (  0)  75066960 (  0)        295124  1.5.19  218108.663380241 SOL (0.07%)
  EZUyrtZoBWMMoXTLzDo4RVTXi5XKYCs1kq7oESFLDvCP  2PC2DCk8C7n4jhXA427a1qL9CFRu7PtrzKHixU6LXocN   10%   75066989 ( -2)  75066952 ( -8)        298571  1.5.19  218620.047862685 SOL (0.07%)
  ExnG6VdMubFJfrB7qereo3YxUMyeZvTuZUVKfv7R1YZD  EHhxJpoAZXSSmUBhUXUMNd29verm4WdmaHQ2eZtU7fWR   10%   75066990 ( -1)  75066953 ( -7)        294678  1.5.19  219034.158099543 SOL (0.07%)
  AWHfhBSMN7rX7NEDrKNfjnUyR7W1xpfistTy6w9GqT6U  5PKBzZEbwGEXhQj47eiwH6RHAPSCxcVhdYcFttW5gLWQ   10%   75066991 (  0)  75066960 (  0)        298261  1.5.19  219240.950806565 SOL (0.07%)
  HwdfNWCqP2vXRvaHqQhoVUM2uPndaY8DDJzzBxCoPNHU  EzZEi48kvDWPJJBM1z9nmjpgRyMwWJwaa4xVW2U6psPr   10%   75066989 ( -2)  75066952 ( -8)        299154  1.5.19  219390.474196778 SOL (0.07%)
  2KnfYVJiAtxiSPfHYRzTsGGhUAhDTNrJXWn7n6K6giJU  BUTBf3nPUdD8zEKSqZgpHzYqpzPc74ueXaGMBkVAe19U   10%   75066990 ( -1)  75066953 ( -7)        297849  1.5.19  220639.979904344 SOL (0.07%)
  narPxmKTwkUxvcXhueccHT8xbE8og2Vb7NrLBm8kcrh   pgbB8erEYkTBKxyuXJRB4HrwtgyKYqeGCQnuPAH8yVw    10%   75066991 (  0)  75066960 (  0)        298515  1.5.19  220726.166707152 SOL (0.07%)
  EEP7VNrtLt3tjM8DvqUVEsmwZxLKwHgEFWDtoMx2PQZB  3isNAiakCHwSrZEWyBEccFtejydwAbsdbWzTp5FkXDAP    4%   75066989 ( -2)  75066952 ( -8)        296941  1.5.19  221308.739886585 SOL (0.07%)
  7aR6AjK87ehUxwEJrPpnoizUKtxJKxqZCQCEGD3h1Xgf  B1UaJwhZ9rmB6rUwCikMCkYEPVgpGYgmmrKqfAHCZanw   10%   75066990 ( -1)  75066953 ( -7)        298201  1.5.19  221874.099696079 SOL (0.07%)
  DeCcxJU8AqxPRZ27wrKYKRgmNHwCUK7MvW5XPGWh8WZF  AqSThC5LAYcfiPM73cdwDfgrCvDKHXe1TmCpUE5tnmSs   10%   75066991 (  0)  75066960 (  0)        298897  1.5.19  222754.517664049 SOL (0.07%)
  BZBKHmW1DhBaAPojxWBQ26vGz42Y7MtNviFZWpc6nGLb  AGXZemZbyZjz5NBhufcob2pf8AXnr9HaGFUGNCfooWrB   10%   75066990 ( -1)  75066953 ( -7)        295388  1.5.19  222979.196716923 SOL (0.07%)
  zeroT6PTAEjipvZuACTh1mbGCqTHgA6i1ped9DcuidX   5BAi9YGCipHq4ZcXuen5vagRQqRTVTRszXNqBZC6uBPZ   10%   75066988 ( -3)  75066951 ( -9)        299035  1.5.19  223111.175332042 SOL (0.07%)
  BFMufPp4wW276nFzB7FVHgtY8FTahzn53kxxJaNpPGu6  7K8DVxtNJGnMtUY1CQJT5jcs8sFGSZTDiG7kowvFpECh   10%   75066991 (  0)  75066960 (  0)        298898  1.5.19  224398.807251963 SOL (0.07%)
  2VA3q6DbiLjbrLgnkiZ2fdyuRyVBkYRgqBDwA6qYiSDD  g4MMQmzT7182jBvyUebMvsgvNM3RVrVFFJd9tRjWXhf    10%   75066990 ( -1)  75066950 (-10)        298220  1.5.19  227042.759542334 SOL (0.07%)
  2mDrrmhSzpSyaF12izGk8hnFjtKCGeCFPwQHpRiJDby2  9wQQnnnkk5b5GkQWTW9L4kEA3CjFv6CqsQd5gt6tRsHK   10%   75066991 (  0)  75066960 (  0)        295924  1.5.19  227319.154793336 SOL (0.07%)
  Fa17nmHFt62kmerRQNGtgVWDxnuf7UD3PY2eeFfhpz2t  E3yhPs5PPN4RZh8FbJo2eqtdrAYKCK9H7pcSD1vCNCP4   10%   75066990 ( -1)  75066952 ( -8)        298107  1.5.18  228118.053959122 SOL (0.07%)
  J5MroSHMzcPUTXcnpXygeCFnXJ39NBG5RRQRYD29ZwNy  CYqN2y4bGWXbJUUxVNGMxq8bQVZCQYE8T2Ryfvra17RL   10%   75066989 ( -2)  75066952 ( -8)        275407  1.5.19  228322.953940788 SOL (0.07%)
  12oRmi8YDbqpkn326MdjwFeZ1bh3t7zVw8Nra2QK2SnR  7vJeGJzbaa3bghTsPsFi83T6VCy5uLaQ12r32JA6s8xc   10%   75066990 ( -1)  75066953 ( -7)        298410  1.5.19  228744.420448541 SOL (0.07%)
  HoXANZnWTGeePertqWkMEnnhgXjTjzpfWaT2kja2ZgVU  4dmBLJcchyfucSxkVmrymmsnVpZm3rBkp7kbF7qHTTYW   10%   75066990 ( -1)  75066953 ( -7)        298202  1.5.19  228948.928400391 SOL (0.07%)
  AUa3iN7h4c3oSrtP5pmbRcXJv8QSo4HGHPqXT4WnHDnp  5nps5qfhJuzMQvZMDi5knRMdsDKw67WR3JsEoNSN4DDX    9%   75066991 (  0)  75066960 (  0)        296998  1.5.19  228976.033843473 SOL (0.07%)
  Bb4BP3EvsPyBuqSAABx7KmYAp3mRqAZUYN1vChWsbjDc  13HNYUVBVHgJSfNKvgXgKia3bywzXabGzQjFyMQxLMjS    9%   75066990 ( -1)  75066953 ( -7)        277421  1.5.19  229104.675651101 SOL (0.07%)
  CyyXgFh1cQsrrqUvwDTWFXKFMnSe2H7H4VgX6uEkfBL8  FtsqyUJbftuKnXEicPt6Kq6HgdTxoJw4Cv3sPmsK5ooL   10%   75066991 (  0)  75066960 (  0)        295400  1.5.19  229359.164982703 SOL (0.07%)
  FAwRZwJgi7h81ZphYhLauZKvBHvkr3Dbhh6R8DsaD4Xv  7uwo6mYWMRvPhCG1p1Rsc7iuZsv9cN8Gas6n3cLJ1w7b   10%   75066990 ( -1)  75066950 (-10)        298651  1.5.19  229393.860797398 SOL (0.07%)
  77xRWv8Z3kaQpD9K3Den7YWJ7sxsf1KTnw5MdcM7Gtnw  HSV5EHECVyJhDFV1uApzXbU7uK17Z1FXcvtJvHyi2jXC   10%   75066989 ( -2)  75066951 ( -9)        298821  1.5.19  229403.418107331 SOL (0.07%)
  2VAofLE2bYNM3ZPGfn6dJh2zqtnmNdD1bGqFet8mVYFN  4nCK62aZ126pC1krepWXXxgBJGgPUGEWni2iCABZop81   10%   75066990 ( -1)  75066953 ( -7)        298941  1.5.19  229789.275168558 SOL (0.07%)
  BPX9WVx1MjVpSWY9fo3hGXfE74yKXcreS2XYKjpiyJfZ  26WLNWKrYSyPkQF2NU9LgLSRXGiXWjhPnvspG8ibFh63   10%   75066991 (  0)  75066960 (  0)        298850  1.5.19  230017.510836234 SOL (0.07%)
  DTF5DM36Jc4vkVPJXbSm7wRLe6eeX1UWo72WQhxKJpR2  FqUfeiVYd1LMNSDw1t9WE6fAXAKzjUVHf8YzZXWQjSMd   10%   75066990 ( -1)  75066953 ( -7)        299043  1.5.18  230434.739323914 SOL (0.07%)
  67joanjyAoVmb9nZLyX8p3Gx9tAxzXaUgHDe3kaUH4wf  9vNLTLdwAdHw8EdiZrLzDtUpm3hp7E2MGaNi6SmdfBGz   10%   75066990 ( -1)  75066953 ( -7)        298815  1.5.19  230601.904096108 SOL (0.07%)
  CYkkaM5KwoxaFtZcximkm1DFYnABdiUUUvJg1WURDRsh  AddV9GtVTtepbNa7MBNDx81wTMRgnXf8nzhRuyinVab5   10%   75066990 ( -1)  75066953 ( -7)        297775  1.5.19  230881.218808262 SOL (0.07%)
  5p8qKVyKthA9DUb1rwQDzjcmTkaZdwN97J3LiaEywUjd  EsEtxoyhFTgfvudcy2VwwQJ1qA6BScLUW39PKpYczuxF   10%   75066991 (  0)  75066960 (  0)        298608  1.5.19  231629.405857382 SOL (0.07%)
  EqxCV4fz2unNzt8ydGrVyz24ngkH5n13x2wDSJ8DY6qi  HpeyxYuEXXdB7Xx58pWN6o6aKdw6mxSRBHYAZpXsdkpS   10%   75066990 ( -1)  75066953 ( -7)        294186  1.5.19  232332.27530884 SOL (0.07%)
  ExyEA6EegthLVNEjkj37FDGUnokPHqtEEe58ncBAww7u  6JqhxW5Jt9q6ScUxaJNVGAHskCYYkA2mJpDiqeDPrUQi   10%   75066991 (  0)  75066960 (  0)        296324  1.5.18  232421.54518586 SOL (0.07%)
  3xGDUST5CKZsiW4L5PPp9E63X3PdqRvCJfDM3q9EYcB4  7hPeqN1TMeHMdpkNDkrnQE13nru8RfVcScrgEHHAPoS2   10%   75066990 ( -1)  75066953 ( -7)        297574  1.5.19  232430.336772796 SOL (0.07%)
  GKu2xfGZopa8C9K11wduQWgP4W4H7EEcaNdsUb7mxhyr  DDiyGnCTNq9ZPeXXGc9ogTFHACXq1KUEUrmafWiaWiho   10%   75066991 (  0)  75066960 (  0)        298209  1.5.19  232430.791170579 SOL (0.07%)
  FZktwGYwu2JfbECgs4ELaXXJtz9oZyJEfiWa234is6By  7hn1WZh1YbjnJSMyPnWAVt31RmMUcBDVVBXUq4aFgrmy   10%   75066990 ( -1)  75066953 ( -7)        298125  1.5.19  232431.555353149 SOL (0.07%)
  4K5SSpWHqTbx5N5Ytjj9iWXNd5zZEZ6fkwjGA2KKafgv  6ffMj7zE7u9PKBzQHfv5nsHEGvDXZkntLnFeXRRudEEH   10%   75066991 (  0)  75066960 (  0)        297806  1.5.19  232448.92058401 SOL (0.07%)
  9yWfnPBofnWjXRvfEKLioMb4VrCkwqcYBbWj1uF9P6X8  2kQZfvm5tqcBhXnscT3xe5SbCDttkipxgy1wCqhzqL2a   10%   75066989 ( -2)  75066952 ( -8)        298005  1.5.19  232646.947265921 SOL (0.07%)
  245B9WFHUGuWycSXHagHXwsXGcxDkNYfxWBaeh7vAHDU  BfF3vg8UFcwTkFJNgQ7KQTAZBUWegF3spPm79d71WNwo   10%   75066990 ( -1)  75066953 ( -7)        296059  1.5.18  232655.666294464 SOL (0.07%)
  CVAAQGA8GBzKi4kLdmpDuJnpkSik6PMWSvRk3RDds9K8  XBtfuT5gYU27UAukT3pEzgiKgHpHNQhSoa3zX2PYtiT    10%   75066990 ( -1)  75066950 (-10)        298244  1.5.19  232927.179602069 SOL (0.07%)
  Dq3piY2ZcBvNN84j2EhDLtTzRAw95za7Eau89pNcmSd5  2PEyBgsPYBQ8pMdXQtEaPGNqWQHE9GCnmV2tTVN4GMru    1%   75066991 (  0)  75066960 (  0)        298594  1.5.19  233519.255564076 SOL (0.07%)
  BGBxrNWZ9HDBQoXA9Zq6t4auRXa59qrLQkiTGtZK6FU9  462gkiydfX1ks7bzS71j4k5uLZaonTzcz5mfJpgLGe9k   10%   75066990 ( -1)  75066952 ( -8)        298755  1.5.19  233544.712765222 SOL (0.07%)
  2PotfnmMDS2mdzoggBnNhNVu5NxZsQB8RaxjbDRPJhSA  7z1hAsX5MoyDLmsDvqPKBPeh8ZRdvAR3dqXHg3TpcBkW   10%   75066991 (  0)  75066960 (  0)        298854  1.5.19  233574.34914826 SOL (0.07%)
  rusx3KV69WGvsEbWa2HxjXp9GfHpjojM94BqsnfxKhx   mitDy3gnfY2kcvvrHygv3qsU5i6QqNve3cw6LDibRS2    10%   75066990 ( -1)  75066953 ( -7)        299092  1.5.19  233844.521476301 SOL (0.07%)
  2LhJjdjNic2BcsRLN8opwd9KZSsabebQMnxj2PkU8ADQ  6mCs4Gi8L5RQ4ewnabmwCcVDQLBuvyqnaRcm5LGTciyz   10%   75066980 (-11)  75066926 (-34)        297686  1.5.19  233869.757243351 SOL (0.07%)
  9Mo3ap3jpuqQpLi75EsiXLWfTr1cbBhrJNumoq1wnVp6  9K67smfw8hDXmtnR7uvP62WrKUcB7oZ8cvpsB1hmrCzf   10%   75066991 (  0)  75066960 (  0)        298662  1.5.19  233997.486044711 SOL (0.07%)
  9q16BB7WGmBxf1nJTdxH5zPnBUhtHqdqXqRFjSjuM4k7  GK9MfwWEK7BvMS8eQDaiEPnKcqMJoS7SKUiEBQY2pfxC   10%   75066990 ( -1)  75066950 (-10)        293996  1.5.19  234020.264844256 SOL (0.07%)
  3m4ov8U9Gccm8Wmkmuk4xJJajF6dSUQWZFDp36y7zHdW  CrdE3LLgvBDU7wkiPJHRH6mtspSWwZmUd9ybhrtG9adv   10%   75066991 (  0)  75066960 (  0)        298841  1.5.19  234190.607698021 SOL (0.07%)
  6M6aAEdhcWZGUeaRdc7cd81YFZ9UahgZP5D1x3MzXxmJ  CqDoHdWCXukjFNSSiqBcfNJHswbY6f8dp6vQQwtsm2AH   10%   75066990 ( -1)  75066953 ( -7)        298241  1.5.19  234447.575533751 SOL (0.07%)
  5Q1tTkjCtYFAtyZ4fxYYND8Ru65LLPuZSo8or9Keyq1h  TxDx1cjjxb15qEUSZWDpHyRnPb1vkYB8djt1scaFfhm    10%   75066991 (  0)  75066960 (  0)        298332  1.5.19  234468.555957196 SOL (0.07%)
  Gdo6FaCtTQqGYsmDQrX2icSZeqDCdVizGzBDNbiqCGbJ  6k6Yo8kmwssvayHTzp8K4jtVwmyRyjvQx6T96DNJAdYH   10%   75066991 (  0)  75066960 (  0)        298664  1.5.19  234476.580344112 SOL (0.07%)
  69QvoqgDazuAqD4ygG6YsA31HBV31L497XqmQohF5ajH  8rKN2jdSDUeDX4niaKhvcsuSbN31b9JsMJCDwuFVmqtv   10%   75066989 ( -2)  75066952 ( -8)        296888  1.5.19  234481.679063403 SOL (0.07%)
  34t5CmGFfVkdpxQgKbAJKje1NLXa1AdWUerSjnomPgxH  Ae8UD1RQBAR7PcyTJjmEsypojpuKmMGDBJxwiZorZA2M   10%   75066990 ( -1)  75066950 (-10)        298648  1.5.19  234482.009925815 SOL (0.07%)
  2dxz129YxB1xtf7Mx6HUT5JspexArNNtQt84FYueWZV7  5RCD4pZcKH3NHN4XHxvsnVAawSaConLgsZbreSN5dXpZ   10%   75066990 ( -1)  75066953 ( -7)        296336  1.5.19  234485.439863416 SOL (0.07%)
  Ge8M91yymAsZmjCyeAVCpxGA9k62CR8cxf4AhP1c38UP  GjYzNftUGedbXZXMVm8P9HsvJDosFgPEwJSzjswuDZv6   10%   75066989 ( -2)  75066952 ( -8)        296929  1.5.19  234487.653752446 SOL (0.07%)
  2zAbHUpE4MRgEwq1MWh3i9aJyzazSjUUPrmhNViqQn5W  ESjX2jRuvdr2a47JfEKjP24gNsgreE9PvSpkw4t5Xn2M   10%   75066990 ( -1)  75066953 ( -7)        296110  1.5.19  234557.271088302 SOL (0.07%)
  AhL6bJ2BF2qWSP9aE3VnFZFm9DXxWdfc4PjFBKPncsrV  BPvrBMcWjic7uu2nWragvinycHPvBCo7bCNq4pncGiag   10%   75066990 ( -1)  75066953 ( -7)        294953  1.5.19  234570.538170679 SOL (0.07%)
  5nU3v9FRnmTT7LrGKRj1reQoWWHHmPj5sfh2mTSti1uU  CTiKogMqUKyR1a4dYv4up49q4HGrLBmTNoWvXoATBAdB   10%   75066989 ( -2)  75066952 ( -8)        297885  1.5.19  234589.311691059 SOL (0.07%)
  GYZdxwPwV3wTFVC8DdZuJQZDDRCg45AtqUai8cpSyAga  FxXCvhMZGUCSi8ZwYFLVSZBFyBNyFGLEmLQxc8BLctDt   10%   75066984 ( -7)  75066929 (-31)        298289  1.5.19  234600.146666789 SOL (0.07%)
  N55CrumveNQrXFn9oL4xkFDB9K8q5PxwrYaAomrxemz   6UgxzgqkEyXMwPtgyDZSn8WGzJ5GdSWG1bSPvy7V6QwV   10%   75066991 (  0)  75066960 (  0)        298624  1.5.19  234938.992520856 SOL (0.07%)
  2D2v7sMqDuq2ekZnFhaQm4k2ErWHemZQuYf5qaVTPFmg  FQtMWMc1yo8NGpvPEFck2eCaTexa1434i87X5T9qDwqX   10%   75066989 ( -2)  75066952 ( -8)        297393  1.5.19  235344.198848421 SOL (0.07%)
  9gJT69qJUibNmfnAPBgsqSmLqVxc3kxXpw1Vk7APDrso  C6DEs3i448uhrsWMMnWYq7WsxkujcgADrCJQ4AMJ8ipj   10%   75066991 (  0)  75066960 (  0)        298901  1.5.19  235991.09137303 SOL (0.07%)
  3LboiLyZ3U1556ZBnNi9384C8Gz1LxFmzRnAojumnCJB  Dx5yxUL3DwNfhsYhLtZisNXrrecnDvoNVTougqaF7Vm    10%   75066990 ( -1)  75066953 ( -7)        298145  1.5.19  236532.172892671 SOL (0.07%)
  4ULWSuaNnhQntP3DVxg1xa4yeNLNpDnAw3gTtrhPHzEA  G7akZgkcuqr7Vw1bHXqerKeawQ2Uc1WK9VaYq9APWNLE   10%   75066990 ( -1)  75066953 ( -7)        298628  1.5.19  236807.279812024 SOL (0.07%)
  A31PGH4i5xGn7SHWpsQRhpBYUwanRuqNrHBp8bSeCSEr  EkLA4nA5jtM2t2FkNWo6XWAyvQyaJJUZoX5p7LMawoaz   10%   75066990 ( -1)  75066953 ( -7)        298120  1.5.19  236818.88382002 SOL (0.07%)
  9e7XGRqQqEvppx4Lkj6P1S7k65yWQpf3vcNzWecKSzDd  6hkfqeNAbURk7CmAQsP4Qm6WwHVF4LxHupEvQf7Tkrf1   10%   75066991 (  0)  75066960 (  0)        268764  1.5.19  237114.423782399 SOL (0.07%)
  D6pUrfgc5ZyXSfgtCBYozydRSz92pse1S7AZP58muEYk  6yg4Usyr8VkvY8m7wrbLHegKVnKaAQmEwS6cuGnwB6wT   10%   75066991 (  0)  75066960 (  0)        295971  1.5.19  239161.058993174 SOL (0.07%)
  E6cyDdEH8fiyCTusmWcZVhapAvvp2LK24zMLg4KrrAkt  BohPTSY4vvjGd9ARFqDJjRaHZKP4bde7ewBq65WgRGmA   10%   75066989 ( -2)  75066952 ( -8)        298523  1.5.19  239596.42950886 SOL (0.07%)
  8gcMmaEAXfRZKcXHeHV2x8R1ebb78inTr9xhhEuNNoTt  LneJvrSDJGqfpbsKHPfyaxSPecBtqRHTJnwyBzG4vPC    10%   75066990 ( -1)  75066952 ( -8)        296006  1.5.19  239650.026523262 SOL (0.07%)
  8GLRbAstsabZuZUx73AoyfGi1FRCWSUhRgMugFyofEz7  7PmWxxiTneGteGxEYvzj5pGDVMQ4nuN9DfUypEXmaA8o   10%   75066991 (  0)  75066960 (  0)        295831  1.5.19  239740.020286531 SOL (0.07%)
  ALp2GdA1eJV8vZHMHazCtTxNXe3BLUSco9LDASgjDs8R  4tuMshQNpAFpy1YtEHnSsE5EPN1mAT8FevWvn2UPJHNM   10%   75066991 (  0)  75066960 (  0)        298539  1.5.19  239859.810476013 SOL (0.07%)
  Fudp7uPDYNYQRxoq1Q4JiwJnzyxhVz37bGqRki3PBzS   4QhNoG3PN1FXXFhAEA2QWdor6xjXvM9pjq6MXAUV8Zg2   10%   75066990 ( -1)  75066953 ( -7)        298038  1.5.19  239913.137836698 SOL (0.07%)
  4h5muqwz35tyPQdAXkZMyVM5cnGN5oXouTZL2AFA1Fjh  9v5gci7uDiaGKRmQ2dn6WJMB94YqFaVFBTiFzBzNhyaw   10%   75066991 (  0)  75066960 (  0)        296120  1.5.19  240153.837896406 SOL (0.07%)
  12CUDzb3oe8RBQ4tYGqsuPsCbsVE4KWfktXRihXf8Ggq  AruoPzGrtfAaqUFPUDVmzUHDBjxFJLD8nodzArr346yR   10%   75066991 (  0)  75066960 (  0)        298663  1.5.19  240504.949066903 SOL (0.07%)
  9ppJrpsbbuGNjiMhhD52Ueco4KXUzVfrtNQ6tAcDab4f  EcjtYtuxBuupjeyXNdttATwoQoNL5Ck7bmrqDCj3ALT4   10%   75066990 ( -1)  75066953 ( -7)        298283  1.5.19  240574.177320647 SOL (0.07%)
  2NJZ1Ajcwtc7hZdVXTXrh2SAiAXnFkVm6MWcGjBZfPkS  H1wiu8FvJZ3UqUCGMNfVeAbtB7YJMcL9aHGPJdvPNKTm   10%   75066990 ( -1)  75066953 ( -7)        297302  1.5.19  240645.470599414 SOL (0.07%)
  2jS8AX38m8F9C5juToW1FmTufEbb1DfDzZJj9HSJcWwo  D3DfFvmLBKkX9JJNEpJRXpM1pYTVPQ5dpPQRc9F49xk4   10%   75066990 ( -1)  75066953 ( -7)        296452  1.5.19  241224.863838217 SOL (0.07%)
  FCWkGAHDWK41ANjiaoPudkCZRkvTecaEkoZQugezUnpr  GptPXjYUBUjxpRmueH6F5JcqizvjPTRDShTJQ8Vp6uN1   10%   75066989 ( -2)  75066951 ( -9)        298528  1.5.19  242495.637117764 SOL (0.07%)
  5SpUnGmQBUYVWJgagPjgyvXW4ivYPi6v3n453Qi81Lox  Fpd3kqPgWrC7J2NroJLfSeswn4oSFgySW1KhggTBjHKh   10%   75066991 (  0)  75066960 (  0)        297718  1.5.19  242590.639612165 SOL (0.07%)
  768DzddM8MGfosYS5JxjzpY2FcnqeNHUxMerFQHWmwKy  4Dh6frby55aY5NXdCGYe6yU61xusgzNHBa3GeXWMLPJ4   10%   75066991 (  0)  75066960 (  0)        297254  1.5.19  242727.031445117 SOL (0.07%)
  FvBrooVoGfi56ux9L49HZLDHMt4iznYVDAMtyH2ee2LD  5U2ConkgHqm1URrahezEs2KakfkY3hqaUJqJ7ooUy6iJ   10%   75066990 ( -1)  75066953 ( -7)        298922  1.5.19  242781.677664728 SOL (0.07%)
  6kDyGMHbuWekkcquroYNp8VRL5pQiUzEJ11gJ75qJsRy  7yXM5mUSAtBuh2TcCABvSJa3LouZ8wcLps5zTEMiwxvj   10%   75066990 ( -1)  75066953 ( -7)        295582  1.5.19  243150.072260577 SOL (0.07%)
  34viN9UrGJaVabrrSZDs8MnKwVt34nw2wv4Xkwk64shV  GhKEDkvGkf2kceG45ppzqnPD6BPXi1PyW1xGNWJdh5QW   10%   75066987 ( -4)  75066950 (-10)        299127  1.5.18  243828.454178829 SOL (0.07%)
  HzvGtvXFzMeJwNYcUu5pw8yyRxF2tLEvDSSFsAEBcBK2  9diFDvrfJD8RvBBF36cMLyQAnvafbeu3KFKAwv4od5nX   10%   75066991 (  0)  75066960 (  0)        298331  1.5.19  245208.430839317 SOL (0.07%)
  8ebFZA8NPLBZD91CwsG1HWQsa2B5Ludgdyf5Hi3sYhhs  9tedbEYypEKXAMkHcg42rn3fXY1B8hB6cdE3ZTFouXLL   10%   75066987 ( -4)  75066950 (-10)        299211  1.5.18  245757.432598007 SOL (0.07%)
  HW7ntfUHapD5o7McDuPfGvkfzrPcmuPSbZMMoe2gksKQ  5TFdzjKE6LnkhQArxWjt26yVCXskPo3fUXE8F351Cfn7   10%   75066990 ( -1)  75066953 ( -7)        298088  1.5.19  246745.240235254 SOL (0.07%)
  4QNekaDqrLmUENqkVhGCJrgHziPxkX9kridbKwunx9su  GNZ1PAAS33davY4Q1BMEpZEpVBtRtGvSpcTH5wYVkkVt   10%   75066991 (  0)  75066960 (  0)        286672  1.5.19  246824.668719685 SOL (0.07%)
  DohaxzeUj6ma9shCykxGxi7FbWnMyW9hzNjwQjZHEDV7  AdtBv6jyEjY74XhkxZkowMuGfV1r58j44WCRyeToic99   10%   75066990 ( -1)  75066953 ( -7)        298558  1.5.19  247244.086896222 SOL (0.07%)
  Arvv3uwEyDPKckw3wEFiCq9hgxMw8kawFU6doxFAWGYR  27MtjMSAQ2BGkXNuJEJkxFyCJT8dugGAaHJ9T7Gc6x4x   10%   75066991 (  0)  75066960 (  0)        296463  1.5.18  247301.875126942 SOL (0.07%)
  6wsSvrZPbjWeNNZ92KWtj94pdHj8v8sRbKsu1ZSpztpP  2naPB8XC4FWp4er8M2nxDsphXHEyQH2CSQMFaSHxWWd8   10%   75066989 ( -2)  75066949 (-11)        296479  1.5.19  247454.818534997 SOL (0.07%)
  H3WEH47bHUy9i9rmzdaxmoLagoUhbcJ2shPXcsp8QdJC  5swjaAU8NapTsmgHGAPe8wUjrK1HYPL2G64AwyH4djhz   10%   75066990 ( -1)  75066953 ( -7)        295082  1.5.19  248260.649845406 SOL (0.07%)
  2B5wMmBQkMHu9V5JbUyJuf2mJJUU286qKPsZzvQQjTNQ  3LAzTH2Fbsza2wbnKgrDcQKdaQRshQkb4w4vk12E5KH2   10%   75066991 (  0)  75066960 (  0)        298444  1.5.19  248648.2401182 SOL (0.07%)
  6hpMTcU76pCkSZHG1qdfw8ZrM9rFwFCpmKZWqBc5VXr3  J9Go27V87fCdJtjMxmFJu48ctrHzFoe6xQpA6Ecq4Wkw   10%   75066988 ( -3)  75066951 ( -9)        291735  1.5.19  250267.990111205 SOL (0.08%)
  5ya8UPujuXN8cys1EaaqMMH3auby3HTHZki73Q4Yfkff  3ZhXyB3T1uTnUPmehLnBUF5Go4Zyh1NQdCPseayr6BPq   10%   75066991 (  0)  75066960 (  0)        297955  1.5.19  250684.88633592 SOL (0.08%)
  DNWsTLvsMixgUcbM93437U8DWmJ9bZikQeBxQLHbeH5L  7X5QAMP54upioBdu75QZkz8hpYcwz7UmzAHGB2STq9da   10%   75066991 (  0)  75066960 (  0)        297418  1.5.19  250764.939321021 SOL (0.08%)
  452L4U6HbzT59EP9vLyPxddF99FBBZ7foCrcn5A9HLMK  3TC4dtYuka6fhC54ZM7yicFLmSJq38GPs1KCSwap6CuT   10%   75066991 (  0)  75066960 (  0)        296875  1.5.19  250791.838638005 SOL (0.08%)
  2us4ysyNvYJHkYi7CtRuW413Mwi34kjjFQGhZDch4DEN  5dwMHTsS52QGYVGWyui1P555XkBPzGa8uoQc7aZunLGy   10%   75066991 (  0)  75066960 (  0)        294811  1.5.19  250816.366994518 SOL (0.08%)
  D32cBNvo9qmMyMSJzWqDPQ3ujYFuW9HHNjVkwxspezQr  YT7i3TkDv9GpbQ5qEkL7dvb3fAXSuNKwCotw2MjrWxZ    10%   75066989 ( -2)  75066949 (-11)        288707  1.5.19  254616.037183775 SOL (0.08%)
  GwHH8ciFhR8vejWCqmg8FWZUCNtubPY2esALvy5tBvji  3iPuTgpWaaC6jYEY7kd993QBthGsQTK3yPCrNJyPMhCD   10%   75066991 (  0)  75066960 (  0)        295872  1.5.19  258341.864917222 SOL (0.08%)
  9rVx9wo6d3Akaq9YBw4sFuwc9oFGtzs8GsTfkHE7EH6t  EgxeaUnq3ZrZBqBDwfK1SMckhF4DKT7bfxitc1vdAsPC   10%   75066991 (  0)  75066960 (  0)        298987  1.5.19  259448.628411749 SOL (0.08%)
  3rqEEEGjHRyndHuduBcjkf17rX3hgmGACpYTQYeZ5Ltk  8xV77wuFP5BkMDdb1845hRRWZNbDNAbcV75BjMuViWpf   10%   75066990 ( -1)  75066953 ( -7)        298506  1.5.18  259933.939456803 SOL (0.08%)
  AmhQFcGvH2hjkucP78rn6GMKSbstYwyFpCDVKZUwBGrG  BdM7KCd6ZYWcaCMmHVi8YeL4jFzDVM9cRLJUeRAGSvMS   10%   75066990 ( -1)  75066953 ( -7)        296605  1.5.19  259951.102255562 SOL (0.08%)
  42xkjBQvPyicSdCET8eTWwJayTBofFLGWyCiuodd1SoT  B4H6ZdnJ8YisoYJdbkRjzt9uE5uRSpVqiFGBK9fHNugk   10%   75066991 (  0)  75066960 (  0)        298595  1.5.18  259956.435879379 SOL (0.08%)
  3RbsAuNknCTXuLyqmasnvYRpQg3MfWZ5N7WTi7ZGqdms  4aFj9VyXDrqU4TnKWnMVuPaAVTaF4GZLLKZN2iZivW29   10%   75066991 (  0)  75066960 (  0)        298533  1.5.19  260261.50134528 SOL (0.08%)
  CeuvteY4yfdbFT26h4Bj4QfDYwrRX3ZSHzdEh2X4Qw23  FHiPzSoUWggjg2K2j32TDZCUL9YLu6vJNifefGfqdqSP  100%   75066988 ( -3)  75066951 ( -9)        298078  1.5.19  260919.068489674 SOL (0.08%)
  A79u1awz7CqnxmNYEVtzWwSzup3eKPNW6w2Jrd56oZ3y  6UDU4Z9TTbYy8gcRKBd7RX3Lm2qMsSR4PMuzoyYPzLma   10%   75066991 (  0)  75066960 (  0)        299075  1.5.19  263442.919132063 SOL (0.08%)
  6h9jyRgfpmgXNyaWpbDpbxbCoF56WEbzsruhMwDn2om4  3Pb8KDmneuKuS38aDSFnK1WGz18aZN5Tuyu5ZaNUnySc   10%   75066990 ( -1)  75066952 ( -8)        296030  1.5.19  264552.346519543 SOL (0.08%)
  3fA6TU7fQCkNDDKYJeCY4Ag2gCatEsYnYL4SpkSDYfCw  7Hs9z4qsGCbQE9cy2aqgsvWupeZZGiKJgeb1eG4ZKYUH   10%   75066989 ( -2)  75066952 ( -8)        299070  1.5.18  266353.265103703 SOL (0.08%)
  2Ue9zGmDnvYRrJNEjuAdNkbbickw6fKWtbeNM7T2rakg  6559KMdiUseNSAkRcK9WcFcNTppoj6jWtKVedpMkBYCn   10%   75066991 (  0)  75066960 (  0)        278304  1.5.19  270000.238071564 SOL (0.08%)
  spcti6GQVvinbtHU9UAkbXhjTcBJaba1NVx4tmK4M5F   49DJjUX3cwFvaZD5rCAwubiz7qdRWDez9xmB381XdHru   10%   75066991 (  0)  75066960 (  0)        297040  1.5.19  273581.007786015 SOL (0.08%)
  9JJQN1WpJ8QvH6XK1xAMbSpgHSiwqBgWaeCh3ViEFmtN  6GJ23VmcWCVnm6iATqdmTjX6JeFDCCaG7PuRh5ehn9af   10%   75066990 ( -1)  75066953 ( -7)        298115  1.5.19  273933.064542829 SOL (0.08%)
  6cgsK8ph5tNUCiKG5WXLMZFX1CoL4jzuVouTPBwPC8fk  9iEFfC6qCU6DBTWxL84YQdpNmpZ9yBBu4sW62yeiEVKe   10%   75066990 ( -1)  75066953 ( -7)        297401  1.5.19  274057.093219933 SOL (0.08%)
  6BDNnr38moGRQyvx1Ehs9cM6tJDFK7LF6mUvLziyNrzW  7NwjYBhHVqEbqmMHqMqUnQ4thomaX9qXytMEULQFkAny    8%   75066991 (  0)  75066960 (  0)        298046  1.5.19  275514.818512711 SOL (0.08%)
  SFundNVpuWk89g211WKUZGkuu4BsKSp7PbnmRsPZLos   SFund7s2YPS7iCu7W2TobbuQEpVEAv9ZU7zHKiN1Gow    10%   75066990 ( -1)  75066950 (-10)        298828  1.5.19  279393.974703197 SOL (0.08%)
  4XspXDcJy3DWZsVdaXrt8pE1xhcLpXDKkhj9XyjmWWNy  9sWYTuuR4s12Q4SuSfo5CfWaFggQwA6Z8pf8dWowN5rk   10%   75066988 ( -3)  75066951 ( -9)        298615  1.5.19  287779.168059868 SOL (0.09%)
  7nYNfGS6VVxzCZmfbLGpsXYFm2LS9XRrva9hZahFmpqh  4nQTSgKURZexbRj4oXKwMiqwVCPobTcaTyMph728J3XR   10%   75066989 ( -2)  75066952 ( -8)        297169  1.5.19  290213.875477376 SOL (0.09%)
  4tS3UZfuRHzXuPenvErtRPtnZfY1KHhT96JBCQsLzKqW  D9iBN61a4MAt49gvaovohF77zqzoyQ5ERVZUoAfub3o2   10%   75066989 ( -2)  75066952 ( -8)        295299  1.5.19  290766.805440741 SOL (0.09%)
  3nRhescC7HMYC8gKti3ENiBe8LnKZUs2gzYPAjYniQCP  4ScTBfR4v2Lw9hWuzayJjfZdBKQVUeg7Rm92Rsq7HZkV   10%   75066991 (  0)  75066960 (  0)        296887  1.5.19  291284.993485464 SOL (0.09%)
  FGiEdzde7Fco2WLpNQMat299hUVoykJdaA5hxdmCzHiS  3r5ZXC1yFqMmk8VwDdUJbEdPmZ8KZvEkzd5ThEYRetTk   10%   75066990 ( -1)  75066953 ( -7)        297126  1.5.19  295790.538534392 SOL (0.09%)
  Ev8D9dwYdfebkdLgAjwiJtCkqS882Uvrit5qN6NTeHMy  BNTmegvdXzNVyc3UMTWSMSfJUryjr3fXEVErtdqrfs6y   10%   75066990 ( -1)  75066953 ( -7)        294917  1.5.19  296557.569014775 SOL (0.09%)
  8pyp3vfVPRziYdAYEyqkwytdBbdVbQmHqfQAVDcRV3w   GB44NXtM7zGm6QnzQjzHZcRKSswkJbox8aJsKiXGbFJr   10%   75066991 (  0)  75066960 (  0)        298467  1.5.19  302976.045776777 SOL (0.09%)
  CVRr5oHCAAooVbYze7CvXtRp4FUtkMCSqBZU7MVu8v8e  cover89z945JotsCRGdbjakJm4rnL5XspFSPgN1mVZj     7%   75066989 ( -2)  75066951 ( -9)        297839  1.5.19  304301.060312048 SOL (0.09%)
  3KNGMiXwhy2CAWVNpLoUt25sNngFnX1mZpaiEeVccBA6  BH7asDZbKkTmT3UWiNfmMVRgQEEpXoVThGPmQfgWwDhg   10%   75066990 ( -1)  75066953 ( -7)        296954  1.5.19  309786.173823122 SOL (0.09%)
⚠️ CjmXSapt1ouz3CZzgkRJckBEwMSo5fVdVrizLeRscwYD  B6nDYYLc2iwYqY3zdmavMmU9GjUL2hf79MkufviM2bXv    5%   74693180        74693139                   0  1.5.18  313077.311625085 SOL (0.09%)
  HahmUFR44BXFP7fVLsnd4pyaE7GoN1KKV1hdL2eVUpok  F5XcsWb5YCA39hLGP1tpUZDScTTgn1k2UeHXyaFpGXMj   10%   75066991 (  0)  75066960 (  0)        297842  1.5.19  342286.467516974 SOL (0.10%)
  ABC1U4cf9DZMwqy8ktEr4WJj8VHmVBQibbC57gEJthwY  abc1zP7ihWsgQW8z5YmfQNqMckJE5Dfx8fwUNMNVNkY    10%   75066991 (  0)  75066960 (  0)        297592  1.5.19  349350.027447009 SOL (0.11%)
  FoigPJ6kL6Gth5Er6t9d1Nkh96Skadqw63Ciyjxc1f8H  irKsY8c3sQur1XaYuQ811hzsEQJ5Hq3Yu3AAoXYnp8W     8%   75066988 ( -3)  75066951 ( -9)        296595  1.5.19  353507.009865732 SOL (0.11%)
  46yVA2WVP6ah5wLREZGHbGvvPaM55rcwDKDDikzn3BVV  2dCciM6aFdDXeiirHNTmkxh9Sda1DwfPfF1HVNZSrPcE   10%   75066991 (  0)  75066960 (  0)        298092  1.5.19  402265.856306561 SOL (0.12%)
  A1GgCvSs374GeXguQk1u91sWSLZqGVUsfJ4B4KrKjEhp  EXrWdDxFaE3Sfsbh3TV5ToGhMqu53xrmeoVdvn467jUH  100%   75066990 ( -1)  75066953 ( -7)        298434  1.5.19  416030.525062114 SOL (0.13%)
  4YGgmwyqztpJeAi3pzHQ4Gf9cWrMHCjZaWeWoCK6zz6X  8jxSHbS4qAnh5yueFp4D9ABXubKqMwXqF3HtdzQGuphp   10%   75066989 ( -2)  75066951 ( -9)        297369  1.5.19  455280.114446317 SOL (0.14%)
  a1phaKk6UbG1P2ZCpfMVFUeRM5E2EZhGvUjqWHRsrip   5CgQubGD1uwodwCe5UXDADbC69SiqXR8qq6pDMSm7ut5    5%   75066991 (  0)  75066960 (  0)        298772  1.5.19  488775.608903843 SOL (0.15%)
  GLh2T3KmWH4dUxGEVFuExffK4TLotmfeSCGg6fP7g78z  CertuszpSb9Qb8v5HhndmSdCC3WoKAiFoBEXmNhjJPUg   10%   75066989 ( -2)  75066952 ( -8)        297499  1.5.19  509448.65507104 SOL (0.15%)
  A3nXCY954bSWMo9kB9FQWoeenNq3ZBnzUwFLWHfduCSs  Co1z6vVfiric28GUV5cxum1brp2zhhF7xVo6ofkrwCpE   10%   75066990 ( -1)  75066953 ( -7)        297261  1.5.18  556560.520426153 SOL (0.17%)
  CimwgSedA3DDWwc19MHkvaCe325aqhpdwSQkzLr5UXor  B1nhG2DnVDxm42yoKyt1bwKriupytK93KoTn54LgwFyA  100%   75066989 ( -2)  75066952 ( -8)        295684  1.5.18  601680.854402431 SOL (0.18%)
  23SUe5fzmLws1M58AnGnvnUBRUKJmzCpnFQwv4M4b9Er  4PsiLMyoUQ7QRn1FFiFCvej4hsUTFzfvJnyN4bj1tmSN    0%   75066991 (  0)  75066960 (  0)        297341  1.5.19  733727.197393879 SOL (0.22%)
  ateamuvZX4iy2xYrNrMyGgwtTpWFiurNzHpmJMZwjar   ateamaZDqNWDztxnVKZhRsp4ac53KvT1rVKyU5LnL6o    10%   75066989 ( -2)  75066952 ( -8)        294143  1.5.19  761855.977380511 SOL (0.23%)
  AoUwfPuiEek2thVRDhMP7HbQb9rguyab4rDiz2NAfwwA  8HR5rCobbFMDe5EbgKdJLNDWVCeGG79w837BUxtsCngs    5%   75066991 (  0)  75066960 (  0)        296021  1.5.19  777214.706653764 SOL (0.23%)
  6yf57R7U1J1VXszE7CobdYEWQNJMPRfWEgGfaRsVNk32  CRzMxdyS56N2vkb55X5q155sSdVkjZhiFedWcbscCf7K   10%   75066990 ( -1)  75066953 ( -7)        295954  1.5.19  781247.665660425 SOL (0.23%)
  HpNN9PPbBUMQ1XEdonS8oWgRSkzQpsc3U8vP5HTZ6K9d  ALLESvwNJwM5RMn6sJtm3gw1hgpfixTr2o9qEgpAE3JD  100%   75066991 (  0)  75066960 (  0)        296960  1.5.19  867515.94475173 SOL (0.26%)
  G2TBEh2ahNGS9tGnuBNyDduNjyfUtGhMcssgRb8b6KfH  F5b1wSUtpaYDnpjLQonCZC7iyFvizLcNqTactZbwSEXK    0%   75066989 ( -2)  75066952 ( -8)        298311  1.5.19  870414.015702828 SOL (0.26%)
  3B2mGaZoFwzAnWCoZ4EAKdps4FbYbDKQ48jo8u1XWynU  edu1fZt5i82cFm6ujUoyXLMdujWxZyWYC8fkydWHRNT     0%   75066991 (  0)  75066960 (  0)        297244  1.5.18  934688.796851081 SOL (0.28%)
  9CqDvvGvSNVZDo5RskCAv4fTubpFCs9RLTrjUxEYrvNA  DzCirYWNsCECVHgSaMVg1mqMzKwtGuN2Pqm2a4HqVpTE    8%   75066991 (  0)  75066960 (  0)        298534  1.5.19  1034027.16097367 SOL (0.31%)
  5XKJwdKB2Hs7pkEXzifAysjSk6q7Rt6k5KfHwmAMPtoQ  GMpKrAwQ9oa4sJqEYQezLr8Z2TUAU72tXD4iMyfoJjbh    5%   75066989 ( -2)  75066949 (-11)        297584  1.5.18  1114285.739968496 SOL (0.34%)
  3RXKQBRv7xKTQeNdLSPhCiD4QcUfxEQ12rtgUkMf5LnS  BxFf75Vtzro2Hy3coFHKxFMZo5au8W7J8BmLC3gCMotU    0%   75066991 (  0)  75066960 (  0)        298708  1.5.19  1229340.141140472 SOL (0.37%)
  2qprVNLV2BP3yApsiFgXoArtsX2oqX3hi7XsKo1vSDVP  6BbTNwyvqyuS52eg3q5JGQprCVui8yysKUA6KQheEEPE  100%   75066989 ( -2)  75066949 (-11)        294913  1.5.19  1231537.582939561 SOL (0.37%)
  FzPoMxzQJRH87NVy1YJF8BqjrygzLLHz8MYjEZPhQHnL  EZDpkpqW2dn6aFme3HyRHmrvua7jYPiVJc6dyyT6mPxj  100%   75066990 ( -1)  75066953 ( -7)        297064  1.5.19  1231583.01804104 SOL (0.37%)
  7wohoM4oWam7mRKEYCxU3xKoFf6Q7ft1wHqkfFrmaz7K  2A14qTTZmX8wo37TJAEoAovrdugMdrbf29mSCsDerd4d  100%   75066991 (  0)  75066960 (  0)        297966  1.5.19  1231853.639411471 SOL (0.37%)
  3ab2iWqdSjyaVVrPTJXRR9e9orqK4AMCrnc5EMEHGu2e  AEf73zebQDu19NGW8R4G8HSoNos2cJe8n3Q1sRTh6XvV  100%   75066990 ( -1)  75066953 ( -7)        297281  1.5.19  1231944.476560061 SOL (0.37%)
  FfM4oeeszBdHJys9yPFV6hkcZB3ea5GuMYj81aVLURVK  HUAnFVknuEfzsybhCqKsLZoST3UCxppxfVKJ4mT9u4mL  100%   75066991 (  0)  75066960 (  0)        297249  1.5.19  1232226.743100265 SOL (0.37%)
  HzrEstnLfzsijhaD6z5frkSE2vWZEH5EUfn3bU9swo1f  DQ7D6ZRtKbBSxCcAunEkoTzQhCBKLPdzTjPRRnM6wo1f   10%   75066991 (  0)  75066960 (  0)        297115  1.5.19  1307353.755083244 SOL (0.39%)
  4o27cX8MsYmyzbYq9V5a2aMTW6eC4wxonVfkik6xGYHD  Er5zuVZSrv8hwWo3gUt2XWDwTSLUhgYD64gidQKJ7G7q  100%   75066991 (  0)  75066960 (  0)        294968  1.5.19  1330677.153264851 SOL (0.40%)
  2nDgb2py4cjHy8dxWGPUUR5641SSHCBEjnGGcuyqCKCh  5oMEJuodtg8tY7gQX3MHzGgwb242gjpxuVdnT71GD4uT  100%   75066991 (  0)  75066960 (  0)        295475  1.5.19  1330770.767962037 SOL (0.40%)
  5hqMpEYqpTFfC9YWvaud3nbUcrHf6VvXM5Emrg35ZE6W  5NVTnNN6huDEdoqR9J6Vi99mAEXFRNJqDiMmWuRHSPvJ  100%   75066990 ( -1)  75066953 ( -7)        297528  1.5.19  1332776.935049584 SOL (0.40%)
  Ninja1spj6n9t5hVYgF3PdnYz2PLnkt7rvaw3firmjs   BLADE1qNA1uNjRgER6DtUFf7FU3c1TWLLdpPeEcKatZ2    0%   75066989 ( -2)  75066952 ( -8)        296537  1.5.19  1368486.621069616 SOL (0.41%)
  2av6anjHUwvpzhbCa9vAUEEybEHs9MSU6qw3pB3o6cC7  APsCdoetBykSA9USZ6gQ2YQYSrYBR7Z2M83mg5jG9AJi  100%   75066990 ( -1)  75066952 ( -8)        297222  1.5.19  1397932.222799797 SOL (0.42%)
  JBKb5SrSLmaxVXgZMn8QKCBmCFRy92F7u6QCF64VBB4S  T3HspUFX2d3avFTNqbNPUqz9R2tVLfxVFwszQ8QpdMH   100%   75066990 ( -1)  75066953 ( -7)        297653  1.5.19  1397965.430485632 SOL (0.42%)
  ATPHtHvsQuZm3gBgm7r9fhuXWgTPB9pr6ZLD6UbDYLTP  9DyyPH6J3hQAJUc5WHowAxK2wqNPjKY3GCeULaMfg1L2  100%   75066990 ( -1)  75066953 ( -7)        297298  1.5.19  1398105.168703656 SOL (0.42%)
  2iGccofYbsAwg9GnxJA45iRNoGQfR4oYNjnptSzNx217  8d3VenzijrC639xcj4Cwii5rQMBggd81Zk4pDY1tLddi  100%   75066990 ( -1)  75066950 (-10)        295297  1.5.19  1398210.814951123 SOL (0.42%)
  Hr7bZkosETza1xtSRgmTr8DBSMMif9RPE4U49H7LXSNu  7YGvxJu1bp2X3hTctYRKEitkqpXDRSMsDo3LSZVNBXhv  100%   75066990 ( -1)  75066953 ( -7)        297133  1.5.19  1398351.066407659 SOL (0.42%)
  forb5u56XgvzxiKfRt4FVNFQKJrd2LWAfNCsCqL6P7q   76nwV8zz8tLz97SBRXH6uwHvgHXtqJDLQfF66jZhQ857    9%   75066991 (  0)  75066960 (  0)        297309  1.5.19  1681952.027081909 SOL (0.51%)
  MCFmmmXdzTKjBEoMggi8JGFJmd856uYSowuH2sCU5kx   2het6nBRLq9LLZER8fqUEk7j5pbLxq2mVGqSse2nS3tf   10%   75066990 ( -1)  75066953 ( -7)        294287  1.5.19  2030599.139304973 SOL (0.61%)
  RBFiUqjYuy4mupzZaU96ctXJBy23sRBRsL3KivDAsFM   RBFvvcGPBpgkBYmJGsphoDQJD8sszSuorM7TorWm12Y     5%   75066991 (  0)  75066960 (  0)        296731  1.5.19  2362912.835252409 SOL (0.71%)
  G28BTgvAEGH1QtPWhQkak7yHj5ucgno6dPRq4FyDsApU  HLviE73Y8623DNGEM7ZbUKS9WmnF9UhMup53vmdXnXYa  100%   75066990 ( -1)  75066953 ( -7)        297461  1.5.19  2400662.178624106 SOL (0.72%)
  cCvd3F8D9ddpdyYNb911sGsMGDm6SEYVm2Dnqhw3Qjx   A2r41ggPsPMZgUx7CEzTr2CUb3veT4p1FwsoovWaNKg3  100%   75066990 ( -1)  75066953 ( -7)        298242  1.5.19  2429163.585178631 SOL (0.73%)
  7iZsWtW2dGtLowEYT88hvtDL3AWA3cwiyqcaVPwA5hUX  Bzzg46oWfoh73TPVKMPgebLRe7kB589s1yq2AsXRXXsX  100%   75066990 ( -1)  75066953 ( -7)        298178  1.5.19  2508915.435225777 SOL (0.75%)
  EaRH6eMWnXKXhw1tqve6eUXnsoWYUFKeSbXUyBEqyPYJ  31J7VodSmMhebVNmR3BVsBkv4rrpbEmGKQm1yKscsZH5  100%   75066991 (  0)  75066960 (  0)        298582  1.5.19  2597811.783344479 SOL (0.78%)
  AZhAEf79gCXexHTasaByZQS8fks73aCUkDW8rRVLiiXT  CLj8p2TfHaAr5GxjxyksrrQcww1pjkPhamcNvixwwAFW  100%   75066990 ( -1)  75066953 ( -7)        297378  1.5.19  2608627.618636232 SOL (0.78%)
  74theNPM1xGD5Pk54vV8hyBKF8Fqs1BgzTceGmcYVxiW  4mZNLsaTzAwAS2a3dvRLQY9vKtAXhGFPrdq7c748SdRB  100%   75066991 (  0)  75066960 (  0)        297562  1.5.19  2684282.655359147 SOL (0.81%)
  et1UAfLg4ZNL3j9qXDs64hC55KizrDmCZz1oPtbnAKP   8XaZS8ymkxYc9B1XzNuhcmqso4pR6Mm3ws8Xd261jDtq  100%   75066989 ( -2)  75066952 ( -8)        298431  1.5.19  2724051.824327517 SOL (0.82%)
⚠️ 13XGR5D1oj53bRk2Gos8eMMS5eQThMrJuJJCDriX3Qne  GMYrVQ97gvkYYxJj9bxXt8cLUHmnN6ZA4EAFk1eiEEJN   10%       -               -                      0  1.5.18  2759342.007644852 SOL (0.83%)
  91yWMWM2wtsftTtgE1HA1bseRhLAg7yCuHg5GKFHn9r3  BsoS25dKjQmF2jQbpVXfzjTozYtD15z5XD6MwS2DEagW  100%   75066990 ( -1)  75066953 ( -7)        295640  1.5.19  2764562.274286489 SOL (0.83%)
  85kaUrHo3Lb4AcbVhfzceFUxEkCNBV69Uxi2HRYE5VDh  Fq5h2LenftoswobmbuWSyQtditt5UoyMvGvWH6xQVU5u  100%   75066990 ( -1)  75066953 ( -7)        297771  1.5.19  2783978.61867634 SOL (0.84%)
  3CvpuaRmrj2qFHQmPHiNpnHs3EosGSdbAHMbC2mqN9wB  sQxQAAM6RwxqAC5Ws4LkrGbErtde4F2LLCeEVAaueqW   100%   75066990 ( -1)  75066953 ( -7)        297715  1.5.19  2789438.491157745 SOL (0.84%)
  AefX1EZnJ2KALA7e9BE1ktZWrzYiBvYgatB5Rv4vr2mt  3VyAAWmYecqmzq2zzwhDRXmrLSRh3616n2zsDpWXseeb  100%   75066990 ( -1)  75066953 ( -7)        297345  1.5.19  2795556.27836651 SOL (0.84%)
  6dN61K6Y9mQpjQde4ToSjUqK6CfE1XQt8EwLeBNx6Rqm  25xzEf8cqLLEm2wyZTEBtCDchsUFm3SVESjs6eEFHJWe  100%   75066990 ( -1)  75066953 ( -7)        297433  1.5.19  2795574.586052458 SOL (0.84%)
  F9EcdNAwmvV7RtD4N9ncoPgMe3coVx2YNkJFWQM5yXgV  5rHH5G9FgxaVXwrSy37sCNwy7AjnrbgPqVs9Z6pqRnz3  100%   75066990 ( -1)  75066952 ( -8)        297783  1.5.19  2814563.903203278 SOL (0.85%)
  7brZvwqepBtCLe5NhqcENFS8jPJagny2f9eVqZoRYhMz  RqQhSG4UFgs42mJctgWGjGk4pRQpE54pr3unxfaPf25   100%   75066990 ( -1)  75066953 ( -7)        297731  1.5.19  2874908.811148203 SOL (0.86%)
  CKUox1FamkkPhzahNjaBytS5FCB7cJggCvcNLDLZCaDY  Fo2htW2sCDMc4VhPcYanoKMQiLV9UmHCBcefmWEFVULe  100%   75066990 ( -1)  75066953 ( -7)        297415  1.5.19  2877263.685596025 SOL (0.87%)
  6UFbAZu536R6H9n2EtVEdB8dbZKLBax5eoPxDWgQZvRd  E8EVmJkRAy3UhufCnpH9TCqCd5GwXUGiKHAFWc34iVU1  100%   75066991 (  0)  75066960 (  0)        298855  1.5.19  2889692.113380596 SOL (0.87%)
  4oSR2zEN1rXyo84x2CuMf9vRo6pdiQEALAAwpmToCHgJ  kki83aJMPxn1t19tmLn2fhLeUmpz3uEkjLF4EKC1gPK   100%   75066991 (  0)  75066960 (  0)        294844  1.5.19  2930805.651370946 SOL (0.88%)
  fishfishrD9BwrQQiAcG6YeYZVUYVJf3tb9QGQPMJqF   7VGU4ZwR1e1AFekqbqv2gvjeg47e1PwMPm4BfLt6rxNk    8%   75066991 (  0)  75066960 (  0)        297395  1.5.19  2939279.759599286 SOL (0.88%)
  2mMGsb5uy1Q4Dvezr8HK2E8SJoChcb2X7b61tJPaVHHd  CAf8jfgqhia5VNrEF4A7Y9VLD3numMq9DVSceq7cPhNY    7%   75066990 ( -1)  75066953 ( -7)        293139  1.5.19  2942661.492130774 SOL (0.88%)
  7sFeozTSUYomDExsQTq59W58s4Fyj4xGmh8KLGxmEjzb  7MwgR6dGw6E4G9zjG6aLHtNV9F2oha5yHUoT11a5YqK9  100%   75066989 ( -2)  75066952 ( -8)        298321  1.5.19  2980343.791595594 SOL (0.90%)
  8UQ45XfTAHb8n6kMhix3MAsinKggHTGja4LE6nnQsJPf  mMbT1QzXV3zKC2brGDuMBdAvzE4S53TDdpG7P9PMiBL   100%   75066989 ( -2)  75066952 ( -8)        297963  1.5.19  2988395.901074516 SOL (0.90%)
  8T7Lxkvk95nqbQ3rk9fhL58brZJ3gq5jRyHYeAEaScLs  85XktcbZhsnbJhsb1vueeUJhpDetkk8UQ2UzobSYEjUS  100%   75066989 ( -2)  75066952 ( -8)        298110  1.5.19  3005483.539717424 SOL (0.90%)
  38mTPbpSCC3iPrdM5KKy2kXAjdSdw6GyNndMKvgHDoNv  EKFjC8KjxYBnynAukz4Qxb6uLviuCYpwQB35WpoEt5Ck  100%   75066990 ( -1)  75066952 ( -8)        297329  1.5.19  3171102.673022762 SOL (0.95%)
  kkSzRKB5bkrtUF2MkkPUFqFmbuPppm8dkwH9SWMULJn   67HaT7sw5wEBsWojMaVb8gfbDq2uRsfg42i1YA3R91xa  100%   75066990 ( -1)  75066953 ( -7)        298105  1.5.19  3240681.616741995 SOL (0.97%)
  AuhGb13rqqCJEc8WRXfkm27SujQFdJoZKfKfr4pfrRpE  8Pep3GmYiijRALqrMKpez92cxvF4YPTzoZg83uXh14pW  100%   75066988 ( -3)  75066951 ( -9)        298483  1.5.19  3531787.04027365 SOL (1.06%)
  hz4sZFVC1MccN3WaSZ8YjHcZLewLw2JL7t1RfaRa4Pe   5KAXjaRTfyuUqd94y7rj1HmTbmdjt3266j9c44RrPRuw  100%   75066990 ( -1)  75066953 ( -7)        298031  1.5.19  3577891.99475371 SOL (1.08%)
  7cVfgArCheMR6Cs4t6vz5rfnqd56vZq4ndaBrY5xkxXy  FQwewNXahV7MiZcLpY6p1xhUs2acVGQ3U5Xxc7FzV571    8%   75066989 ( -2)  75066952 ( -8)        298502  1.5.19  4242745.353330315 SOL (1.28%)
  CAo1dCGYrB6NhHh5xb1cGjUiu86iyCfMTENxgHumSve4  51JBzSTU5rAM8gLAVQKgp4WoZerQcSqWC7BitBzgUNAm   10%   75066990 ( -1)  75066953 ( -7)        296734  1.5.19  4263297.257341886 SOL (1.28%)
  Bx7SNaHfyeLZDeja9HPsn61XYjeForHtrTxLCb6cu62o  55bmPBVG1xNHSmQ9iqzvm78Gvr8rE9wiG48MLQgJ1ywA  100%   75066989 ( -2)  75066952 ( -8)        297566  1.5.19  4508491.954785769 SOL (1.36%)
  LunaowJnt875WWoqDkhHhE93SNYHa6tfFNVn1rqc57c   LunaFpQkZsZVJL2P2BUqNDJqyVYqrw9buQnjQtMLXdK     8%   75066989 ( -2)  75066952 ( -8)        297908  1.5.19  4527870.753929689 SOL (1.36%)
  DDnAqxJVFo2GVTujibHt5cjevHMSE9bo8HJaydHoshdp  9GJmEHGom9eWo4np4L5vC6b6ri1Df2xN8KFoWixvD1Bs   10%   75066990 ( -1)  75066952 ( -8)        297870  1.5.19  4765227.618907618 SOL (1.43%)
  tDw82DpNQwC4sUqRt5zCihm1N9ktta51ZyXbWewMsWp   47e2BfrzbWdDieFZA7fc9fX1f73WTtkok2fQxSb8GueY    8%   75066988 ( -3)  75066951 ( -9)        299480  1.5.19  5066539.502411624 SOL (1.52%)
  Certusm1sa411sMpV9FPqU5dXAYhmmhygvxJ23S6hJ24  CertusDeBmqN8ZawdkxK5kFGMwBXdudvWHYwtNgNhvLu   10%   75066990 ( -1)  75066953 ( -7)        297005  1.5.19  5388046.797895501 SOL (1.62%)
  8WjzRz9HM2bWBopD6UPbi3swyRE5kk9aJojhn6fBKTJk  9c5bpzVRbfsYY2fannb4hyX5CJUPg3BfH2cL6sR7kJM4    5%   75066991 (  0)  75066960 (  0)        298040  1.5.18  5533152.378196223 SOL (1.66%)
  6TkKqq15wXjqEjNg9zqTKADwuVATR9dW3rkNnsYme1ea  GvZEwtCHZ7YtCkQCaLRVEXsyVvQkRDhJhQgB6akPme1e    0%   75066989 ( -2)  75066952 ( -8)        297665  1.5.19  5721279.349328179 SOL (1.72%)
  Fd7btgySsrjuo25CJCj7oE7VPMyezDhnx7pZkj2v69Nk  CcaHc2L43ZWjwCHART3oZoJvHLAe9hzT2DJNUpBzoTN1    7%   75066990 ( -1)  75066953 ( -7)        295249  1.5.19  6413580.090205886 SOL (1.93%)
  XkCriyrNwS3G4rzAXtG5B1nnvb5Ka1JtCku93VqeKAr   beefKGBWeSpHzYBHZXwp5So7wdQGX6mu4ZHCsH3uTar     8%   75066990 ( -1)  75066950 (-10)        293733  1.5.19  6467391.301889784 SOL (1.94%)
  Dokia75SVtetShgapUBoVFfYjL99fQyr1twxKKyTZKa3  H3GhqPMwvGLdxWg3QJGjXDSkFSJCsFk3Wx9XBTdYZykc    5%   75066990 ( -1)  75066953 ( -7)        297753  1.5.19  7856674.694277764 SOL (2.36%)
  DWvDTSh3qfn88UoQTEKRV2JnLt5jtJAVoiCo3ivtMwXP  FKsC411dik9ktS6xPADxs4Fk2SCENvAiuccQHLAPndvk    7%   75066991 (  0)  75066960 (  0)        298172  1.5.19  11040192.114345411 SOL (3.32%)
  EvnRmnMrd69kFdbLMxWkTn1icZ7DCceRhvmb2SJXqDo4  9QU2QSxhb24FUX3Tu2FpczXjpK3VYrvRudywSZaM29mF    7%   75066989 ( -2)  75066952 ( -8)        294048  1.5.19  12513403.285579691 SOL (3.76%)
  Awes4Tr6TX8JDzEhCZY2QVNimT6iD1zWHzf1vNyGvpLM  DumiCKHVqoCQKD8roLApzR5Fit8qGV5fVQsJV9sTZk4a    8%   75066989 ( -2)  75066948 (-12)        291754  1.5.19  13525282.340553779 SOL (4.07%)
  ChorusmmK7i1AxXeiTtQgQZhQNiXYU84ULeaYF1EH15n  Chorus6Kis8tFHA7AowrPMcRJk3LbApHTYpgSNXzY5KE    8%   75066990 ( -1)  75066953 ( -7)        296798  1.5.18  15304668.44361392 SOL (4.60%)
```